<!DOCTYPE html>
<html>

  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <title>2019-07-22</title>
    <meta name="description" content="rOpenSci Newsletter, including software, software reviews, what's happening in the community, events, and more.">
     <meta property="og:title" content="2019-07-22">
    <meta property="og:type" content="article">
    <meta property="description" content="rOpenSci Newsletter, including software, software reviews, what's happening in the community, events, and more.">
    <meta property="og:description" content="rOpenSci Newsletter, including software, software reviews, what's happening in the community, events, and more.">
   <meta name="twitter:card" content="summary">
   <meta name="twitter:image" content="http://ropensci.org/android-chrome-512x512.png" >


<meta name="twitter:creator" content="@rOpenSci">
<meta name="twitter:site" content="@rOpenSci">

    <link rel="stylesheet" href="/css/main.css">

	<link rel="stylesheet" href="/css/bootstrap/css/bootstrap.css">

    <link rel="canonical" href="http://localhost:4000/2019-07-22/">

    <link rel="shortcut icon" href="../assets/img/favicon-16x16.png">

    <link rel="icon" type="image/png" sizes="32x32" href="/assets/img/favicon-32x32.png">
    <link rel="icon" type="image/png" sizes="16x16" href="/assets/img/favicon-16x16.png">

    <link rel="alternate" title="rOpenSci Newsletter" type="application/json" href="http://localhost:4000/feed.json" />

    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.9/css/all.css" integrity="sha384-5SOiIsAziJl6AWe0HWRKTXlfcSHKmYV4RBF18PPJ173Kzn7jzMyFuTtk8JA7QQG1" crossorigin="anonymous">
</head>



  <body>

    <header class="site-header">

  <div class="wrapper">

    <a class="site-title" href="/">rOpenSci Newsletter</a>

    <nav class="site-nav">
      <a href="#" class="menu-icon">
        <svg viewBox="0 0 18 15">
          <path fill="#424242" d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.031C17.335,0,18,0.665,18,1.484L18,1.484z"/>
          <path fill="#424242" d="M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0c0-0.82,0.665-1.484,1.484-1.484 h15.031C17.335,6.031,18,6.696,18,7.516L18,7.516z"/>
          <path fill="#424242" d="M18,13.516C18,14.335,17.335,15,16.516,15H1.484C0.665,15,0,14.335,0,13.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.031C17.335,12.031,18,12.696,18,13.516L18,13.516z"/>
        </svg>
      </a>

      <div class="trigger">
        
          
        
          
        
          
        
          
        
      </div>
    </nav>

  </div>

</header>


    <div class="page-content">
      <div class="wrapper">
        <div class="post">

  <header class="post-header">
  	<center><img src="../assets/img/icon_lettering_color.svg" width="375"></center>
    <!-- <h1 class="post-title">2019-07-22</h1> -->
    <center><h1 style="color:grey">Jul 22, 2019</h1></center>
  </header>
  <br>
  <article class="post-content">
    <h2 id="ropensci-hq">rOpenSci HQ</h2>

<ul>
  <li><a href="https://ropensci.org/commcalls/2019-06-28/">Recording and notes</a> and <a href="https://discuss.ropensci.org/t/community-call-managing-multilingual-communities/1722">public discussion</a> are up from our latest <a href="https://ropensci.org/blog/2019/06/13/commcall-jun2019/">community call on Involving Multilingual Communities</a>.</li>
  <li>Join us for our next community call <a href="https://ropensci.org/blog/2019/07/11/commcall-jul2019/">Reproducible Research with R</a> on July 30th, 2019. Speakers are <a href="https://ropensci.org/authors/ben-marwick/">Ben Marwick</a>, <a href="https://ropensci.org/authors/anna-krystalli/">Anna Krystalli</a>, and <a href="https://ropensci.org/authors/karthik-ram/">Karthik Ram</a>.</li>
  <li><a href="https://ropensci.org/blog/2019/07/15/expanding-software-review/">rOpenSci Announces $678K Award from the Sloan Foundation to Expand Software Peer Review</a> &amp; <a href="https://ropensci.org/blog/2019/07/18/ropensci-hiring/">rOpenSci Hiring for New Position in Statistical Software Testing and Peer Review</a></li>
</ul>

<p><br /></p>

<h2 id="software-">Software üì¶</h2>

<p>CRAN: <img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px" />
GitHub: <img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px" /></p>

<h3 id="new-versions">New Versions</h3>

<ul>
  <li>A new version (<code class="highlighter-rouge">v7.5.2</code>) of <code class="highlighter-rouge">drake</code> is on CRAN - a pipeline toolkit for reproducible computation at scale. See the <a href="https://github.com/ropensci/drake/releases/tag/v7.5.2">release notes</a> for changes. Checkout the <a href="https://ropensci.github.io/drake/">docs</a> to get started. <a target="_blank" href="https://cran.rstudio.com/web/packages/drake"><img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a> <a target="_blank" href="https://github.com/ropensci/drake"><img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a>
    <blockquote>
      <p>remove accidental creation of a dot file</p>
    </blockquote>
  </li>
  <li>A new version (<code class="highlighter-rouge">v0.6.4</code>/<code class="highlighter-rouge">v0.6.5</code>) of <code class="highlighter-rouge">rerddap</code> is on CRAN - General Purpose Client for ERDDAP Servers. See the <a href="https://github.com/ropensci/rerddap/releases/tag/v0.6.5">release notes</a> for changes. Checkout the <a href="https://cran.r-project.org/web/packages/rerddap/vignettes/">vignettes</a> to get started. <a target="_blank" href="https://cran.rstudio.com/web/packages/rerddap"><img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a> <a target="_blank" href="https://github.com/ropensci/rerddap"><img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a>
    <blockquote>
      <p>fix internal http request error handler; fix a test</p>
    </blockquote>
  </li>
  <li>A new version (<code class="highlighter-rouge">v1.0.4</code>) of <code class="highlighter-rouge">available</code> is on CRAN - check if the title of a package is available. See the <a href="https://github.com/ropenscilabs/available/blob/master/NEWS.md">release notes</a> for changes. Checkout the <a href="https://github.com/ropenscilabs/available#available">README</a> to get started. <a target="_blank" href="https://cran.rstudio.com/web/packages/available"><img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a> <a target="_blank" href="https://github.com/ropenscilabs/available"><img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a>
    <blockquote>
      <p>tests requiring network access are not skipped on CRAN</p>
    </blockquote>
  </li>
  <li>A new version (<code class="highlighter-rouge">v1.2</code>) of <code class="highlighter-rouge">cld3</code> is on CRAN - Bindings to Google‚Äôs Compact Language Detector 3. See the <a href="https://github.com/ropensci/cld3/blob/master/NEWS">release notes</a> for changes. Checkout the <a href="https://docs.ropensci.org/cld3/">docs</a> to get started. <a target="_blank" href="https://cran.rstudio.com/web/packages/cld3"><img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a> <a target="_blank" href="https://github.com/ropensci/cld3"><img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a>
    <blockquote>
      <p>update to protobuf 3.8 for windows; suppress warnings about pragmas in autogen protobuf headers</p>
    </blockquote>
  </li>
  <li>A new version (<code class="highlighter-rouge">v0.2.7</code>) of <code class="highlighter-rouge">fingertipsR</code> is on CRAN - Fingertips data for public health <a href="http://fingertips.phe.org.uk/">http://fingertips.phe.org.uk/</a>. See the <a href="https://github.com/ropensci/fingertipsR/blob/master/NEWS.md">release notes</a> for changes. Checkout the <a href="https://cran.rstudio.com/web/packages/fingertipsR/vignettes/">vignettes</a> to get started. <a target="_blank" href="https://cran.rstudio.com/web/packages/fingertipsR"><img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a> <a target="_blank" href="https://github.com/ropensci/fingertipsR"><img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a>
    <blockquote>
      <p>add option <code class="highlighter-rouge">AreaTypeId="All</code> in <code class="highlighter-rouge">fingertips_data()</code>; added param to <code class="highlighter-rouge">area_types()</code>; change to <code class="highlighter-rouge">category_types()</code>; add http request retry</p>
    </blockquote>
  </li>
  <li>A new version (<code class="highlighter-rouge">v0.2.0</code>) of <code class="highlighter-rouge">vitae</code> is on CRAN - R Markdown Resumes and CVs. See the <a href="https://github.com/ropenscilabs/vitae/releases/tag/v0.2.0">release notes</a> for changes. Checkout the <a href="https://docs.ropensci.org/vitae/">docs</a> to get started. <a target="_blank" href="https://cran.rstudio.com/web/packages/vitae"><img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a> <a target="_blank" href="https://github.com/ropenscilabs/vitae"><img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a>
    <blockquote>
      <p>theme support for moderncv; other template changes; breaking change to <code class="highlighter-rouge">bibliography_entries</code></p>
    </blockquote>
  </li>
  <li>A new version (<code class="highlighter-rouge">v2.0.2</code>) of <code class="highlighter-rouge">ijtiff</code> is on CRAN - Comprehensive TIFF I/O with full support for ImageJ TIFF files. See the <a href="https://github.com/ropensci/ijtiff/blob/master/NEWS.md">release notes</a> for changes. Checkout the <a href="https://ropensci.github.io/ijtiff">docs</a> to get started. <a target="_blank" href="https://cran.rstudio.com/web/packages/ijtiff"><img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a> <a target="_blank" href="https://github.com/ropensci/ijtiff"><img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a>
    <blockquote>
      <p>bug fixes</p>
    </blockquote>
  </li>
  <li>A new version (<code class="highlighter-rouge">v0.6.0</code>) of <code class="highlighter-rouge">bomrang</code> is on CRAN - Australian government Bureau of Meteorology (BOM) data. See the <a href="https://github.com/ropensci/bomrang/blob/master/NEWS.md">release notes</a> for changes. Checkout the <a href="https://docs.ropensci.org/bomrang/">docs</a> to get started. <a target="_blank" href="https://cran.rstudio.com/web/packages/bomrang"><img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a> <a target="_blank" href="https://github.com/ropensci/bomrang"><img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a>
    <blockquote>
      <p>many bug fixes</p>
    </blockquote>
  </li>
  <li>A new version (<code class="highlighter-rouge">v0.26.1</code>) of <code class="highlighter-rouge">git2r</code> is on CRAN - an interface to the libgit2 library, a pure C implementation of the Git core methods. See the <a href="https://github.com/ropensci/git2r/releases/tag/v0.26.1">release notes</a> for changes. Checkout the <a href="https://github.com/ropensci/git2r#introduction">README</a> to get started. <a target="_blank" href="https://cran.rstudio.com/web/packages/git2r"><img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a> <a target="_blank" href="https://github.com/ropensci/git2r"><img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a>
    <blockquote>
      <p>fixed the broken build on Solaris on CRAN</p>
    </blockquote>
  </li>
  <li>A new version (<code class="highlighter-rouge">v0.4.2</code>) of <code class="highlighter-rouge">traits</code> is on CRAN - species trait data from around the web. See the <a href="https://github.com/ropensci/traits/releases/tag/v0.4.2">release notes</a> for changes. Checkout the <a href="https://github.com/ropensci/traits#traits">README</a> to get started. <a target="_blank" href="https://cran.rstudio.com/web/packages/traits"><img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a> <a target="_blank" href="https://github.com/ropensci/traits"><img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a>
    <blockquote>
      <p>minor betydb change; <code class="highlighter-rouge">taxa_search()</code> breaking change (Traitbank service completely changed)</p>
    </blockquote>
  </li>
  <li>A new version (<code class="highlighter-rouge">v0.8.0</code>) of <code class="highlighter-rouge">crul</code> is on CRAN - an R6 based HTTP client for R made for developers. See the <a href="https://github.com/ropensci/crul/releases/tag/v0.8.0">release notes</a> for changes. Checkout the <a href="https://ropensci.github.io/crul/">docs</a> to get started. <a target="_blank" href="https://cran.rstudio.com/web/packages/crul"><img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a> <a target="_blank" href="https://github.com/ropensci/crul"><img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a>
    <blockquote>
      <p>can now pass params through <code class="highlighter-rouge">parse()</code> to <code class="highlighter-rouge">iconv()</code>; use httpcode pkg instead of fauxpas inside <code class="highlighter-rouge">ok()</code></p>
    </blockquote>
  </li>
  <li>A new version (<code class="highlighter-rouge">v0.2.0</code>) of <code class="highlighter-rouge">grainchanger</code> is on CRAN - Data aggregation methods for raster data. See the <a href="https://github.com/ropensci/grainchanger/releases/tag/v0.2.0">release notes</a> for changes. Checkout the <a href="https://docs.ropensci.org/grainchanger/">docs</a> to get started. <a target="_blank" href="https://cran.rstudio.com/web/packages/grainchanger"><img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a> <a target="_blank" href="https://github.com/ropensci/grainchanger"><img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a></li>
  <li>A new version (<code class="highlighter-rouge">v0.9.8</code>) of <code class="highlighter-rouge">taxize</code> is on CRAN - taxonomic toolbelt for R. See the <a href="https://github.com/ropensci/taxize/releases/tag/v0.9.8">release notes</a> for changes. Checkout the <a href="https://taxize.dev/">taxize book</a> to get started. <a target="_blank" href="https://cran.rstudio.com/web/packages/taxize"><img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a> <a target="_blank" href="https://github.com/ropensci/taxize"><img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a>
    <blockquote>
      <p>new fxn <code class="highlighter-rouge">taxize_options</code>; all <code class="highlighter-rouge">get_*</code> fxns gain ability to track progress and restart in case of interruption - &amp; prettier progress and summary info; bug fixes</p>
    </blockquote>
  </li>
  <li>A new version (<code class="highlighter-rouge">v0.4.0</code>) of <code class="highlighter-rouge">worrms</code> is on CRAN - client for <a href="http://www.marinespecies.org/">World Register of Marine Species</a>. See the <a href="https://github.com/ropensci/worrms/releases/tag/v0.4.0">release notes</a> for changes. Checkout the <a href="https://cran.rstudio.com/web/packages/worrms/vignettes/worrms.html">vignette</a> to get started. <a target="_blank" href="https://cran.rstudio.com/web/packages/worrms"><img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a> <a target="_blank" href="https://github.com/ropensci/worrms"><img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a>
    <blockquote>
      <p>new fxns <code class="highlighter-rouge">wm_ranks_id()</code>/<code class="highlighter-rouge">wm_ranks_name()</code> for getting taxonomic ranks by rank identifier or rank name; new fxn <code class="highlighter-rouge">wm_records_rank()</code> for getting AphiaRecords for a given rank id</p>
    </blockquote>
  </li>
  <li>A new version (<code class="highlighter-rouge">v0.9.0</code>) of <code class="highlighter-rouge">bold</code> is on CRAN - interface to Bold Systems (<a href="http://www.boldsystems.org/">http://www.boldsystems.org/</a>) API. See the <a href="https://github.com/ropensci/bold/releases/tag/v0.9.0">release notes</a> for changes. Checkout the <a href="https://cran.rstudio.com/web/packages/bold/vignettes/bold.html">vignette</a> to get started. <a target="_blank" href="https://cran.rstudio.com/web/packages/bold"><img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a> <a target="_blank" href="https://github.com/ropensci/bold"><img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a>
    <blockquote>
      <p>allow curl options to be passed into <code class="highlighter-rouge">bold_identify_parents()</code>; bug fix in <code class="highlighter-rouge">bold_identify()</code></p>
    </blockquote>
  </li>
  <li>A new version (<code class="highlighter-rouge">v1.3.0</code>) of <code class="highlighter-rouge">fulltext</code> is on CRAN - full text of scholarly articles across many data sources. See the <a href="https://github.com/ropensci/fulltext/releases/tag/v1.3.0">release notes</a> for changes. Checkout the <a href="https://ropenscilabs.github.io/fulltext-book/">fulltext book</a> to get started. <a target="_blank" href="https://cran.rstudio.com/web/packages/fulltext"><img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a> <a target="_blank" href="https://github.com/ropensci/fulltext"><img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a>
    <blockquote>
      <p><code class="highlighter-rouge">ft_get()</code> gains new data source: ScienceDirect; <code class="highlighter-rouge">ft_get()</code> performance improvements; curl options can now be passed through nearly all fxns</p>
    </blockquote>
  </li>
  <li>A new version (<code class="highlighter-rouge">v3.0.4</code>) of <code class="highlighter-rouge">rfishbase</code> is on CRAN - interface to Fishbase data. See the <a href="https://github.com/ropensci/rfishbase/blob/master/NEWS.md">release notes</a> for changes. Checkout the <a href="http://ropensci.github.io/rfishbase/">docs</a> to get started. <a target="_blank" href="https://cran.rstudio.com/web/packages/rfishbase"><img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a> <a target="_blank" href="https://github.com/ropensci/rfishbase"><img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a>
    <blockquote>
      <p>use latest data version</p>
    </blockquote>
  </li>
  <li>A new version (<code class="highlighter-rouge">v1.1.0</code>) of <code class="highlighter-rouge">jsonvalidate</code> is on CRAN - Validate JSON. See the <a href="https://github.com/ropensci/jsonvalidate/releases/tag/v1.1.0">release notes</a> for changes. Checkout the <a href="http://ropensci.github.io/jsonvalidate/">docs</a> to get started. <a target="_blank" href="https://cran.rstudio.com/web/packages/jsonvalidate"><img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a> <a target="_blank" href="https://github.com/ropensci/jsonvalidate"><img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a></li>
</ul>

<p><br /><br /></p>

<h2 id="software-review-">Software Review ‚úî</h2>

<p>We accept community contributed packages via our software review system - an open software review system, sorta like scholarly paper review, but way better. We‚Äôll highlight newly onboarded packages here. A huge thanks to our reviewers, who do a lot of work reviewing (see the <a href="https://ropensci.org/blog/2016/03/28/software-review">blog post on our review system</a>),
and the authors of the packages!</p>

<p>If you want to be a reviewer fill out <a href="https://ropensci.org/onboarding/">this short form</a>, and we‚Äôll ping you when there‚Äôs a submission that fits in your area of expertise.</p>

<p>The following two packages recently went through our software-review process and have been approved:</p>

<ul>
  <li><a href="https://github.com/ropensci/grainchanger">grainchanger</a> &gt; Moving-Window and Direct Data Aggregation
    <ul>
      <li>Author: <a href="https://github.com/laurajanegraham">Laura Jane Graham</a></li>
      <li>Issue: <a href="https://github.com/ropensci/onboarding/issues/289">ropensci/onboarding#289</a></li>
      <li>Reviewers: not yet assigned
        <ul>
          <li><a href="https://github.com/mbjoseph">Max Joseph</a></li>
          <li><a href="https://github.com/johnbaums">John Baumgartner</a></li>
        </ul>
      </li>
    </ul>
  </li>
  <li><a href="https://github.com/AldoCompagnoni/popler">popler</a> &gt; Browse and query the popler database
    <ul>
      <li>Author: <a href="https://github.com/AldoCompagnoni">Aldo Compagnoni</a></li>
      <li>Issue: <a href="https://github.com/ropensci/software-review/issues/254">ropensci/software-review#254</a></li>
      <li>Reviewers:
        <ul>
          <li><a href="https://github.com/cgries">Corinna Gries</a></li>
          <li><a href="https://github.com/bpbond">Ben Bond-Lamberty</a></li>
        </ul>
      </li>
    </ul>
  </li>
</ul>

<p><br /><br /></p>

<h2 id="on-the-blog">On the blog</h2>

<p><a href="https://ropensci.org/authors/stefanie-butland/">Stefanie Butland</a> wrote about <a href="https://ropensci.org/blog/2019/07/08/user2019/">folks talking about rOpenSci projects at useR in Toulouse, France</a>. The conference already happened; slides are linked from the <a href="http://www.user2019.fr/talk_schedule/">talk schedule page</a>, and video of talks may be up later.</p>

<p><br /></p>

<p><a href="https://ropensci.org/authors/laura-graham/">Laura Graham</a> wrote about her R package that recently went through software review <a href="https://ropensci.org/blog/2019/07/16/grainchanger/">Aggregating spatial data with the grainchanger package</a>. <a href="https://docs.ropensci.org/grainchanger/">grainchanger</a> facilitates spatial data aggregation.</p>

<p><img src="../assets/img/grainchainger-logo.png" width="200" /></p>

<p><br /><br /></p>

<h2 id="use-cases">Use Cases</h2>

<p>The following 20 works use/cite rOpenSci software:</p>

<ul>
  <li>Steidinger et al. cited <a href="https://github.com/ropensci/taxize">taxize</a> in their paper <a href="https://doi.org/10.1038/s41586-019-1128-0">Climatic controls of decomposition drive the global biogeography of forest-tree symbioses</a> <sup id="fnref:1" role="doc-noteref"><a href="#fn:1" class="footnote">1</a></sup></li>
  <li>Mitchell used <a href="https://github.com/ropensci/RSelenium">RSelenium</a> in their thesis <a href="https://uknowledge.uky.edu/cgi/viewcontent.cgi?article=1042&amp;context=biochem_etds">Computational Tools for the Untargeted Assignment of FT-MS Metabolomics Datasets</a> <sup id="fnref:2" role="doc-noteref"><a href="#fn:2" class="footnote">2</a></sup></li>
  <li>Bagley et al. used <a href="https://github.com/ropensci/taxize">taxize</a> in their paper <a href="https://doi.org/10.1016/j.ecolind.2019.04.088">High-throughput environmental DNA analysis informs a biological assessment of an urban stream</a> <sup id="fnref:3" role="doc-noteref"><a href="#fn:3" class="footnote">3</a></sup></li>
  <li>Foisy et al. used <a href="https://github.com/ropensci/taxize">taxize</a> in their paper <a href="https://doi.org/10.1111/1365-2745.13203">Do latex and resin canals spur plant diversification? Re‚Äêexamining a classic example of escape and radiate coevolution</a> <sup id="fnref:4" role="doc-noteref"><a href="#fn:4" class="footnote">4</a></sup></li>
  <li>Og≈Çuszka et al. used <a href="https://github.com/ropensci/plotly">plotly</a> in their paper <a href="https://doi.org/10.1016/j.cmpb.2019.05.023">Evaluate Cutpoints: Adaptable continuous data distribution system for determining survival in Kaplan-Meier estimator</a> <sup id="fnref:5" role="doc-noteref"><a href="#fn:5" class="footnote">5</a></sup></li>
  <li>Westmeijer et al. used <a href="https://github.com/ropensci/rgbif">rgbif</a> in their paper <a href="https://doi.org/10.1016/j.algal.2019.101529">Mechanistic niche modelling to identify favorable growth sites of temperate macroalgae</a> <sup id="fnref:6" role="doc-noteref"><a href="#fn:6" class="footnote">6</a></sup></li>
  <li>Kortz et al. used <a href="https://github.com/ropensci/plotly">plotly</a> in their paper <a href="https://doi.org/10.1098/rsbl.2019.0133">Increases in local richness (Œ±-diversity) following invasion are offset by biotic homogenization in a biodiversity hotspot</a> <sup id="fnref:7" role="doc-noteref"><a href="#fn:7" class="footnote">7</a></sup></li>
  <li>P√©rez-Palma et al. used <a href="https://github.com/ropensci/plotly">plotly</a> in their paper <a href="https://doi.org/10.1093/nar/gkz411">Simple ClinVar: an interactive web server to explore and retrieve gene and disease variants aggregated in ClinVar database</a> <sup id="fnref:8" role="doc-noteref"><a href="#fn:8" class="footnote">8</a></sup></li>
  <li>Green et al. used <a href="https://github.com/ropensci/hunspell">hunspell</a> and <a href="https://github.com/ropensci/cld2">cld2</a> in their paper <a href="https://doi.org/10.12688/gatesopenres.12999.1">What is the best method of family planning for me?: a text mining analysis of messages between users and agents of a digital health service in Kenya</a> <sup id="fnref:9" role="doc-noteref"><a href="#fn:9" class="footnote">9</a></sup></li>
  <li>Mihalitsis &amp; Bellwood used <a href="https://github.com/ropensci/rotl">rotl</a> in their research <a href="https://doi.org/10.1007/s00338-019-01820-w">Morphological and functional diversity of piscivorous fishes on coral reefs</a> <sup id="fnref:10" role="doc-noteref"><a href="#fn:10" class="footnote">10</a></sup></li>
  <li>Kannenberg et al. used <a href="https://github.com/ropensci/MODIStools">MODIStools</a> in their paper <a href="https://doi.org/10.1111/gcb.14710">Linking drought legacy effects across scales: From leaves to tree rings to ecosystems</a> <sup id="fnref:11" role="doc-noteref"><a href="#fn:11" class="footnote">11</a></sup></li>
  <li>Aspillaga et al. used <a href="https://github.com/ropensci/plotly">plotly</a> in their paper <a href="https://doi.org/10.1111/2041-210x.13232">Modelling the three‚Äêdimensional space use of aquatic animals combining topography and Eulerian telemetry data</a> <sup id="fnref:12" role="doc-noteref"><a href="#fn:12" class="footnote">12</a></sup></li>
  <li>Gibson et al. used <a href="https://github.com/ropensci/plotly">plotly</a> in their paper <a href="https://doi.org/10.1038/s41598-019-43875-3">Gut microbiome differences between wild and captive black rhinoceros ‚Äì implications for rhino health</a> <sup id="fnref:13" role="doc-noteref"><a href="#fn:13" class="footnote">13</a></sup></li>
  <li>Sinval et al. used <a href="https://github.com/ropensci/skimr">skimr</a> in their paper <a href="https://doi.org/10.1007/s11482-019-09730-3">The Quality of Work Life Scale: Validity Evidence from Brazil and Portugal</a> <sup id="fnref:14" role="doc-noteref"><a href="#fn:14" class="footnote">14</a></sup></li>
  <li>Iserman et al. used <a href="https://github.com/ropensci/hunspell">hunspell</a> in their conference paper <a href="https://www.aclweb.org/anthology/W19-3025">Dictionaries and Decision Trees for the 2019 CLPsych Shared Task</a> <sup id="fnref:15" role="doc-noteref"><a href="#fn:15" class="footnote">15</a></sup></li>
  <li>Goodman et al. used <a href="https://github.com/ropensci/rfishbase">rfishbase</a> in their paper <a href="https://doi.org/10.1111/jbi.13595">The relationship between geographic range extent, sea surface temperature and adult traits in coastal temperate fishes</a> <sup id="fnref:16" role="doc-noteref"><a href="#fn:16" class="footnote">16</a></sup></li>
  <li>Fioravanti et al. used <a href="https://github.com/ropensci/RSelenium">RSelenium</a> in their paper <a href="https://doi.org/10.1002/joc.6177">A new homogenized daily data set for temperature variability assessment in Italy</a> <sup id="fnref:17" role="doc-noteref"><a href="#fn:17" class="footnote">17</a></sup></li>
  <li>Logemann et al. used <a href="https://github.com/ropensci/magick">magick</a> in their paper <a href="https://doi.org/10.1016/j.chemosphere.2019.05.246">Using the HPTLC-bioluminescence bacteria assay for the determination of acute toxicities in marine sediments and its eligibility as a monitoring assessment tool</a> <sup id="fnref:18" role="doc-noteref"><a href="#fn:18" class="footnote">18</a></sup></li>
  <li>Al-Mustanjid used <a href="https://github.com/ropensci/rentrez">rentrez</a> in their paper <a href="http://www.iaees.org/publications/journals/nb/articles/2019-9(2)/design-of-a-common-pathway-drug-for-cardiovascular-diseases.pdf">Design of a common pathway drug for all types of cardiovascular diseases: A network biology approach</a> <sup id="fnref:19" role="doc-noteref"><a href="#fn:19" class="footnote">19</a></sup></li>
  <li>Wambui &amp; Musenge used <a href="https://github.com/ropensci/MODIStools">MODIStools</a> in their paper <a href="https://doi.org/10.1186/s40795-019-0296-5">A space-time analysis of recurrent malnutrition-related hospitalisations in Kilifi, Kenya for children under-5 years</a> <sup id="fnref:20" role="doc-noteref"><a href="#fn:20" class="footnote">20</a></sup></li>
</ul>

<p><br /><br /></p>

<h2 id="call-for-contributors">Call For Contributors</h2>

<p>Part of the mission of rOpenSci is making sustainable software that users can rely on. Some software maintainers need to give up maintenance due to a variety of circumstances. When that happens we try to find new maintainers. Checkout our <a href="https://devguide.ropensci.org/contributingguide.html">Contributing Guide</a> for why and how to contribute to rOpenSci.</p>

<p>We‚Äôve had six recent examples of maintainer transitions within rOpenSci:</p>

<ul>
  <li><a href="https://github.com/ropensci/RSelenium">RSelenium</a>: now maintained by <a href="https://github.com/juyeongkim">Ju Kim</a></li>
  <li><a href="https://github.com/ropensci/chromer">chromer</a>: now maintained by <a href="https://github.com/orchid00">Paula Andrea</a></li>
  <li><a href="https://github.com/ropensci/qualtRics">qualtRics</a>: now maintained by <a href="https://github.com/juliasilge">Julia Silge</a></li>
  <li><a href="https://github.com/ropensci/rsnps">rsnps</a>: now maintained by <a href="https://github.com/jooolia">Julia Gustavsen</a> and <a href="https://github.com/sinarueeger">Sina R√ºeger</a></li>
  <li><a href="https://github.com/ropensci/rdpla">rdpla</a>: now maintained by <a href="https://github.com/acolum">Alyssa Columbus</a></li>
  <li><a href="https://github.com/ropensci/webchem">webchem</a>: now maintained by <a href="https://github.com/eriksapper">Erik Sapper</a></li>
</ul>

<p>We don‚Äôt have any packages looking for new maintainers - if you are a maintainer and want to hand over that role to someone else get in touch with us.</p>

<p><br /><br /></p>

<hr style="display: block; height: 1px; border: 0; border-top: 3px solid #7CCCC8; margin: 1em 0; padding: 0; " />

<p><br /><br /></p>

<h3 id="keep-up-with-ropensci">Keep up with rOpenSci</h3>

<ul>
  <li>Mailing list: Sign up with an email address to get this newsletter sent to your inbox -&gt; <a href="https://ropensci.org/#subscribe">ropensci.org/#subscribe</a></li>
  <li>Alternatively, you can subscribe to this newsletter via our XML feed at <a href="https://news.ropensci.org/feed.xml">https://news.ropensci.org/feed.xml</a> or our JSON feed at <a href="https://news.ropensci.org/feed.json">https://news.ropensci.org/feed.json</a></li>
  <li>rOpenSci on Twitter: <a href="https://twitter.com/ropensci">@ropensci</a></li>
  <li>The rOpenSci blog at <a href="https://ropensci.org/blog">ropensci.org/blog</a> - you can subscribe in any RSS aggregator, or manually via <a href="https://ropensci.org/feed.xml">https://ropensci.org/feed.xml</a>. We also announce new blog posts on our Twitter account.</li>
</ul>

<p><br /></p>

<h4 id="footnotes">Footnotes</h4>

<div class="footnotes" role="doc-endnotes">
  <ol>
    <li id="fn:1" role="doc-endnote">
      <p>Steidinger, B. S., Crowther, T. W., Liang, J., Van Nuland, M. E., Werner, G. D. A., ‚Ä¶ Peay, K. G. (2019). Climatic controls of decomposition drive the global biogeography of forest-tree symbioses. Nature, 569(7756), 404‚Äì408. <a href="https://doi.org/10.1038/s41586-019-1128-0">https://doi.org/10.1038/s41586-019-1128-0</a>¬†<a href="#fnref:1" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:2" role="doc-endnote">
      <p>Mitchell, J. M. (2019). Computational Tools for the Untargeted Assignment of FT-MS Metabolomics Datasets. Dissertation. <a href="https://uknowledge.uky.edu/cgi/viewcontent.cgi?article=1042&amp;context=biochem_etds">https://uknowledge.uky.edu/cgi/viewcontent.cgi?article=1042&amp;context=biochem_etds</a>¬†<a href="#fnref:2" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:3" role="doc-endnote">
      <p>Bagley, M., Pilgrim, E., Knapp, M., Yoder, C., Santo Domingo, J., &amp; Banerji, A. (2019). High-throughput environmental DNA analysis informs a biological assessment of an urban stream. Ecological Indicators, 104, 378‚Äì389. <a href="https://doi.org/10.1016/j.ecolind.2019.04.088">https://doi.org/10.1016/j.ecolind.2019.04.088</a>¬†<a href="#fnref:3" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:4" role="doc-endnote">
      <p>Foisy, M. R., Albert, L. P., Hughes, D. W. W., &amp; Weber, M. G. (2019). Do latex and resin canals spur plant diversification? Re‚Äêexamining a classic example of escape and radiate coevolution. Journal of Ecology. <a href="https://doi.org/10.1111/1365-2745.13203">https://doi.org/10.1111/1365-2745.13203</a>¬†<a href="#fnref:4" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:5" role="doc-endnote">
      <p>Og≈Çuszka, M., Orzechowska, M., Jƒôdroszka, D., Witas, P., &amp; Bednarek, A. K. (2019). Evaluate Cutpoints: Adaptable continuous data distribution system for determining survival in Kaplan-Meier estimator. Computer Methods and Programs in Biomedicine, 177, 133‚Äì139. <a href="https://doi.org/10.1016/j.cmpb.2019.05.023">https://doi.org/10.1016/j.cmpb.2019.05.023</a>¬†<a href="#fnref:5" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:6" role="doc-endnote">
      <p>Westmeijer, G., Everaert, G., Pirlet, H., De Clerck, O., &amp; Vandegehuchte, M. B. (2019). Mechanistic niche modelling to identify favorable growth sites of temperate macroalgae. Algal Research, 41, 101529. <a href="https://doi.org/10.1016/j.algal.2019.101529">https://doi.org/10.1016/j.algal.2019.101529</a>¬†<a href="#fnref:6" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:7" role="doc-endnote">
      <p>Kortz, A. R., &amp; Magurran, A. E. (2019). Increases in local richness (Œ±-diversity) following invasion are offset by biotic homogenization in a biodiversity hotspot. Biology Letters, 15(5), 20190133. <a href="https://doi.org/10.1098/rsbl.2019.0133">https://doi.org/10.1098/rsbl.2019.0133</a>¬†<a href="#fnref:7" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:8" role="doc-endnote">
      <p>P√©rez-Palma, E., Gramm, M., N√ºrnberg, P., May, P., &amp; Lal, D. (2019). Simple ClinVar: an interactive web server to explore and retrieve gene and disease variants aggregated in ClinVar database. Nucleic Acids Research. <a href="https://doi.org/10.1093/nar/gkz411">https://doi.org/10.1093/nar/gkz411</a>¬†<a href="#fnref:8" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:9" role="doc-endnote">
      <p>Green, E. P., Whitcomb, A., Kahumbura, C., Rosen, J. G., Goyal, S., Achieng, D., &amp; Bellows, B. (2019). What is the best method of family planning for me?: a text mining analysis of messages between users and agents of a digital health service in Kenya. Gates Open Research, 3, 1475. <a href="https://doi.org/10.12688/gatesopenres.12999.1">https://doi.org/10.12688/gatesopenres.12999.1</a>¬†<a href="#fnref:9" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:10" role="doc-endnote">
      <p>Mihalitsis, M., &amp; Bellwood, D. R. (2019). Morphological and functional diversity of piscivorous fishes on coral reefs. Coral Reefs. <a href="https://doi.org/10.1007/s00338-019-01820-w">https://doi.org/10.1007/s00338-019-01820-w</a>¬†<a href="#fnref:10" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:11" role="doc-endnote">
      <p>Kannenberg, S. A., Novick, K. A., Alexander, M. R., Maxwell, J. T., Moore, D. J. P., Phillips, R. P., &amp; Anderegg, W. R. L. (2019). Linking drought legacy effects across scales: From leaves to tree rings to ecosystems. Global Change Biology. <a href="https://doi.org/10.1111/gcb.14710">https://doi.org/10.1111/gcb.14710</a>¬†<a href="#fnref:11" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:12" role="doc-endnote">
      <p>Aspillaga, E., Safi, K., Hereu, B., &amp; Bartumeus, F. (2019). Modelling the three‚Äêdimensional space use of aquatic animals combining topography and Eulerian telemetry data. Methods in Ecology and Evolution. <a href="https://doi.org/10.1111/2041-210x.13232">https://doi.org/10.1111/2041-210x.13232</a>¬†<a href="#fnref:12" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:13" role="doc-endnote">
      <p>Gibson, K. M., Nguyen, B. N., Neumann, L. M., Miller, M., Buss, P., Daniels, S., ‚Ä¶ Pukazhenthi, B. (2019). Gut microbiome differences between wild and captive black rhinoceros ‚Äì implications for rhino health. Scientific Reports, 9(1). <a href="https://doi.org/10.1038/s41598-019-43875-3">https://doi.org/10.1038/s41598-019-43875-3</a>¬†<a href="#fnref:13" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:14" role="doc-endnote">
      <p>Sinval, J., Sirgy, M. J., Lee, D.-J., &amp; Mar√¥co, J. (2019). The Quality of Work Life Scale: Validity Evidence from Brazil and Portugal. Applied Research in Quality of Life. <a href="https://doi.org/10.1007/s11482-019-09730-3">https://doi.org/10.1007/s11482-019-09730-3</a>¬†<a href="#fnref:14" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:15" role="doc-endnote">
      <p>Iserman, M., Nalabandian, T., &amp; Ireland, M. (2019, June). Dictionaries and Decision Trees for the 2019 CLPsych Shared Task. In Proceedings of the Sixth Workshop on Computational Linguistics and Clinical Psychology (pp. 188-194). <a href="https://www.aclweb.org/anthology/W19-3025">https://www.aclweb.org/anthology/W19-3025</a>¬†<a href="#fnref:15" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:16" role="doc-endnote">
      <p>Goodman, M. C., Hannah, S. M., &amp; Ruttenberg, B. I. (2019). The relationship between geographic range extent, sea surface temperature and adult traits in coastal temperate fishes. Journal of Biogeography. <a href="https://doi.org/10.1111/jbi.13595">https://doi.org/10.1111/jbi.13595</a>¬†<a href="#fnref:16" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:17" role="doc-endnote">
      <p>Fioravanti, G., Piervitali, E., &amp; Desiato, F. (2019). A new homogenized daily data set for temperature variability assessment in Italy. International Journal of Climatology. <a href="https://doi.org/10.1002/joc.6177">https://doi.org/10.1002/joc.6177</a>¬†<a href="#fnref:17" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:18" role="doc-endnote">
      <p>Logemann, A., Schafberg, M., &amp; Brockmeyer, B. (2019). Using the HPTLC-bioluminescence bacteria assay for the determination of acute toxicities in marine sediments and its eligibility as a monitoring assessment tool. Chemosphere. <a href="https://doi.org/10.1016/j.chemosphere.2019.05.246">https://doi.org/10.1016/j.chemosphere.2019.05.246</a>¬†<a href="#fnref:18" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:19" role="doc-endnote">
      <p>Al-Mustanjid, A. (2019). Design of a common pathway drug for all types of cardiovascular diseases: A network biology approach. Network Biology, 9(2), 28. <a href="http://www.iaees.org/publications/journals/nb/articles/2019-9(2)/design-of-a-common-pathway-drug-for-cardiovascular-diseases.pdf">http://www.iaees.org/publications/journals/nb/articles/2019-9(2)/design-of-a-common-pathway-drug-for-cardiovascular-diseases.pdf</a>¬†<a href="#fnref:19" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:20" role="doc-endnote">
      <p>Wambui, K. M., &amp; Musenge, E. (2019). A space-time analysis of recurrent malnutrition-related hospitalisations in Kilifi, Kenya for children under-5 years. BMC Nutrition, 5(1). <a href="https://doi.org/10.1186/s40795-019-0296-5">https://doi.org/10.1186/s40795-019-0296-5</a>¬†<a href="#fnref:20" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
  </ol>
</div>

  </article>

</div>

      </div>
    </div>

    <footer class="site-footer">

  <div class="wrapper">

    <h2 class="footer-heading">rOpenSci Newsletter</h2>

    <div class="footer-col-wrapper">
      <div class="footer-col  footer-col-1">
        <ul class="contact-list">
          <!-- <li>rOpenSci Newsletter</li> -->
          <li><a href="https://ropensci.org/">rOpenSci.org</a></li>
          <li><a href="mailto:info@ropensci.org">info@ropensci.org</a></li>
        </ul>
      </div>

      <div class="footer-col  footer-col-2">
        <ul class="social-media-list">
          
          <li>
            <a href="https://github.com/ropensci">
              <span class="icon  icon--github">
                <svg viewBox="0 0 16 16">
                  <path fill="#828282" d="M7.999,0.431c-4.285,0-7.76,3.474-7.76,7.761 c0,3.428,2.223,6.337,5.307,7.363c0.388,0.071,0.53-0.168,0.53-0.374c0-0.184-0.007-0.672-0.01-1.32 c-2.159,0.469-2.614-1.04-2.614-1.04c-0.353-0.896-0.862-1.135-0.862-1.135c-0.705-0.481,0.053-0.472,0.053-0.472 c0.779,0.055,1.189,0.8,1.189,0.8c0.692,1.186,1.816,0.843,2.258,0.645c0.071-0.502,0.271-0.843,0.493-1.037 C4.86,11.425,3.049,10.76,3.049,7.786c0-0.847,0.302-1.54,0.799-2.082C3.768,5.507,3.501,4.718,3.924,3.65 c0,0,0.652-0.209,2.134,0.796C6.677,4.273,7.34,4.187,8,4.184c0.659,0.003,1.323,0.089,1.943,0.261 c1.482-1.004,2.132-0.796,2.132-0.796c0.423,1.068,0.157,1.857,0.077,2.054c0.497,0.542,0.798,1.235,0.798,2.082 c0,2.981-1.814,3.637-3.543,3.829c0.279,0.24,0.527,0.713,0.527,1.437c0,1.037-0.01,1.874-0.01,2.129 c0,0.208,0.14,0.449,0.534,0.373c3.081-1.028,5.302-3.935,5.302-7.362C15.76,3.906,12.285,0.431,7.999,0.431z"/>
                </svg>
              </span>

              <span class="username">ropensci</span>
            </a>
          </li>
          

          
          <li>
            <a href="https://twitter.com/ropensci">
              <span class="icon  icon--twitter">
                <svg viewBox="0 0 16 16">
                  <path fill="#828282" d="M15.969,3.058c-0.586,0.26-1.217,0.436-1.878,0.515c0.675-0.405,1.194-1.045,1.438-1.809
                  c-0.632,0.375-1.332,0.647-2.076,0.793c-0.596-0.636-1.446-1.033-2.387-1.033c-1.806,0-3.27,1.464-3.27,3.27 c0,0.256,0.029,0.506,0.085,0.745C5.163,5.404,2.753,4.102,1.14,2.124C0.859,2.607,0.698,3.168,0.698,3.767 c0,1.134,0.577,2.135,1.455,2.722C1.616,6.472,1.112,6.325,0.671,6.08c0,0.014,0,0.027,0,0.041c0,1.584,1.127,2.906,2.623,3.206 C3.02,9.402,2.731,9.442,2.433,9.442c-0.211,0-0.416-0.021-0.615-0.059c0.416,1.299,1.624,2.245,3.055,2.271 c-1.119,0.877-2.529,1.4-4.061,1.4c-0.264,0-0.524-0.015-0.78-0.046c1.447,0.928,3.166,1.469,5.013,1.469 c6.015,0,9.304-4.983,9.304-9.304c0-0.142-0.003-0.283-0.009-0.423C14.976,4.29,15.531,3.714,15.969,3.058z"/>
                </svg>
              </span>

              <span class="username">ropensci</span>
            </a>
          </li>
          
        </ul>
      </div>

      <div class="footer-col  footer-col-3">
        <p class="text">rOpenSci Newsletter, including software, software reviews, what's happening in the community, events, and more.</p>
      </div>
    </div>

  </div>

<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-63168295-2', 'auto');
  ga('send', 'pageview');

</script>

</footer>


  </body>

</html>
