<!DOCTYPE html>
<html>

  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <title>Update 2018-03-26</title>
    <meta name="description" content="rOpenSci Newsletter, including software, software reviews, what's happening in the community, events, and more.">
     <meta property="og:title" content="Update 2018-03-26">
    <meta property="og:type" content="article">
    <meta property="description" content="rOpenSci Newsletter, including software, software reviews, what's happening in the community, events, and more.">
    <meta property="og:description" content="rOpenSci Newsletter, including software, software reviews, what's happening in the community, events, and more.">
   <meta name="twitter:card" content="summary">
   <meta name="twitter:image" content="http://ropensci.org/android-chrome-512x512.png" >


<meta name="twitter:creator" content="@rOpenSci">
<meta name="twitter:site" content="@rOpenSci">

    <link rel="stylesheet" href="/css/main.css">

	<link rel="stylesheet" href="/css/bootstrap/css/bootstrap.css">

    <link rel="canonical" href="http://localhost:4000/update-2018-03-26/">

    <link rel="shortcut icon" href="../assets/img/favicon-16x16.png">

    <link rel="icon" type="image/png" sizes="32x32" href="/assets/img/favicon-32x32.png">
    <link rel="icon" type="image/png" sizes="16x16" href="/assets/img/favicon-16x16.png">

    <link rel="alternate" title="rOpenSci Newsletter" type="application/json" href="http://localhost:4000/feed.json" />

    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.9/css/all.css" integrity="sha384-5SOiIsAziJl6AWe0HWRKTXlfcSHKmYV4RBF18PPJ173Kzn7jzMyFuTtk8JA7QQG1" crossorigin="anonymous">
</head>



  <body>

    <header class="site-header">

  <div class="wrapper">

    <a class="site-title" href="/">rOpenSci Newsletter</a>

    <nav class="site-nav">
      <a href="#" class="menu-icon">
        <svg viewBox="0 0 18 15">
          <path fill="#424242" d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.031C17.335,0,18,0.665,18,1.484L18,1.484z"/>
          <path fill="#424242" d="M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0c0-0.82,0.665-1.484,1.484-1.484 h15.031C17.335,6.031,18,6.696,18,7.516L18,7.516z"/>
          <path fill="#424242" d="M18,13.516C18,14.335,17.335,15,16.516,15H1.484C0.665,15,0,14.335,0,13.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.031C17.335,12.031,18,12.696,18,13.516L18,13.516z"/>
        </svg>
      </a>

      <div class="trigger">
        
          
        
          
        
          
        
          
        
      </div>
    </nav>

  </div>

</header>


    <div class="page-content">
      <div class="wrapper">
        <div class="post">

  <header class="post-header">
  	<center><img src="../assets/img/icon_lettering_color.svg" width="375"></center>
    <!-- <h1 class="post-title">Update 2018-03-26</h1> -->
    <center><h1 style="color:grey">Mar 26, 2018</h1></center>
  </header>
  <br>
  <article class="post-content">
    <h2 id="software-">Software ðŸ“¦</h2>

<h3 id="releases">Releases</h3>

<ul>
  <li>A new version (<code class="language-plaintext highlighter-rouge">v0.1.4</code>) of <code class="language-plaintext highlighter-rouge">bomrang</code> is on <a href="https://cran.rstudio.com/web/packages/bomrang">CRAN</a>. See the <a href="https://github.com/ropensci/bomrang/releases/tag/v0.1.4">release notes</a> for details. Check out the <a href="https://ropensci.github.io/bomrang/">bomrang docs</a> to get started. <a href="https://github.com/ropensci/bomrang">Repository on GitHub</a></li>
  <li>A new version (<code class="language-plaintext highlighter-rouge">v0.2.14</code>) of <code class="language-plaintext highlighter-rouge">rglobi</code> is on <a href="https://cran.rstudio.com/web/packages/rglobi">CRAN</a>. See the <a href="https://cran.r-project.org/web/packages/rglobi/NEWS">NEWS</a> for details. Check out the <a href="https://cran.r-project.org/web/packages/rglobi/vignettes/rglobi_vignette.html">rglobi vignette</a> to get started. <a href="https://github.com/ropensci/rglobi">Repository on GitHub</a></li>
  <li>A new version (<code class="language-plaintext highlighter-rouge">v5.1.0</code>) of <code class="language-plaintext highlighter-rouge">drake</code> is on <a href="https://cran.rstudio.com/web/packages/drake">CRAN</a>. See the <a href="https://github.com/ropensci/drake/releases/tag/v5.1.0">release notes</a> for details. Check out the <a href="https://ropensci.github.io/drake/">drake docs</a> to get started. <a href="https://github.com/ropensci/drake">Repository on GitHub</a></li>
  <li>A new version (<code class="language-plaintext highlighter-rouge">v0.4.0</code>) of <code class="language-plaintext highlighter-rouge">mapr</code> is on <a href="https://cran.rstudio.com/web/packages/mapr">CRAN</a>. See the <a href="https://github.com/ropensci/mapr/releases/tag/v0.4.0">release notes</a> for details. Check out the <a href="https://cran.r-project.org/web/packages/mapr/vignettes/mapr_vignette.html">mapr vignette</a> to get started. <a href="https://github.com/ropensci/mapr">Repository on GitHub</a></li>
  <li>A new version (<code class="language-plaintext highlighter-rouge">v0.2.0</code>) of <code class="language-plaintext highlighter-rouge">tokenizers</code> is on <a href="https://cran.rstudio.com/web/packages/tokenizers">CRAN</a>. See the <a href="https://github.com/ropensci/tokenizers/releases/tag/v0.2.0">release notes</a> for details. Check out the <a href="https://cran.r-project.org/web/packages/tokenizers/vignettes/">tokenizers vignettes</a> to get started. <a href="https://github.com/ropensci/tokenizers">Repository on GitHub</a></li>
  <li>A new version (<code class="language-plaintext highlighter-rouge">v0.1.5</code>) of <code class="language-plaintext highlighter-rouge">codemetar</code> is on <a href="https://cran.rstudio.com/web/packages/codemetar">CRAN</a>. See the <a href="https://cran.r-project.org/web/packages/codemetar/news.html">NEWS</a> for details. Check out the <a href="https://cran.r-project.org/web/packages/codemetar/vignettes/">codemetar vignettes</a> to get started. <a href="https://github.com/ropensci/codemetar">Repository on GitHub</a></li>
  <li>A new version (<code class="language-plaintext highlighter-rouge">v0.1.8</code>) of <code class="language-plaintext highlighter-rouge">geoops</code> is on <a href="https://cran.rstudio.com/web/packages/geoops">CRAN</a>. See the <a href="https://github.com/ropensci/geoops/releases/tag/v0.1.8">release notes</a> for details. Check out the <a href="https://cran.r-project.org/web/packages/geoops/vignettes/geoops_vignette.html">geoops vignette</a> to get started. <a href="https://github.com/ropensci/geoops">Repository on GitHub</a></li>
  <li>A new version (<code class="language-plaintext highlighter-rouge">v1.8</code>) of <code class="language-plaintext highlighter-rouge">magick</code> is on <a href="https://cran.rstudio.com/web/packages/magick">CRAN</a>. See the <a href="https://github.com/ropensci/magick/releases/tag/v1.8">release notes</a> for details. Check out the <a href="https://cran.r-project.org/web/packages/magick/vignettes/intro.html">magick vignette</a> to get started. <a href="https://github.com/ropensci/magick">Repository on GitHub</a></li>
  <li>A new version (<code class="language-plaintext highlighter-rouge">v0.1.1</code>) of <code class="language-plaintext highlighter-rouge">cRegulome</code> is on <a href="https://cran.rstudio.com/web/packages/cRegulome">CRAN</a>. See the <a href="https://cran.r-project.org/web/packages/cRegulome/news.html">NEWS</a> for details. Check out the <a href="https://cran.r-project.org/web/packages/cRegulome/vignettes/">cRegulome vignettes</a> to get started. <a href="https://github.com/ropensci/cRegulome">Repository on GitHub</a></li>
  <li>A new version (<code class="language-plaintext highlighter-rouge">v1.0.0</code>) of <code class="language-plaintext highlighter-rouge">ijtiff</code> is on <a href="https://cran.rstudio.com/web/packages/ijtiff">CRAN</a>. This versions marks a move to rOpenSci. Check out the <a href="https://cran.r-project.org/web/packages/ijtiff/vignettes/ijtiff.html">ijtiff vignette</a> to get started. <a href="https://github.com/ropensci/ijtiff">Repository on GitHub</a></li>
  <li>A new version (<code class="language-plaintext highlighter-rouge">v1.0.0</code>) of <code class="language-plaintext highlighter-rouge">hydroscoper</code> is on <a href="https://cran.rstudio.com/web/packages/hydroscoper">CRAN</a>. This versions marks a move to rOpenSci. Check out the <a href="https://ropensci.github.io/hydroscoper/">hydroscoper docs</a> to get started. <a href="https://github.com/ropensci/hydroscoper">Repository on GitHub</a></li>
  <li>A new version (<code class="language-plaintext highlighter-rouge">v0.2.1</code>) of <code class="language-plaintext highlighter-rouge">patentsview</code> is on <a href="https://cran.rstudio.com/web/packages/patentsview">CRAN</a>. See the <a href="https://github.com/ropensci/patentsview/releases/tag/v0.2.1">release notes</a> for details. Check out the <a href="https://ropensci.github.io/patentsview/index.html">patentsview docs</a> to get started. <a href="https://github.com/ropensci/patentsview">Repository on GitHub</a></li>
  <li>A new version (<code class="language-plaintext highlighter-rouge">v2.5.2</code>) of <code class="language-plaintext highlighter-rouge">FedData</code> is on <a href="https://cran.rstudio.com/web/packages/FedData">CRAN</a>. See the <a href="https://cran.r-project.org/web/packages/FedData/news.html">release notes</a> for details. Check out the <a href="https://ropensci.github.io/FedData/">FedData docs</a> to get started. <a href="https://github.com/ropensci/FedData">Repository on GitHub</a></li>
  <li>A new version (<code class="language-plaintext highlighter-rouge">v1.0.12</code>) of <code class="language-plaintext highlighter-rouge">lingtypology</code> is on <a href="https://cran.rstudio.com/web/packages/ijtiff">CRAN</a>. See the <a href="https://github.com/ropensci/lingtypology/releases/tag/v1.0.12">release notes</a> for details. Check out the <a href="https://ropensci.github.io/lingtypology/">lingtypology docs</a> to get started. <a href="https://github.com/ropensci/ijtiff">Repository on GitHub</a></li>
  <li>A new version (<code class="language-plaintext highlighter-rouge">v0.1.1</code>) of <code class="language-plaintext highlighter-rouge">bikedata</code> is on <a href="https://cran.rstudio.com/web/packages/bikedata">CRAN</a>. See the <a href="https://github.com/ropensci/bikedata/releases/tag/v0.1.1">release notes</a> for details. Check out the <a href="https://cran.r-project.org/web/packages/bikedata/vignettes/bikedata.html">bikedata vignette</a> to get started. <a href="https://github.com/ropensci/bikedata">Repository on GitHub</a></li>
  <li>A new version (<code class="language-plaintext highlighter-rouge">v0.5.2</code>) of <code class="language-plaintext highlighter-rouge">gender</code> is on <a href="https://cran.rstudio.com/web/packages/gender">CRAN</a>. See the <a href="https://github.com/ropensci/gender/releases/tag/v0.5.2">release notes</a> for details. Check out the <a href="https://cran.r-project.org/web/packages/gender/vignettes/predicting-gender.html">gender vignette</a> to get started. <a href="https://github.com/ropensci/gender">Repository on GitHub</a></li>
  <li>A new version (<code class="language-plaintext highlighter-rouge">v0.2.3</code>) of <code class="language-plaintext highlighter-rouge">stplanr</code> is on <a href="https://cran.rstudio.com/web/packages/stplanr">CRAN</a>. See the <a href="https://github.com/ropensci/stplanr/releases/tag/0.2.3">release notes</a> for details. Check out the <a href="https://ropensci.github.io/stplanr/">stplanr docs</a> to get started. <a href="https://github.com/ropensci/stplanr">Repository on GitHub</a></li>
  <li>A new version (<code class="language-plaintext highlighter-rouge">v1.2.1</code>) of <code class="language-plaintext highlighter-rouge">rentrez</code> is on <a href="https://cran.rstudio.com/web/packages/rentrez">CRAN</a>. See the <a href="https://github.com/ropensci/rentrez/releases/tag/v1.2.1">release notes</a> for details. Check out the <a href="https://cran.r-project.org/web/packages/rentrez/vignettes/rentrez_tutorial.html">rentrez vignette</a> and the newly published <a href="https://journal.r-project.org/archive/2017/RJ-2017-058/index.html">rentrez paper in the R Journal</a> to get started. <a href="https://github.com/ropensci/rentrez">Repository on GitHub</a></li>
  <li>A new version (<code class="language-plaintext highlighter-rouge">v0.2.0</code>) of <code class="language-plaintext highlighter-rouge">webmockr</code> is on <a href="https://cran.rstudio.com/web/packages/webmockr">CRAN</a>. See the <a href="https://github.com/ropensci/webmockr/releases/tag/v0.2.0">release notes</a> for details. Check out the <a href="https://github.com/ropensci/webmockr#webmockr">webmockr README</a> to get started. <a href="https://github.com/ropensci/webmockr">Repository on GitHub</a></li>
  <li>A new version (<code class="language-plaintext highlighter-rouge">v2.0</code>) of <code class="language-plaintext highlighter-rouge">tesseract</code> is on <a href="https://cran.rstudio.com/web/packages/tesseract">CRAN</a>. See the <a href="https://github.com/ropensci/tesseract/releases/tag/v2.0">release notes</a> for details. Check out the <a href="https://cran.r-project.org/web/packages/tesseract/vignettes/intro.html">tesseract vignette</a> to get started. <a href="https://github.com/ropensci/tesseract">Repository on GitHub</a></li>
  <li>Two new versions (<code class="language-plaintext highlighter-rouge">v0.9.2</code>/<code class="language-plaintext highlighter-rouge">v0.9.3</code>) of <code class="language-plaintext highlighter-rouge">taxize</code> have been released to <a href="https://cran.rstudio.com/web/packages/taxize">CRAN</a> over the last month. See the release notes for <a href="https://github.com/ropensci/taxize/releases/tag/v0.9.2">v0.9.2</a> and <a href="https://github.com/ropensci/taxize/releases/tag/v0.9.3">v0.9.3</a> for details. Check out the <a href="https://taxize.dev/">taxize book</a> to get started. <a href="https://github.com/ropensci/taxize">Repository on GitHub</a></li>
  <li>A new version (<code class="language-plaintext highlighter-rouge">v0.2.0</code>) of <code class="language-plaintext highlighter-rouge">fauxpas</code> is on <a href="https://cran.rstudio.com/web/packages/fauxpas">CRAN</a>. See the <a href="https://github.com/ropensci/fauxpas/releases/tag/v0.2.0">release notes</a> for details. Check out the <a href="https://cran.r-project.org/web/packages/fauxpas/vignettes/fauxpas-vignette.html">fauxpas vignette</a> to get started. <a href="https://github.com/ropenscilabs/fauxpas">Repository on GitHub</a></li>
  <li>A new version (<code class="language-plaintext highlighter-rouge">v0.2.6</code>) of <code class="language-plaintext highlighter-rouge">ropenaq</code> is on <a href="https://cran.rstudio.com/web/packages/ropenaq">CRAN</a>. See the <a href="https://cran.r-project.org/web/packages/ropenaq/news.html">NEWS</a> for details. Check out the <a href="https://ropensci.github.io/ropenaq/">ropenaq docs</a> to get started. <a href="https://github.com/ropensci/ropenaq">Repository on GitHub</a></li>
  <li>A new version (<code class="language-plaintext highlighter-rouge">v0.3.3</code>) of <code class="language-plaintext highlighter-rouge">tidyhydat</code> is on <a href="https://cran.rstudio.com/web/packages/tidyhydat">CRAN</a>. See the <a href="https://github.com/ropensci/tidyhydat/releases/tag/v0.3.3">release notes</a> for details. Check out the <a href="https://cran.r-project.org/web/packages/tidyhydat/vignettes/">tidyhydat vignettes</a> to get started. <a href="https://github.com/ropensci/tidyhydat">Repository on GitHub</a></li>
</ul>

<p><br /><br /></p>

<h2 id="software-review-">Software Review âœ”</h2>

<p>We accept community contributed packages via our onboarding system - an open software review system, sorta like scholarly paper review, but way better. Weâ€™ll highlight newly onboarded packages here. A huge thanks to our reviewers, who do a lot of work reviewing (see the <a href="https://ropensci.org/blog/2016/03/28/software-review">blog post on our review system</a>),
and the authors of the packages!</p>

<p>If you want to be a reviewer fill out <a href="https://ropensci.org/onboarding/">this short form</a>, and weâ€™ll ping you when thereâ€™s a submission that fits in your area of expertise.</p>

<p>The following four packages recently went through our onboarding process and has been approved!</p>

<ul>
  <li><a href="https://github.com/ropensci/essurvey">essurvey</a> &gt; Download Data from the European Social Survey on the Fly
    <ul>
      <li>Author: <a href="https://github.com/cimentadaj">Jorge Cimentada</a></li>
      <li>Issue: <a href="https://github.com/ropensci/onboarding/issues/201">ropensci/onboarding#201</a></li>
      <li>Reviewers:
        <ul>
          <li><a href="https://github.com/leeper">Thomas Leeper</a></li>
          <li><a href="https://github.com/nujcharee">Nujcharee</a></li>
        </ul>
      </li>
    </ul>
  </li>
  <li><a href="https://github.com/ropensci/rtika">rtika</a> &gt; R Interface to Apache Tika
    <ul>
      <li>Author: <a href="https://github.com/predict-r">Sasha Goodman</a></li>
      <li>Issue: <a href="https://github.com/ropensci/onboarding/issues/191">ropensci/onboarding#191</a></li>
      <li>Reviewers:
        <ul>
          <li><a href="https://github.com/juliasilge">Julia Silge</a></li>
          <li><a href="https://github.com/davidgohel">David Gohel</a></li>
        </ul>
      </li>
    </ul>
  </li>
  <li><a href="https://github.com/ropensci/nlmr">nlmr</a> &gt; Simulating Neutral Landscape Models
    <ul>
      <li>Author: <a href="https://github.com/marcosci">Marco Sciaini</a></li>
      <li>Issue: <a href="https://github.com/ropensci/onboarding/issues/188">ropensci/onboarding#188</a></li>
      <li>Reviewers:
        <ul>
          <li><a href="https://github.com/laurajanegraham">Laura Jana Graham</a></li>
          <li><a href="https://github.com/jhollist">Jeff Hollister</a></li>
        </ul>
      </li>
    </ul>
  </li>
  <li><a href="https://github.com/ropensci/hydroscoper">hydroscoper</a> &gt; Interface to Hydroscope
    <ul>
      <li>Author: <a href="https://github.com/cimentadaj">Jorge Cimentada</a></li>
      <li>Issue: <a href="https://github.com/ropensci/onboarding/issues/185">ropensci/onboarding#185</a></li>
      <li>Reviewers:
        <ul>
          <li><a href="https://github.com/timtrice">Tim Trice</a></li>
          <li><a href="https://github.com/sharlagelfand">Sharla Gelfand</a></li>
        </ul>
      </li>
    </ul>
  </li>
</ul>

<p>The following two packages were recently submitted:</p>

<ul>
  <li><a href="https://github.com/rubenarslan/codebook">codebook</a> &gt; Automatic Codebooks from Survey Metadata Encoded in Attributes
    <ul>
      <li>Author: <a href="https://github.com/rubenarslan">Ruben Arslan</a></li>
      <li>Issue: <a href="https://github.com/ropensci/onboarding/issues/206">ropensci/onboarding#206</a></li>
      <li>Note: still going through editor check</li>
    </ul>
  </li>
  <li><a href="https://github.com/azizka/CoordinateCleaner">CoordinateCleaner</a> &gt; Automated Cleaning of Occurrence Records from Biological Collections
    <ul>
      <li>Author: <a href="https://github.com/azizka">Alexander Zizka</a></li>
      <li>Issue: <a href="https://github.com/ropensci/onboarding/issues/199">ropensci/onboarding#199</a></li>
      <li>Note: still going through editor check</li>
    </ul>
  </li>
</ul>

<p><br /><br /></p>

<h2 id="on-the-blog">On the blog</h2>

<h3 id="main-blog">main blog</h3>

<p><a href="http://steffilazerte.ca/">Steffi LaZerte</a> wrote about her package <code class="language-plaintext highlighter-rouge">weathercan</code> recently onboarded with rOpenSci: <a href="https://ropensci.org/blog/2018/03/06/weathercan/">Integrating data from weathercan</a></p>

<hr />

<p><a href="https://github.com/softloud">Charles Gray</a> wrote about her experience in reviewing her first R package: <a href="https://ropensci.org/blog/2018/03/13/ode-to-testing/">An Ode to Testing, my first review</a>.</p>

<hr />

<p>The rOpenSci Editors wrote a post <a href="https://ropensci.org/blog/2018/03/16/thanking-reviewers-in-metadata/">Thanking Your Reviewers: Gratitude through Semantic Metadata</a> announcing that we can now include reviewers in the Authors section of an R packageâ€™s DESCRIPTION file. Yay!</p>

<hr />

<p><a href="https://david-winter.info/">David Winter</a> wrote a post (<a href="https://ropensci.org/blog/2018/03/20/rentrez-paper/">A rentrez paper, and how to use the NCBIâ€™s new API keys</a>) about his package <a href="https://github.com/ropensci/rentrez">rentrez</a> for interacting with <a href="https://www.ncbi.nlm.nih.gov/">NCBI</a> data. He discussed a recently published <a href="https://journal.r-project.org/archive/2017/RJ-2017-058/index.html">paper</a> on <code class="language-plaintext highlighter-rouge">rentrez</code> and support for API keys for NCBI.</p>

<p><br /><br /></p>

<h2 id="use-cases">Use cases</h2>

<p>If youâ€™ve used rOpenSci software in a blog post or a paper, tell us on the <a href="https://discuss.ropensci.org/t/share-ropensci-package-citations-plz/515/11">discussion forum</a> and weâ€™ll share it with our community here.</p>

<p>The following 15 works use/cite rOpenSci software:</p>

<ul>
  <li>Borcard <em>et al</em>. used the rOpenSci package <a href="https://github.com/ropensci/taxize">taxize</a> in their book chapter <a href="https://doi.org/10.1007/978-3-319-71404-2_8">Community Diversity</a> from the book <em>Numerical Ecology with R</em> <sup id="fnref:1" role="doc-noteref"><a href="#fn:1" class="footnote">1</a></sup></li>
  <li>Cho &amp; Yu used the rOpenSci package <a href="https://github.com/ropensci/rcrossref">rcrossref</a> in their pre-print <a href="https://arxiv.org/pdf/1803.06249.pdf">Link prediction for interdisciplinary collaboration via co-authorship network</a> <sup id="fnref:2" role="doc-noteref"><a href="#fn:2" class="footnote">2</a></sup></li>
  <li>Nabout <em>et al</em>. used the rOpenSci package <a href="https://github.com/ropensci/rAltmetric">rAltmetric</a> in their paper <a href="https://doi.org/10.1007/s11192-018-2678-x">Do traditional scientometric indicators predict social media activity on scientific knowledge? An analysis of the ecological literature</a> <sup id="fnref:3" role="doc-noteref"><a href="#fn:3" class="footnote">3</a></sup></li>
  <li>Sanyal <em>et al</em>. used the rOpenSci package <a href="https://github.com/ropensci/rgbif">rgbif</a> in their paper <a href="https://doi.org/10.1002/ajb2.1014">Intraspecific and interspecific adaptive latitudinal cline in Brassicaceae seed oil traits</a> <sup id="fnref:4" role="doc-noteref"><a href="#fn:4" class="footnote">4</a></sup></li>
  <li>Lakiotaki <em>et al</em>. used the rOpenSci package <a href="https://github.com/ropensci/rentrez">rentrez</a> in their paper <a href="https://doi.org/10.1093/database/bay011">BioDataome: a collection of uniformly preprocessed and automatically annotated datasets for data-driven biology</a> <sup id="fnref:5" role="doc-noteref"><a href="#fn:5" class="footnote">5</a></sup></li>
  <li>Vieilledent <em>et al</em>. used the rOpenSci package <a href="https://github.com/ropensci/taxize">taxize</a> in their pre-print <a href="https://doi.org/10.1101/274068">New formula and conversion factor to compute tree species basic wood density from a global wood technology database</a> <sup id="fnref:6" role="doc-noteref"><a href="#fn:6" class="footnote">6</a></sup></li>
  <li>Foster, Chamberlain, &amp; GrÃ¼nwald wrote a paper describing the rOpenSci <a href="https://github.com/ropensci/taxa">taxa</a> package: <a href="https://doi.org/10.12688/f1000research.14013.1">taxa: An R package implementing data standards and methods for taxonomic data</a> <sup id="fnref:7" role="doc-noteref"><a href="#fn:7" class="footnote">7</a></sup></li>
  <li>Farquharson <em>et al</em>. used the package <a href="https://github.com/ropensci/rotl">rotl</a> in their paper <a href="https://doi.org/10.1038/s41467-018-03500-9">A meta-analysis of birth-origin effects on reproduction in diverse captive environments</a> <sup id="fnref:8" role="doc-noteref"><a href="#fn:8" class="footnote">8</a></sup></li>
  <li>Bemmels <em>et al</em>. used the packages <a href="https://github.com/ropensci/rgbif">rgbif</a> and <a href="https://github.com/ropensci/brranching">brranching</a> in their paper <a href="https://doi.org/10.1111/ecog.03473">Filter-dispersal assembly of lowland Neotropical rainforests across the Andes</a>&gt; <sup id="fnref:9" role="doc-noteref"><a href="#fn:9" class="footnote">9</a></sup></li>
  <li>Ng <em>et al</em>. mentioned our package <a href="https://github.com/ropensci/biomartr">biomartr</a> in their paper <a href="https://doi.org/10.1016/j.ccell.2018.01.021">Systematic Functional Annotation of Somatic Mutations in Cancer</a> <sup id="fnref:10" role="doc-noteref"><a href="#fn:10" class="footnote">10</a></sup></li>
  <li>Bennett <em>et al</em>. used <a href="https://github.com/ropensci/taxize">taxize</a> in their paper <a href="https://doi.org/10.1038/sdata.2018.22">GlobTherm, a global database on thermal tolerances for aquatic and terrestrial organisms</a> <sup id="fnref:11" role="doc-noteref"><a href="#fn:11" class="footnote">11</a></sup></li>
  <li>Correia <em>et al</em>. used <a href="https://github.com/ropensci/taxize">taxize</a> in their paper <a href="https://doi.org/10.1016/j.ecolind.2018.02.059">Nomenclature instability in species culturomic assessments: Why synonyms matter</a> <sup id="fnref:12" role="doc-noteref"><a href="#fn:12" class="footnote">12</a></sup></li>
  <li>Reibe <em>et al</em>. used <a href="https://github.com/ropensci/rentrez">rentrez</a> in their paper <a href="https://www.physiology.org/doi/pdf/10.1152/physiolgenomics.00127.2017">GeneXX: An online tool for the exploration of transcript changes in skeletal muscle associated with exercise</a> <sup id="fnref:13" role="doc-noteref"><a href="#fn:13" class="footnote">13</a></sup></li>
  <li>Sinval <em>et al</em>. used <a href="https://github.com/ropensci/skimr">skimr</a> in their paper <a href="https://doi.org/10.3389/fpsyg.2018.00353">Brazil-Portugal Transcultural Adaptation of the UWES-9: Internal Consistency, Dimensionality, and Measurement Invariance</a> <sup id="fnref:14" role="doc-noteref"><a href="#fn:14" class="footnote">14</a></sup></li>
  <li>Bergmann <em>et al</em>. used <a href="https://github.com/ropensci/webchem">webchem</a> in their paper <a href="https://link.springer.com/article/10.1007/s00216-018-0997-7">Development of quantitative screen for 1550 chemicals with GC-MS</a> <sup id="fnref:15" role="doc-noteref"><a href="#fn:15" class="footnote">15</a></sup></li>
</ul>

<p><br /><br /></p>

<h2 id="in-the-news">In the news</h2>

<p><a href="https://twitter.com/microbeEcology">David Clark</a> shared a screenshot of packages he was using for a project, including rOpenSciâ€™s <a href="https://github.com/ropensci/geojsonio">geojsonio</a> package to do GeoJSON I/O (input/output):</p>
<blockquote class="twitter-tweet" data-cards="hidden" data-lang="en"><p lang="en" dir="ltr">Sometimes you just need all the packages... <a href="https://twitter.com/hashtag/rstats?src=hash&amp;ref_src=twsrc%5Etfw">#rstats</a> <a href="https://t.co/cl8lK1ESbe">pic.twitter.com/cl8lK1ESbe</a></p>&mdash; David Clark (@microbeEcology) <a href="https://twitter.com/microbeEcology/status/974276898040754176?ref_src=twsrc%5Etfw">March 15, 2018</a></blockquote>
<script async="" src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>

<p><br /></p>

<p><a href="https://twitter.com/CieraReports">Ciera Martinez</a> and Caryn Johansen describe in a <a href="https://cabinetofcuriosity.github.io/cabinetofcuriosity_site/retrieving-summary-neotoma/">blog post</a> how to use our <a href="https://github.com/ropensci/neotoma">neotoma</a> package to access and explore mammal bone records</p>
<blockquote class="twitter-tweet" data-cards="hidden" data-lang="en"><p lang="en" dir="ltr">Our first post is Part 1 of a short series where we access and explore the mammal bone records in <a href="https://twitter.com/neotomadb?ref_src=twsrc%5Etfw">@neotomadb</a> with the use of the neotoma package from <a href="https://twitter.com/rOpenSci?ref_src=twsrc%5Etfw">@rOpenSci</a>. ðŸ’€ðŸ’€ðŸ’€<br /><br />site: <a href="https://t.co/BShEDADMWK">https://t.co/BShEDADMWK</a> <a href="https://t.co/Kk8g1pCMIV">pic.twitter.com/Kk8g1pCMIV</a></p>&mdash; Ciera Martinez (@CieraReports) <a href="https://twitter.com/CieraReports/status/976522640419561472?ref_src=twsrc%5Etfw">March 21, 2018</a></blockquote>

<p><br /><br /></p>

<hr style="display: block; height: 1px; border: 0; border-top: 3px solid #7CCCC8; margin: 1em 0; padding: 0; " />

<p><br /><br /></p>

<h3 id="keep-up-with-ropensci">Keep up with rOpenSci</h3>

<ul>
  <li>Mailing list: Sign up with an email address to get this newsletter sent to your inbox -&gt; <a href="https://ropensci.org/#subscribe">ropensci.org/#subscribe</a></li>
  <li>Alternatively, you can subscribe to this newsletter via our XML feed at <a href="https://news.ropensci.org/feed.xml">https://news.ropensci.org/feed.xml</a> or our JSON feed at <a href="https://news.ropensci.org//feed.json">https://news.ropensci.org//feed.json</a></li>
  <li>rOpenSci on Twitter: <a href="https://twitter.com/ropensci">@ropensci</a></li>
  <li>The rOpenSci blog at <a href="https://ropensci.org/blog">ropensci.org/blog</a> - you can subscribe in any RSS aggregator, or manually via <a href="https://ropensci.org/feed.xml">https://ropensci.org/feed.xml</a>. We also announce new blog posts on our Twitter account.</li>
</ul>

<p><br /></p>

<h4 id="footnotes">Footnotes</h4>

<div class="footnotes" role="doc-endnotes">
  <ol>
    <li id="fn:1" role="doc-endnote">
      <p>Borcard D., Gillet F., Legendre P. (2018) Community Diversity. In: Numerical Ecology with R. Use R! Springer, Cham <a href="https://doi.org/10.1007/978-3-319-71404-2_8">https://doi.org/10.1007/978-3-319-71404-2_8</a>Â <a href="#fnref:1" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:2" role="doc-endnote">
      <p>Cho, H., &amp; Yu, Y. (2018). Link prediction for interdisciplinary collaboration via co-authorship network. arXiv preprint arXiv:1803.06249. <a href="https://arxiv.org/pdf/1803.06249.pdf">https://arxiv.org/pdf/1803.06249.pdf</a>Â <a href="#fnref:2" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:3" role="doc-endnote">
      <p>Nabout, J. C., Teresa, F. B., Machado, K. B., do Prado, V. H. M., Bini, L. M., &amp; Diniz-Filho, J. A. F. (2018). Do traditional scientometric indicators predict social media activity on scientific knowledge? An analysis of the ecological literature. Scientometrics. <a href="https://doi.org/10.1007/s11192-018-2678-x">https://doi.org/10.1007/s11192-018-2678-x</a>Â <a href="#fnref:3" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:4" role="doc-endnote">
      <p>Sanyal, A., Lenoir, J., Oâ€™Neill, C., Dubois, F., &amp; Decocq, G. (2018). Intraspecific and interspecific adaptive latitudinal cline in Brassicaceae seed oil traits. American Journal of Botany, 105(1), 85â€“94. <a href="https://doi.org/10.1002/ajb2.1014">https://doi.org/10.1002/ajb2.1014</a>Â <a href="#fnref:4" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:5" role="doc-endnote">
      <p>Lakiotaki, K., Vorniotakis, N., Tsagris, M., Georgakopoulos, G., &amp; Tsamardinos, I. (2018). BioDataome: a collection of uniformly preprocessed and automatically annotated datasets for data-driven biology. Database, 2018. <a href="https://doi.org/10.1093/database/bay011">https://doi.org/10.1093/database/bay011</a>Â <a href="#fnref:5" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:6" role="doc-endnote">
      <p>Vieilledent, G., Fischer, F. J., Chave, J., Guibal, D., Langbour, P., &amp; GÃ©rard, J. (2018). New formula and conversion factor to compute tree species basic wood density from a global wood technology database. bioRxiv, 274068. <a href="https://doi.org/10.1101/274068">https://doi.org/10.1101/274068</a>Â <a href="#fnref:6" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:7" role="doc-endnote">
      <p>Foster, Z. S. L., Chamberlain, S., &amp; GrÃ¼nwald, N. J. (2018). Taxa: An R package implementing data standards and methods for taxonomic data. F1000Research, 7, 272. <a href="https://doi.org/10.12688/f1000research.14013.1">https://doi.org/10.12688/f1000research.14013.1</a>Â <a href="#fnref:7" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:8" role="doc-endnote">
      <p>Farquharson, K. A., Hogg, C. J., &amp; Grueber, C. E. (2018). A meta-analysis of birth-origin effects on reproduction in diverse captive environments. Nature Communications, 9(1). <a href="https://doi.org/10.1038/s41467-018-03500-9">https://doi.org/10.1038/s41467-018-03500-9</a>Â <a href="#fnref:8" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:9" role="doc-endnote">
      <p>Bemmels, J. B., Wright, S. J., Garwood, N. C., Queenborough, S. A., Valencia, R., &amp; Dick, C. W. (2018). Filter-dispersal assembly of lowland Neotropical rainforests across the Andes. Ecography. <a href="https://doi.org/10.1111/ecog.03473">https://doi.org/10.1111/ecog.03473</a>Â <a href="#fnref:9" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:10" role="doc-endnote">
      <p>Ng, P. K.-S., Li, J., Jeong, K. J., Shao, S., Chen, H., Tsang, Y. H., â€¦ Mills, G. B. (2018). Systematic Functional Annotation of Somatic Mutations in Cancer. Cancer Cell, 33(3), 450â€“462.e10. <a href="https://doi.org/10.1016/j.ccell.2018.01.021">https://doi.org/10.1016/j.ccell.2018.01.021</a>Â <a href="#fnref:10" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:11" role="doc-endnote">
      <p>Bennett, J. M., Calosi, P., Clusella-Trullas, S., MartÃ­nez, B., Sunday, J., Algar, A. C., â€¦ Morales-Castilla, I. (2018). GlobTherm, a global database on thermal tolerances for aquatic and terrestrial organisms. Scientific Data, 5, 180022. <a href="https://doi.org/10.1038/sdata.2018.22">https://doi.org/10.1038/sdata.2018.22</a>Â <a href="#fnref:11" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:12" role="doc-endnote">
      <p>Correia, R. A., JariÄ‡, I., Jepson, P., Malhado, A. C. M., Alves, J. A., &amp; Ladle, R. J. (2018). Nomenclature instability in species culturomic assessments: Why synonyms matter. Ecological Indicators, 90, 74â€“78. <a href="https://doi.org/10.1016/j.ecolind.2018.02.059">https://doi.org/10.1016/j.ecolind.2018.02.059</a>Â <a href="#fnref:12" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:13" role="doc-endnote">
      <p>Reibe, S., Hjorth, M., Febbraio, M. A., &amp; Whitham, M. (2018). GeneXX: An online tool for the exploration of transcript changes in skeletal muscle associated with exercise. Physiological genomics. <a href="https://doi.org/10.1152/physiolgenomics.00127.2017">https://doi.org/10.1152/physiolgenomics.00127.2017</a>Â <a href="#fnref:13" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:14" role="doc-endnote">
      <p>Sinval, J., Pasian, S., QueirÃ³s, C., &amp; MarÃ´co, J. (2018). Brazil-Portugal Transcultural Adaptation of the UWES-9: Internal Consistency, Dimensionality, and Measurement Invariance. Frontiers in Psychology, 9. <a href="https://doi.org/10.3389/fpsyg.2018.00353">https://doi.org/10.3389/fpsyg.2018.00353</a>Â <a href="#fnref:14" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
    <li id="fn:15" role="doc-endnote">
      <p>Bergmann, A. J., Scott, R. P., Wilson, G., &amp; Anderson, K. A. (2018). Development of quantitative screen for 1550 chemicals with GC-MS. Analytical and Bioanalytical Chemistry, 1-10. <a href="https://link.springer.com/article/10.1007/s00216-018-0997-7">https://link.springer.com/article/10.1007/s00216-018-0997-7</a>Â <a href="#fnref:15" class="reversefootnote" role="doc-backlink">&#8617;</a></p>
    </li>
  </ol>
</div>

  </article>

</div>

      </div>
    </div>

    <footer class="site-footer">

  <div class="wrapper">

    <h2 class="footer-heading">rOpenSci Newsletter</h2>

    <div class="footer-col-wrapper">
      <div class="footer-col  footer-col-1">
        <ul class="contact-list">
          <!-- <li>rOpenSci Newsletter</li> -->
          <li><a href="https://ropensci.org/">rOpenSci.org</a></li>
          <li><a href="mailto:info@ropensci.org">info@ropensci.org</a></li>
        </ul>
      </div>

      <div class="footer-col  footer-col-2">
        <ul class="social-media-list">
          
          <li>
            <a href="https://github.com/ropensci">
              <span class="icon  icon--github">
                <svg viewBox="0 0 16 16">
                  <path fill="#828282" d="M7.999,0.431c-4.285,0-7.76,3.474-7.76,7.761 c0,3.428,2.223,6.337,5.307,7.363c0.388,0.071,0.53-0.168,0.53-0.374c0-0.184-0.007-0.672-0.01-1.32 c-2.159,0.469-2.614-1.04-2.614-1.04c-0.353-0.896-0.862-1.135-0.862-1.135c-0.705-0.481,0.053-0.472,0.053-0.472 c0.779,0.055,1.189,0.8,1.189,0.8c0.692,1.186,1.816,0.843,2.258,0.645c0.071-0.502,0.271-0.843,0.493-1.037 C4.86,11.425,3.049,10.76,3.049,7.786c0-0.847,0.302-1.54,0.799-2.082C3.768,5.507,3.501,4.718,3.924,3.65 c0,0,0.652-0.209,2.134,0.796C6.677,4.273,7.34,4.187,8,4.184c0.659,0.003,1.323,0.089,1.943,0.261 c1.482-1.004,2.132-0.796,2.132-0.796c0.423,1.068,0.157,1.857,0.077,2.054c0.497,0.542,0.798,1.235,0.798,2.082 c0,2.981-1.814,3.637-3.543,3.829c0.279,0.24,0.527,0.713,0.527,1.437c0,1.037-0.01,1.874-0.01,2.129 c0,0.208,0.14,0.449,0.534,0.373c3.081-1.028,5.302-3.935,5.302-7.362C15.76,3.906,12.285,0.431,7.999,0.431z"/>
                </svg>
              </span>

              <span class="username">ropensci</span>
            </a>
          </li>
          

          
          <li>
            <a href="https://twitter.com/ropensci">
              <span class="icon  icon--twitter">
                <svg viewBox="0 0 16 16">
                  <path fill="#828282" d="M15.969,3.058c-0.586,0.26-1.217,0.436-1.878,0.515c0.675-0.405,1.194-1.045,1.438-1.809
                  c-0.632,0.375-1.332,0.647-2.076,0.793c-0.596-0.636-1.446-1.033-2.387-1.033c-1.806,0-3.27,1.464-3.27,3.27 c0,0.256,0.029,0.506,0.085,0.745C5.163,5.404,2.753,4.102,1.14,2.124C0.859,2.607,0.698,3.168,0.698,3.767 c0,1.134,0.577,2.135,1.455,2.722C1.616,6.472,1.112,6.325,0.671,6.08c0,0.014,0,0.027,0,0.041c0,1.584,1.127,2.906,2.623,3.206 C3.02,9.402,2.731,9.442,2.433,9.442c-0.211,0-0.416-0.021-0.615-0.059c0.416,1.299,1.624,2.245,3.055,2.271 c-1.119,0.877-2.529,1.4-4.061,1.4c-0.264,0-0.524-0.015-0.78-0.046c1.447,0.928,3.166,1.469,5.013,1.469 c6.015,0,9.304-4.983,9.304-9.304c0-0.142-0.003-0.283-0.009-0.423C14.976,4.29,15.531,3.714,15.969,3.058z"/>
                </svg>
              </span>

              <span class="username">ropensci</span>
            </a>
          </li>
          
        </ul>
      </div>

      <div class="footer-col  footer-col-3">
        <p class="text">rOpenSci Newsletter, including software, software reviews, what's happening in the community, events, and more.</p>
      </div>
    </div>

  </div>

<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-63168295-2', 'auto');
  ga('send', 'pageview');

</script>

</footer>


  </body>

</html>
