<!DOCTYPE html>
<html>

  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <title>2019-04-01</title>
    <meta name="description" content="rOpenSci Newsletter, including software, software reviews, what's happening in the community, events, and more.">
     <meta property="og:title" content="2019-04-01">
    <meta property="og:type" content="article">
    <meta property="description" content="rOpenSci Newsletter, including software, software reviews, what's happening in the community, events, and more.">
    <meta property="og:description" content="rOpenSci Newsletter, including software, software reviews, what's happening in the community, events, and more.">
   <meta name="twitter:card" content="summary">
   <meta name="twitter:image" content="http://ropensci.org/android-chrome-512x512.png" >


<meta name="twitter:creator" content="@rOpenSci">
<meta name="twitter:site" content="@rOpenSci">

    <link rel="stylesheet" href="/css/main.css">

	<link rel="stylesheet" href="/css/bootstrap/css/bootstrap.css">

    <link rel="canonical" href="http://localhost:4000/2019-04-01/">

    <link rel="shortcut icon" href="../assets/img/favicon-16x16.png">

    <link rel="icon" type="image/png" sizes="32x32" href="/assets/img/favicon-32x32.png">
    <link rel="icon" type="image/png" sizes="16x16" href="/assets/img/favicon-16x16.png">

    <link rel="alternate" title="rOpenSci Newsletter" type="application/json" href="http://localhost:4000/feed.json" />

    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.9/css/all.css" integrity="sha384-5SOiIsAziJl6AWe0HWRKTXlfcSHKmYV4RBF18PPJ173Kzn7jzMyFuTtk8JA7QQG1" crossorigin="anonymous">
</head>



  <body>

    <header class="site-header">

  <div class="wrapper">

    <a class="site-title" href="/">rOpenSci Newsletter</a>

    <nav class="site-nav">
      <a href="#" class="menu-icon">
        <svg viewBox="0 0 18 15">
          <path fill="#424242" d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.031C17.335,0,18,0.665,18,1.484L18,1.484z"/>
          <path fill="#424242" d="M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0c0-0.82,0.665-1.484,1.484-1.484 h15.031C17.335,6.031,18,6.696,18,7.516L18,7.516z"/>
          <path fill="#424242" d="M18,13.516C18,14.335,17.335,15,16.516,15H1.484C0.665,15,0,14.335,0,13.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.031C17.335,12.031,18,12.696,18,13.516L18,13.516z"/>
        </svg>
      </a>

      <div class="trigger">
        
          
          <a class="page-link" href="/about/">About</a>
          
        
          
        
          
        
          
        
          
        
      </div>
    </nav>

  </div>

</header>


    <div class="page-content">
      <div class="wrapper">
        <div class="post">

  <header class="post-header">
  	<center><img src="../assets/img/icon_lettering_color.svg" width="375"></center>
    <!-- <h1 class="post-title">2019-04-01</h1> -->
    <center><h1 style="color:grey">Apr 1, 2019</h1></center>
  </header>
  <br>
  <article class="post-content">
    <h2 id="ropensci-hq">rOpenSci HQ</h2>

<ul>
  <li>More than 50 people attended our Community Call on Research Applications of rOpenSci Taxonomy and Biodiversity Tools. Video, slides, and code are available: <a href="https://ropensci.org/commcalls/2019-03-27/">https://ropensci.org/commcalls/2019-03-27/</a>.  Thank you to our speakers: Kathryn Turner, Zachary Foster, Kari Norman, Margaret Siple, Scott Chamberlain and Ciera Martinez.</li>
  <li>We‚Äôre planning a Community Call on Security for R with Ildi Czeller &amp; Bob Rudis. Create Your Own Security Adventure by <a href="https://discuss.ropensci.org/t/community-call-security-for-r-create-your-own-security-adventure/1612">telling us what you want to know about the topic</a>.</li>
</ul>

<p><br /></p>

<h2 id="software-">Software üì¶</h2>

<p>CRAN: <img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px" />
GitHub: <img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px" /></p>

<h3 id="new-packages">New packages</h3>

<ul>
  <li>The first version (<code class="highlighter-rouge">v1.0.1</code>) of <code class="highlighter-rouge">cyphr</code> is on CRAN - high level encryption wrappers. See the <a href="https://github.com/ropensci/cyphr/releases/tag/v1.0.1">release notes</a> for changes. Checkout the <a href="https://cran.rstudio.com/web/packages/cyphr/vignettes/">vignettes</a> to get started. <a target="_blank" href="https://cran.rstudio.com/web/packages/cyphr"><img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a> <a target="_blank" href="https://github.com/ropensci/cyphr"><img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a></li>
  <li>The first version (<code class="highlighter-rouge">v0.1.0</code>) of <code class="highlighter-rouge">nlrx</code> is on CRAN - setup, run and analyze NetLogo model simulations. See the <a href="https://github.com/ropensci/nlrx/blob/master/NEWS.md">release notes</a> for changes. Checkout the <a href="https://cran.rstudio.com/web/packages/nlrx/vignettes/">vignettes</a> to get started. <a target="_blank" href="https://cran.rstudio.com/web/packages/nlrx"><img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a> <a target="_blank" href="https://github.com/ropensci/nlrx"><img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a></li>
</ul>

<h3 id="new-versions">New Versions</h3>

<ul>
  <li>A new version (<code class="highlighter-rouge">v0.15.7</code>) of <code class="highlighter-rouge">DataPackageR</code> is on CRAN - construct reproducible analytic data sets as R packages. See the <a href="https://github.com/ropensci/DataPackageR/releases/tag/v0.15.7">release notes</a> for changes. Checkout the <a href="https://cran.rstudio.com/web/packages/DataPackageR/vignettes/">vignettes</a> to get started. <a target="_blank" href="https://cran.rstudio.com/web/packages/DataPackageR"><img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a> <a target="_blank" href="https://github.com/ropensci/DataPackageR"><img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a>
    <blockquote>
      <p>fix test/vignette bugs related to upcoming version of <code class="highlighter-rouge">usethis</code></p>
    </blockquote>
  </li>
  <li>A new version (<code class="highlighter-rouge">v0.7.4</code>) of <code class="highlighter-rouge">crul</code> is on CRAN - an R6 based HTTP client for R made for developers. See the <a href="https://github.com/ropensci/crul/releases/tag/v0.7.4">release notes</a> for changes. Checkout the <a href="https://ropensci.github.io/crul/">docs</a> to get started. <a target="_blank" href="https://cran.rstudio.com/web/packages/crul"><img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a> <a target="_blank" href="https://github.com/ropensci/crul"><img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a>
    <blockquote>
      <p>request/response hooks now supported; improve <code class="highlighter-rouge">$parse</code> behavior; <code class="highlighter-rouge">Paginator</code> gains progress bar option</p>
    </blockquote>
  </li>
  <li>A new version (<code class="highlighter-rouge">v0.2.18</code>) of <code class="highlighter-rouge">rglobi</code> is on CRAN - R Interface to Global Biotic Interactions. See the <a href="https://github.com/ropensci/rglobi/releases/tag/v0.2.18">release notes</a> for changes. Checkout the <a href="https://cran.rstudio.com/web/packages/rglobi/vignettes/rglobi_vignette.html">vignette</a> to get started. <a target="_blank" href="https://cran.rstudio.com/web/packages/rglobi"><img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a> <a target="_blank" href="https://github.com/ropensci/rglobi"><img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a>
    <blockquote>
      <p>update cypher query endpoint after server upgrade to neo4j</p>
    </blockquote>
  </li>
  <li>A new version (<code class="highlighter-rouge">v0.5</code>) of <code class="highlighter-rouge">ssh</code> is on CRAN - Secure shell (SSH) client for R. See the <a href="https://github.com/ropensci/ssh/releases/tag/v0.5">release notes</a> for changes. Checkout the <a href="https://cran.rstudio.com/web/packages/ssh/vignettes/intro.html">vignette</a> to get started. <a target="_blank" href="https://cran.rstudio.com/web/packages/ssh"><img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a> <a target="_blank" href="https://github.com/ropensci/ssh"><img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a>
    <blockquote>
      <p>use key management fxns from <code class="highlighter-rouge">credentials</code> pkg; refactor password callback; rename <code class="highlighter-rouge">ssh_info()</code> to <code class="highlighter-rouge">ssh_session_info()</code></p>
    </blockquote>
  </li>
  <li>A new version (<code class="highlighter-rouge">v0.4.0</code>) of <code class="highlighter-rouge">tidyhydat</code> is on CRAN - Water Survey of Canada hydrometric data. See the <a href="https://github.com/ropensci/tidyhydat/blob/master/NEWS.md">release notes</a> for changes. Checkout the <a href="https://cran.r-project.org/web/packages/tidyhydat/vignettes/">vignettes</a> to get started. <a target="_blank" href="https://cran.rstudio.com/web/packages/tidyhydat"><img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a> <a target="_blank" href="https://github.com/ropensci/tidyhydat"><img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a>
    <blockquote>
      <p>all fxns now return either ‚Äúhy‚Äù or ‚Äúrealtime‚Äù class with associated print and plot methods; fxns now fail with an informative error message</p>
    </blockquote>
  </li>
  <li>A new version (<code class="highlighter-rouge">v0.4.2</code>) of <code class="highlighter-rouge">Rpolyhedra</code> is on CRAN - polyhedra database from various sources as R6 objects and visualizing capabilities. See the <a href="https://github.com/ropensci/Rpolyhedra/releases/tag/v0.4.2">release notes</a> for changes. Checkout the <a href="https://cran.rstudio.com/web/packages/Rpolyhedra/vignettes/Rpolyhedra.html">vignette</a> to get started. <a target="_blank" href="https://cran.rstudio.com/web/packages/Rpolyhedra"><img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a> <a target="_blank" href="https://github.com/ropensci/Rpolyhedra"><img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a>
    <blockquote>
      <p>calculates &amp; normalizes polyhedra size using <code class="highlighter-rouge">geometry::convhulln</code> instead of bounding box; a polyhedron now applies internal transformation matrix</p>
    </blockquote>
  </li>
  <li>A new version (<code class="highlighter-rouge">v0.2.8</code>) of <code class="highlighter-rouge">stplanr</code> is on CRAN - sustainable transport planning. See the <a href="https://github.com/ropensci/stplanr/blob/master/NEWS.md">release notes</a> for changes. Checkout the <a href="https://ropensci.github.io/stplanr/">docs</a> to get started. <a target="_blank" href="https://cran.rstudio.com/web/packages/stplanr"><img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a> <a target="_blank" href="https://github.com/ropensci/stplanr"><img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a>
    <blockquote>
      <p>STATS19 fxns such as <code class="highlighter-rouge">dl_stats19()</code> split-out into the new package stats19; new fxns <code class="highlighter-rouge">route_dodgr()</code>, <code class="highlighter-rouge">overline2()</code> ‚Ä¶ a substantial refactoring operation has begun</p>
    </blockquote>
  </li>
  <li>A new version (<code class="highlighter-rouge">v0.5.0</code>) of <code class="highlighter-rouge">bomrang</code> is on CRAN - Australian government Bureau of Meteorology (BOM) data. See the <a href="https://github.com/ropensci/bomrang/blob/master/NEWS.md">release notes</a> for changes. Checkout the <a href="https://ropensci.github.io/bomrang/">docs</a> to get started. <a target="_blank" href="https://cran.rstudio.com/web/packages/bomrang"><img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a> <a target="_blank" href="https://github.com/ropensci/bomrang"><img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a>
    <blockquote>
      <p>some new print methods; new aliases for some fxns; new default image when image failed to download</p>
    </blockquote>
  </li>
  <li>A new version (<code class="highlighter-rouge">v2.0-8</code>) of <code class="highlighter-rouge">CoordinateCleaner</code> is on CRAN - automated cleaning of occurrence records from biological collections. See the <a href="https://github.com/ropensci/CoordinateCleaner/blob/master/NEWS.md">release notes</a> for changes. Checkout the <a href="https://ropensci.github.io/CoordinateCleaner/">docs</a> to get started. <a target="_blank" href="https://cran.rstudio.com/web/packages/CoordinateCleaner"><img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a> <a target="_blank" href="https://github.com/ropensci/CoordinateCleaner"><img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a>
    <blockquote>
      <p>moved vignettes to online documentation; added an area column to the countryref dataset</p>
    </blockquote>
  </li>
  <li>A new version (<code class="highlighter-rouge">v0.0.10</code>) of <code class="highlighter-rouge">osmdata</code> is on CRAN - import OpenStreetMap data as simple features or spatial objects. See the <a href="https://github.com/ropensci/osmdata/releases/tag/v0.0.10">release notes</a> for changes. Checkout the <a href="https://ropensci.github.io/osmdata/">docs</a> to get started. <a target="_blank" href="https://cran.rstudio.com/web/packages/osmdata"><img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a> <a target="_blank" href="https://github.com/ropensci/osmdata"><img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a>
    <blockquote>
      <p>bug fix for vectorized lists of values in <code class="highlighter-rouge">add_osm_feature</code>; bug fix to ensure all sf data.frame objects have stringsAsFactors = FALSE</p>
    </blockquote>
  </li>
  <li>A new version (<code class="highlighter-rouge">v3.2-2</code>) of <code class="highlighter-rouge">clifro</code> is on CRAN - easily download and visualise climate data from CliFlo. See the <a href="https://github.com/ropensci/clifro/releases/tag/v3.2-2">release notes</a> for changes. Checkout the <a href="https://cran.rstudio.com/web/packages/clifro/vignettes/">vignettes</a> to get started. <a target="_blank" href="https://cran.rstudio.com/web/packages/clifro"><img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a> <a target="_blank" href="https://github.com/ropensci/clifro"><img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a>
    <blockquote>
      <p>fixed error in <code class="highlighter-rouge">cf_find_station()</code> fxn when searching for CliFlo stations based on geog. proximity &amp; when using a datatype</p>
    </blockquote>
  </li>
  <li>A new version (<code class="highlighter-rouge">v0.25.2</code>) of <code class="highlighter-rouge">git2r</code> is on CRAN - an interface to the libgit2 library, a pure C implementation of the Git core methods. See the <a href="https://github.com/ropensci/git2r/releases/tag/v0.25.2">release notes</a> for changes. Checkout the <a href="https://github.com/ropensci/git2r#introduction">README</a> to get started. <a target="_blank" href="https://cran.rstudio.com/web/packages/git2r"><img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a> <a target="_blank" href="https://github.com/ropensci/git2r"><img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a>
    <blockquote>
      <p>improved build configuration script</p>
    </blockquote>
  </li>
  <li>A new version (<code class="highlighter-rouge">v0.6.3</code>) of <code class="highlighter-rouge">rdhs</code> is on CRAN - Client for Demographic and Health Survey (DHS) Data. See the <a href="https://github.com/ropensci/rdhs/releases/tag/v0.6.3">release notes</a> for changes. Checkout the <a href="https://ropensci.github.io/rdhs/">docs</a> to get started. <a target="_blank" href="https://cran.rstudio.com/web/packages/rdhs"><img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a> <a target="_blank" href="https://github.com/ropensci/rdhs"><img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px" /></a>
    <blockquote>
      <p>new <code class="highlighter-rouge">delabel_df()</code> fxn for converting labelled data frames to characters</p>
    </blockquote>
  </li>
</ul>

<h3 id="un-archived">Un-Archived</h3>

<ul>
  <li><code class="highlighter-rouge">rdefra</code> was archived on CRAN for a bit - Now it‚Äôs back up.</li>
</ul>

<p><br /><br /></p>

<h2 id="software-review-">Software Review ‚úî</h2>

<p>We accept community contributed packages via our onboarding system - an open software review system, sorta like scholarly paper review, but way better. We‚Äôll highlight newly onboarded packages here. A huge thanks to our reviewers, who do a lot of work reviewing (see the <a href="https://ropensci.org/blog/2016/03/28/software-review">blog post on our review system</a>),
and the authors of the packages!</p>

<p>If you want to be a reviewer fill out <a href="https://ropensci.org/onboarding/">this short form</a>, and we‚Äôll ping you when there‚Äôs a submission that fits in your area of expertise.</p>

<p>The following package was recently submitted:</p>

<ul>
  <li><a href="https://github.com/cstubben/tidypmc">tidypmc</a> &gt; Parse full text XML documents from PMC
    <ul>
      <li>Author: <a href="https://github.com/cstubben">Chris Stubben</a></li>
      <li>Issue: <a href="https://github.com/ropensci/onboarding/issues/290">ropensci/onboarding#290</a></li>
      <li>Reviewers: not yet assigned</li>
    </ul>
  </li>
</ul>

<p><br /><br /></p>

<h2 id="use-cases">Use Cases</h2>

<p>The following 12 works use/cite rOpenSci software:</p>

<ul>
  <li>Phelps et al. used <a href="https://github.com/ropensci/rentrez">rentrez</a> in their paper on <a href="https://doi.org/10.3390/v11030240">Bat Research Networks and Viral Surveillance: Gaps and Opportunities in Western Asia</a> <sup id="fnref:1"><a href="#fn:1" class="footnote">1</a></sup></li>
  <li>Angel &amp; Markus used <a href="https://github.com/ropensci/rnoaa">rnoaa</a> in their article <a href="https://www.ideals.illinois.edu/handle/2142/103172">Frequency Distributions of Heavy Precipitation in Illinois: Updated Bulletin 70</a> <sup id="fnref:2"><a href="#fn:2" class="footnote">2</a></sup></li>
  <li>Mitchell &amp; Moseley used <a href="https://github.com/ropensci/RSelenium">RSelenium</a> in their paper <a href="https://doi.org/10.1101/572883">Deriving Accurate Lipid Classification based on Molecular Formula</a> <sup id="fnref:3"><a href="#fn:3" class="footnote">3</a></sup></li>
  <li>Solt used <a href="https://github.com/ropensci/tabulizer">tabulizer</a> in their work on <a href="http://www.globalfactcheck.org/documents/2019-03-08%20Standardized%20World%20Income%20Inequality%20Database.%20updating_swiid%20Solt.pdf">Measuring Income Inequality Across Countries and Over Time: The Standardized World Income Inequality Database</a> <sup id="fnref:4"><a href="#fn:4" class="footnote">4</a></sup></li>
  <li>Michalak et al. used <a href="https://github.com/ropensci/plotly">plotly</a> in their article <a href="https://doi.org/10.1101/573774">ComplexBrowser: a tool for identification and quantification of protein complexes in large scale proteomics datasets</a> <sup id="fnref:5"><a href="#fn:5" class="footnote">5</a></sup></li>
  <li>Su et al. used <a href="https://github.com/ropensci/plotly">plotly</a> in their article <a href="https://doi.org/10.1186/s13104-019-4179-2">TCC-GUI: a Shiny-based application for differential expression analysis of RNA-Seq count data</a> <sup id="fnref:6"><a href="#fn:6" class="footnote">6</a></sup></li>
  <li>K√∂nig et al. used <a href="https://github.com/ropensci/taxize">taxize</a> in their article <a href="https://doi.org/10.1371/journal.pbio.3000183">Biodiversity data integration‚ÄìThe significance of data resolution and domain</a> <sup id="fnref:7"><a href="#fn:7" class="footnote">7</a></sup></li>
  <li>Ravenhall et al. used <a href="https://github.com/ropensci/plotly">plotly</a> in their article <a href="https://doi.org/10.1186/s12859-019-2718-4">SV-Pop: population-based structural variant analysis and visualization</a> <sup id="fnref:8"><a href="#fn:8" class="footnote">8</a></sup></li>
  <li>Higino &amp; Vital used <a href="https://github.com/ropensci/taxize">taxize</a> in their paper <a href="https://doi.org/10.32942/osf.io/c63vj">Mapping and understanding the digital biodiversity knowledge about vertebrates in the Atlantic Rainforest</a> <sup id="fnref:9"><a href="#fn:9" class="footnote">9</a></sup></li>
  <li>Ahmed et al. wrote a paper about their package <a href="https://github.com/ropensci/colocr">colocr</a> in the rOpenSci suite: <a href="https://doi.org/10.7287/peerj.preprints.27613v1">colocr: An R package for conducting co-localization analysis on fluorescence microscopy images</a> <sup id="fnref:10"><a href="#fn:10" class="footnote">10</a></sup></li>
  <li>Jo et al. used <a href="https://github.com/ropensci/taxize">taxize</a> in their paper <a href="https://doi.org/10.4490/algae.2019.34.2.26">SoEM: a novel PCR-free biodiversity assessment method based on small-organelles enriched metagenomics</a> <sup id="fnref:11"><a href="#fn:11" class="footnote">11</a></sup></li>
  <li>Shefferson et al. used <a href="https://github.com/ropensci/rotl">rotl</a> in their paper <a href="https://doi.org/10.1111/1365-2745.13170">Does evolutionary history determine specificity in broad ecological interactions?</a> <sup id="fnref:12"><a href="#fn:12" class="footnote">12</a></sup></li>
</ul>

<p><br /><br /></p>

<h2 id="in-the-news">In The News</h2>

<p>Erik Gahner Larsen and Zolt√°n Fazekas have a bookdown book titled <em>Quantitative Politics with R</em> in which they include <em>Chapter 4: Get existing data</em> with a section on the <a href="http://qpolr.com/4-datadownload.html#data-european-social-survey-essurvey">European Social Survey</a> using the rOpenSci package <a href="https://github.com/ropensci/esssurvey">esssurvey</a>, maintained by Jorge Cimentada.</p>

<p><br /><br /></p>

<h2 id="call-for-contributors">Call For Contributors</h2>

<p>Part of the mission of rOpenSci is making sustainable software that users can rely on. Some software maintainers need to give up maintenance due to a variety of circumstances. When that happens we try to find new maintainers. We‚Äôve had five recent examples of maintainer transitions within rOpenSci:</p>

<ul>
  <li><a href="https://github.com/ropensci/RSelenium">RSelenium</a>: now maintained by <a href="https://github.com/juyeongkim">Ju Kim</a></li>
  <li><a href="https://github.com/ropensci/chromer">chromer</a>: now maintained by <a href="https://github.com/orchid00">Paula Andrea</a></li>
  <li><a href="https://github.com/ropensci/qualtRics">qualtRics</a>: now maintained by <a href="https://github.com/juliasilge">Julia Silge</a></li>
  <li><a href="https://github.com/ropensci/rsnps">rsnps</a>: now maintained by <a href="https://github.com/jooolia">Julia Gustavsen</a> and <a href="https://github.com/sinarueeger">Sina R√ºeger</a></li>
  <li><a href="https://github.com/ropensci/rdpla">rdpla</a>: now maintained by <a href="https://github.com/acolum">Alyssa Columbus</a></li>
</ul>

<p>We‚Äôve got only one package in need of new maintainers:</p>

<ul>
  <li><a href="https://github.com/ropensci/webchem">webchem</a>: The current maintainer of webchem, <a href="https://github.com/EDiLD">Eduard Sz√∂cs</a>, is looking for a new maintainer. See <a href="https://github.com/ropensci/webchem/issues/155">issue #155</a>. Do ping Eduard in that issue and/or email us at <a href="mailto:info@ropensci.org">info@ropensci.org</a></li>
</ul>

<p><br /><br /></p>

<hr style="display: block; height: 1px; border: 0; border-top: 3px solid #7CCCC8; margin: 1em 0; padding: 0; " />

<p><br /><br /></p>

<h3 id="keep-up-with-ropensci">Keep up with rOpenSci</h3>

<ul>
  <li>Mailing list: Sign up with an email address to get this newsletter sent to your inbox -&gt; <a href="https://ropensci.org/#subscribe">ropensci.org/#subscribe</a></li>
  <li>Alternatively, you can subscribe to this newsletter via our XML feed at <a href="https://news.ropensci.org/feed.xml">https://news.ropensci.org/feed.xml</a> or our JSON feed at <a href="https://news.ropensci.org/feed.json">https://news.ropensci.org/feed.json</a></li>
  <li>rOpenSci on Twitter: <a href="https://twitter.com/ropensci">@ropensci</a></li>
  <li>The rOpenSci blog at <a href="https://ropensci.org/blog">ropensci.org/blog</a> - you can subscribe in any RSS aggregator, or manually via <a href="https://ropensci.org/feed.xml">https://ropensci.org/feed.xml</a>. We also announce new blog posts on our Twitter account.</li>
</ul>

<p><br /></p>

<h4 id="footnotes">Footnotes</h4>

<div class="footnotes">
  <ol>
    <li id="fn:1">
      <p>Phelps, K., Hamel, L., Alhmoud, N., Ali, S., Bilgin, R., Sidamonidze, K., ‚Ä¶ Olival, K. (2019). Bat Research Networks and Viral Surveillance: Gaps and Opportunities in Western Asia. Viruses, 11(3), 240. <a href="https://doi.org/10.3390/v11030240">https://doi.org/10.3390/v11030240</a>¬†<a href="#fnref:1" class="reversefootnote">&#8617;</a></p>
    </li>
    <li id="fn:2">
      <p>Angel, J., &amp; Markus, M. (2019). Frequency Distributions of Heavy Precipitation in Illinois: Updated Bulletin 70. ISWS CR 2019-05. <a href="https://www.ideals.illinois.edu/handle/2142/103172">https://www.ideals.illinois.edu/handle/2142/103172</a>¬†<a href="#fnref:2" class="reversefootnote">&#8617;</a></p>
    </li>
    <li id="fn:3">
      <p>Mitchell, J. M., &amp; Moseley, H. N. B. (2019). Deriving Accurate Lipid Classification based on Molecular Formula. <a href="https://doi.org/10.1101/572883">https://doi.org/10.1101/572883</a>¬†<a href="#fnref:3" class="reversefootnote">&#8617;</a></p>
    </li>
    <li id="fn:4">
      <p>Solt, F. Measuring Income Inequality Across Countries and Over Time: The Standardized World Income Inequality Database [DRAFT]. <a href="http://www.globalfactcheck.org/documents/2019-03-08%20Standardized%20World%20Income%20Inequality%20Database.%20updating_swiid%20Solt.pdf">http://www.globalfactcheck.org/documents/2019-03-08%20Standardized%20World%20Income%20Inequality%20Database.%20updating_swiid%20Solt.pdf</a>¬†<a href="#fnref:4" class="reversefootnote">&#8617;</a></p>
    </li>
    <li id="fn:5">
      <p>Michalak, W., Tsiamis, V., Schw√§mmle, V., &amp; Rogowska-Wrzesi≈Ñska, A. (2019). ComplexBrowser: a tool for identification and quantification of protein complexes in large scale proteomics datasets. <a href="https://doi.org/10.1101/573774">https://doi.org/10.1101/573774</a>¬†<a href="#fnref:5" class="reversefootnote">&#8617;</a></p>
    </li>
    <li id="fn:6">
      <p>Su, W., Sun, J., Shimizu, K., &amp; Kadota, K. (2019). TCC-GUI: a Shiny-based application for differential expression analysis of RNA-Seq count data. BMC Research Notes, 12(1). <a href="https://doi.org/10.1186/s13104-019-4179-2">https://doi.org/10.1186/s13104-019-4179-2</a>¬†<a href="#fnref:6" class="reversefootnote">&#8617;</a></p>
    </li>
    <li id="fn:7">
      <p>K√∂nig, C., Weigelt, P., Schrader, J., Taylor, A., Kattge, J., &amp; Kreft, H. (2019). Biodiversity data integration‚ÄìThe significance of data resolution and domain. PLOS Biology, 17(3), e3000183. <a href="https://doi.org/10.1371/journal.pbio.3000183">https://doi.org/10.1371/journal.pbio.3000183</a>¬†<a href="#fnref:7" class="reversefootnote">&#8617;</a></p>
    </li>
    <li id="fn:8">
      <p>Ravenhall, M., Campino, S., &amp; Clark, T. G. (2019). SV-Pop: population-based structural variant analysis and visualization. BMC Bioinformatics, 20(1). <a href="https://doi.org/10.1186/s12859-019-2718-4">https://doi.org/10.1186/s12859-019-2718-4</a>¬†<a href="#fnref:8" class="reversefootnote">&#8617;</a></p>
    </li>
    <li id="fn:9">
      <p>Higino, G., &amp; Vital, M. V. C. (2019). Mapping and understanding the digital biodiversity knowledge about vertebrates in the Atlantic Rainforest. <a href="https://doi.org/10.32942/osf.io/c63vj">https://doi.org/10.32942/osf.io/c63vj</a>¬†<a href="#fnref:9" class="reversefootnote">&#8617;</a></p>
    </li>
    <li id="fn:10">
      <p>Ahmed, M., Lai, T. H., &amp; Kim, D. R. (2019). colocr: An R package for conducting co-localization analysis on fluorescence microscopy images. <a href="https://doi.org/10.7287/peerj.preprints.27613v1">https://doi.org/10.7287/peerj.preprints.27613v1</a>¬†<a href="#fnref:10" class="reversefootnote">&#8617;</a></p>
    </li>
    <li id="fn:11">
      <p>Jo, J., Lee, H.-G., Kim, K. Y., &amp; Park, C. (2019). SoEM: a novel PCR-free biodiversity assessment method based on small-organelles enriched metagenomics. ALGAE, 34(1), 57‚Äì70. <a href="https://doi.org/10.4490/algae.2019.34.2.26">https://doi.org/10.4490/algae.2019.34.2.26</a>¬†<a href="#fnref:11" class="reversefootnote">&#8617;</a></p>
    </li>
    <li id="fn:12">
      <p>Shefferson, R. P., Bunch, W., Cowden, C. C., Lee, Y., Kartzinel, T. R., Yukawa, T., ‚Ä¶ Jiang, H. (2019). Does evolutionary history determine specificity in broad ecological interactions? Journal of Ecology. <a href="https://doi.org/10.1111/1365-2745.13170">https://doi.org/10.1111/1365-2745.13170</a>¬†<a href="#fnref:12" class="reversefootnote">&#8617;</a></p>
    </li>
  </ol>
</div>

  </article>

</div>

      </div>
    </div>

    <footer class="site-footer">

  <div class="wrapper">

    <h2 class="footer-heading">rOpenSci Newsletter</h2>

    <div class="footer-col-wrapper">
      <div class="footer-col  footer-col-1">
        <ul class="contact-list">
          <li>rOpenSci Newsletter</li>
          <li><a href="mailto:info@ropensci.org">info@ropensci.org</a></li>
        </ul>
      </div>

      <div class="footer-col  footer-col-2">
        <ul class="social-media-list">
          
          <li>
            <a href="https://github.com/ropensci">
              <span class="icon  icon--github">
                <svg viewBox="0 0 16 16">
                  <path fill="#828282" d="M7.999,0.431c-4.285,0-7.76,3.474-7.76,7.761 c0,3.428,2.223,6.337,5.307,7.363c0.388,0.071,0.53-0.168,0.53-0.374c0-0.184-0.007-0.672-0.01-1.32 c-2.159,0.469-2.614-1.04-2.614-1.04c-0.353-0.896-0.862-1.135-0.862-1.135c-0.705-0.481,0.053-0.472,0.053-0.472 c0.779,0.055,1.189,0.8,1.189,0.8c0.692,1.186,1.816,0.843,2.258,0.645c0.071-0.502,0.271-0.843,0.493-1.037 C4.86,11.425,3.049,10.76,3.049,7.786c0-0.847,0.302-1.54,0.799-2.082C3.768,5.507,3.501,4.718,3.924,3.65 c0,0,0.652-0.209,2.134,0.796C6.677,4.273,7.34,4.187,8,4.184c0.659,0.003,1.323,0.089,1.943,0.261 c1.482-1.004,2.132-0.796,2.132-0.796c0.423,1.068,0.157,1.857,0.077,2.054c0.497,0.542,0.798,1.235,0.798,2.082 c0,2.981-1.814,3.637-3.543,3.829c0.279,0.24,0.527,0.713,0.527,1.437c0,1.037-0.01,1.874-0.01,2.129 c0,0.208,0.14,0.449,0.534,0.373c3.081-1.028,5.302-3.935,5.302-7.362C15.76,3.906,12.285,0.431,7.999,0.431z"/>
                </svg>
              </span>

              <span class="username">ropensci</span>
            </a>
          </li>
          

          
          <li>
            <a href="https://twitter.com/ropensci">
              <span class="icon  icon--twitter">
                <svg viewBox="0 0 16 16">
                  <path fill="#828282" d="M15.969,3.058c-0.586,0.26-1.217,0.436-1.878,0.515c0.675-0.405,1.194-1.045,1.438-1.809
                  c-0.632,0.375-1.332,0.647-2.076,0.793c-0.596-0.636-1.446-1.033-2.387-1.033c-1.806,0-3.27,1.464-3.27,3.27 c0,0.256,0.029,0.506,0.085,0.745C5.163,5.404,2.753,4.102,1.14,2.124C0.859,2.607,0.698,3.168,0.698,3.767 c0,1.134,0.577,2.135,1.455,2.722C1.616,6.472,1.112,6.325,0.671,6.08c0,0.014,0,0.027,0,0.041c0,1.584,1.127,2.906,2.623,3.206 C3.02,9.402,2.731,9.442,2.433,9.442c-0.211,0-0.416-0.021-0.615-0.059c0.416,1.299,1.624,2.245,3.055,2.271 c-1.119,0.877-2.529,1.4-4.061,1.4c-0.264,0-0.524-0.015-0.78-0.046c1.447,0.928,3.166,1.469,5.013,1.469 c6.015,0,9.304-4.983,9.304-9.304c0-0.142-0.003-0.283-0.009-0.423C14.976,4.29,15.531,3.714,15.969,3.058z"/>
                </svg>
              </span>

              <span class="username">ropensci</span>
            </a>
          </li>
          
        </ul>
      </div>

      <div class="footer-col  footer-col-3">
        <p class="text">rOpenSci Newsletter, including software, software reviews, what's happening in the community, events, and more.</p>
      </div>
    </div>

  </div>

<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-63168295-2', 'auto');
  ga('send', 'pageview');

</script>

</footer>


  </body>

</html>
