---
date: "2016-01-05T00:00:00Z"
title: Update 2016-01-05
---

## rOpenSci HQ

* We would like a few minutes of your time for a survey so we can learn about your research software needs. [ropensci.org/survey](https://ropensci.org/survey/)

## Software

* A new package (version `0.1.0`) `binomen` is [on CRAN](http://cran.rstudio.com/web/packages/binomen). `binomen` provides various taxonomic classes for defining a single taxon, multiple taxa, and a taxonomic data.frame. It is designed as a companion to [taxize][taxize], where you can get taxonomic data on taxonomic names from the web. See [vignette](https://cran.rstudio.com/web/packages/binomen/vignettes/binomen_vignette.html) for an introduction to the package. [Repository on GitHub][binomen].
* A new version (version `0.0.4`) of `webchem` is [on CRAN](https://cran.rstudio.com/web/packages/webchem/). See [release notes](https://github.com/ropensci/webchem/releases/tag/v0.0.4) for changes. [Repository on GitHub][webchem].
* A new version (`0.13.1`) of `git2r` is [on CRAN](http://cran.rstudio.com/web/packages/git2r). See the [NEWS](https://github.com/ropensci/git2r/blob/master/NEWS#L1-L18) for changes. [Repository on GitHub][git2r].
* A new version (`2.0.16`) of `plotly` is [on CRAN][plotlycran]. See the [NEWS](https://github.com/ropensci/plotly/blob/master/NEWS) for changes. [Repository on GitHub][plotly].
* A new version (`0.3.6`) of `gistr` is [on CRAN](http://cran.rstudio.com/web/packages/gistr). See the [release notes](https://github.com/ropensci/gistr/releases/tag/v0.3.6) for changes. [Repository on GitHub][gistr].
* A new version (`0.2.0`) of `rdryad` is [on CRAN](http://cran.rstudio.com/web/packages/rdryad). See the [release notes](https://github.com/ropensci/rdryad/releases/tag/v0.2.0) for changes. [Repository on GitHub][rdryad].
* A new version (`2.0.5`) of `RNeXML` is [on CRAN](http://cran.rstudio.com/web/packages/RNeXML). See the [NEWS](https://github.com/ropensci/RNeXML/blob/master/NEWS#L30-L40) for changes. [Repository on GitHub][rnexml].
* A new version (`0.2.0`) of `USAboundaries` is [on CRAN](https://cran.rstudio.com/web/packages/USAboundaries/). See the [NEWS](https://github.com/ropensci/USAboundaries/blob/master/NEWS#L1-L5) for changes. [Repository on GitHub][usaboundaries].

## CRAN dependencies

* [rfigshare][rfigshare] has a new package that imports it on CRAN: [zoon][zoon] - for reproducible and remixable species distribution modelling.
* A suite of packages (some older, some new) use [git2r][git2r]: [archivist](https://cran.rstudio.com/web/packages/archivist/index.html), [spm12r](https://cran.rstudio.com/web/packages/spm12r/index.html), [nat.templatebrains](https://cran.rstudio.com/web/packages/nat.templatebrains/index.html), and , [Rcereal](https://cran.rstudio.com/web/packages/Rcereal/index.html).

## Use cases

Eight recent papers cite rOpenSci software:

* Cimiano et al., in a project proposal titled [Continuous quality control for research data to ensure reproducibility: an institutional approach][cimiano] mentions rOpenSci [^1]
* Crouzier in a European Commission report titled [Science Ecosystem 2.0: how will change occur?][crouzier] mentions rOpenSci
* Vandepitte et al. have a paper in _Journal of Marine Science and Engineering_ titled [How Aphia—the platform behind several online and taxonomically oriented databases—can serve both the taxonomic community and the field of biodiversity informatics][vandepitte] in which they cite [taxize][taxize] [^2]
* Bevan has a paper in _Antiquity_ titled [The data deluge][vandepitte] in which he mentions rOpenSci [^3]
* Pilaar Birch & Graham have a letter in _Bioscience_ titled [A stable isotope data repository as part of neotoma, a paleoecological database][birch] in which they cite [neotoma][neotoma] [^4]
* Schaefer et al. have a paper in _Transactions of the American Fisheries Society_ titled [Dynamics of near-coastal fish assemblages following the deepwater horizon oil spill in the northern gulf of mexico][schaefer] in which they cite [rfishbase][rfishbase] [^5]
* Zizka & Antonelli have a pre-print in _Biorxiv_ titled [speciesgeocodeR: An R package for linking species occurrences, user-defined regions and phylogenetic trees for biogeography, ecology and evolution][zizka] in which they cite [rgbif][rgbif] [^6]
* Westcott & Schloss have a paper in _PeerJ_ titled [De novo clustering methods outperform reference-based methods for assigning 16S rRNA gene sequences to operational taxonomic units][westcott] in which they cite [rentrez][rentrez] [^7]

[binomen]: https://github.com/ropensci/binomen
[taxize]: https://github.com/ropensci/taxize
[webchem]: https://github.com/ropensci/webchem
[plotlycran]: https://cran.rstudio.com/web/packages/plotly/
[plotly]: https://github.com/ropensci/plotly
[git2r]: https://github.com/ropensci/git2r
[gistr]: https://github.com/ropensci/gistr
[rnexml]: https://github.com/ropensci/RNeXML
[usaboundaries]: https://github.com/ropensci/USAboundaries
[neotoma]: https://github.com/ropensci/neotoma
[rgbif]: https://github.com/ropensci/rgbif
[rentrez]: https://github.com/ropensci/rentrez
[rfigshare]: https://github.com/ropensci/rfigshare
[zoon]: https://github.com/zoonproject/zoon
[rdryad]: https://github.com/ropensci/rdryad
[rfishbase]: https://github.com/ropensci/rfishbase

[cimiano]: https://zenodo.org/record/31298
[crouzier]: https://zenodo.org/record/33044
[vandepitte]: http://www.mdpi.com/2077-1312/3/4/1448/htm
[birch]: https://bioscience.oxfordjournals.org/content/early/2015/09/16/biosci.biv133.extract
[schaefer]: http://www.tandfonline.com/doi/abs/10.1080/00028487.2015.1111253
[zizka]: http://biorxiv.org/content/early/2015/11/24/032755.abstract
[westcott]: https://peerj.com/articles/1487/

## References

[^1]: Cimiano, Philipp, and Ing Gerhard Sagerer. "Continuous quality control for research data to ensure reproducibility: an institutional approach." [https://doi.org/10.5281/zenodo.31298](https://doi.org/10.5281/zenodo.31298)
[^2]: Vandepitte, Leen, et al. "How Aphia—The Platform behind Several Online and Taxonomically Oriented Databases—Can Serve Both the Taxonomic Community and the Field of Biodiversity Informatics." Journal of Marine Science and Engineering 3.4 (2015): 1448-1473. [https://doi.org/10.3390/jmse3041448](https://doi.org/10.3390/jmse3041448)
[^3]: Bevan, Andrew. "The data deluge." Antiquity 89.348 (2015): 1473-1484. [https://doi.org/10.15184/aqy.2015.102](https://doi.org/10.15184/aqy.2015.102)
[^4]: Birch, Suzanne E. Pilaar, and Russell W. Graham. "A Stable Isotope Data Repository as Part of Neotoma, a Paleoecological Database." BioScience (2015): biv133. [https://doi.org/10.1093/biosci/biv133](https://doi.org/10.1093/biosci/biv133)
[^5]: Schaefer, Jacob, Nkrumah Frazier, and Jonathan Barr. "Dynamics of Near-Coastal Fish Assemblages following the Deepwater Horizon Oil Spill in the Northern Gulf of Mexico." Transactions of the American Fisheries Society 145.1 (2015): 108-119. [https://doi.org/10.1080/00028487.2015.1111253](https://doi.org/10.1080/00028487.2015.1111253)
[^6]: Zizka, Alexander, and Alexandre Antonelli. "speciesgeocodeR: An R package for linking species occurrences, user-defined regions and phylogenetic trees for biogeography, ecology and evolution." bioRxiv (2015): 032755. [https://doi.org/10.1101/032755](https://doi.org/10.1101/032755)
[^7]: Westcott, Sarah L., and Patrick D. Schloss. "De novo clustering methods outperform reference-based methods for assigning 16S rRNA gene sequences to operational taxonomic units." PeerJ 3 (2015): e1487. [https://doi.org/10.7717/peerj.1487](https://doi.org/10.7717/peerj.1487)
