---
date: "2017-01-03T00:00:00Z"
title: Update 2017-01-03
---

## Software

### New packages

* A new package `finch` (`v0.1.0`) is [on CRAN](https://cran.rstudio.com/web/packages/finch). `finch` parses Darwin Core simple and archive directories/zip files. One major source of these kinds of data come from the [Global Biodiversity Information Facility](http://www.gbif.org/) (GBIF). Check out the [README](https://github.com/ropensci/finch#finch) to get started. p.s. depends on `EML` below :) [Repository on GitHub][finch]
* A new package `EML` (`v1.0.1`) is [on CRAN](https://cran.rstudio.com/web/packages/EML). `EML` parses and serializes Ecological Metadata Language (EML, see <https://en.wikipedia.org/wiki/Ecological_Metadata_Language> for more information) files into S4 objects. Check out the vignettes to get started. [Repository on GitHub][EML]

### Releases

* A new version (`v3.1-1`) of `clifro` is on [CRAN](https://cran.rstudio.com/web/packages/clifro). See [release notes](https://github.com/ropensci/clifro/releases/tag/v3.1-1) for changes. [Repository on GitHub][clifro].
* A new version (`v0.3.0`) of `rbhl` is on [CRAN](https://cran.rstudio.com/web/packages/clifro). See [release notes](https://github.com/ropensci/rbhl/releases/tag/v0.3.0) for changes. [Repository on GitHub][rbhl].
* A new version (`v0.1.7-2`) of `stplanr` is on [CRAN](https://cran.rstudio.com/web/packages/stplanr). See [release notes](https://github.com/ropensci/stplanr/releases/tag/0.1.7) for changes. [Repository on GitHub][stplanr].
* Two new versions of `git2r` have hit CRAN since the last rOpenSci News: (`v0.17.0`) and (`v0.18.0`). `git2r` on [CRAN](https://cran.rstudio.com/web/packages/git2r). See release notes for [v0.17.0](https://github.com/ropensci/git2r/releases/tag/v0.17.0) and [v0.18.0](https://github.com/ropensci/git2r/releases/tag/v0.18.0) for changes. [Repository on GitHub][git2r].
* A new version (`v0.2.0`) of `crul` is on [CRAN](https://cran.rstudio.com/web/packages/crul). This version includes more documentation, fixed URL encoding, fixed parsing of request and response headers, and now supports multiple query parameters of the same name. See [release notes](https://github.com/ropensci/crul/releases/tag/v0.2.0) for changes and links to issues. [Repository on GitHub][crul].

<br><br>

## Onboarding

We accept community contributed packages via our onboarding system - a software review system, sorta like scholarly paper review, but way better. We'll highlight new packages here that have come through this system. A huge thanks to our reviewers, who do a lot of work reviewing (see the [blog post on our review system](https://ropensci.org/blog/2016/03/28/software-review)),
and the authors of the packages!

If you want to be a reviewer we have a new [form](https://ropensci.org/onboarding/) you can fill out, and we'll ping you when there's a submission that fits in your area of expertise.

The following package was recently submitted to our onboarding process and is undergoing review:

* [JSTORr][] > Simple text mining and document clustering of JSTOR journal articles
    * Author: [Ben Marwick](https://github.com/benmarwick)
    * Issue: [ropensci/onboarding#86](https://github.com/ropensci/onboarding/issues/86)
    * Reviewers:
        * [Thomas Leeper](https://github.com/leeper)

<br><br>

## Use cases

Two blog posts and many new (and some old) papers cite rOpenSci packages:

* Kan Nishida wrote a nice blog post on [Making Maps for Canada’s Provisions and Census Divisions in R](https://blog.exploratory.io/making-maps-for-canadas-provisions-and-census-divisions-in-r-c189b88ccd8a#.gzi5khrh6), which included use of [geojsonio][geojsonio], an rOpenSci package for working with GeoJSON data. [^1]
* Tuja Sonkkila wrote a nice blog post on [Semi-automatic IT at work](https://blogs.aalto.fi/suoritin/2017/01/02/impact/), which used [rAltmetric][rAltmetric], an rOpenSci package for working with [Altmetric.com](https://altmetric.com/) data. Check out her code in her repository [tts/finunipolicy](https://github.com/tts/finunipolicy). [^2] 
* Gilles _et al_. cite and use [rgbif][rgbif] in their paper [RAINBIO: a mega-database of tropical African vascular plants distributions](https://doi.org/10.3897/phytokeys.74.9723) [^3]
* Lundgren & Christin cite and use [rgbif][rgbif] in their paper [Despite phylogenetic effects, C3-C4 lineages bridge the ecological gap to C4 photosynthesis](https://doi.org/10.1093/jxb/erw451) [^4]
* Galata _et al_. cite and use [taxize][taxize] in their paper [Comparing genome versus proteome-based identification of clinical bacterial isolates](https://doi.org/10.1093/bib/bbw122) [^5]
* Hassall _et al_. cite and use [bold][bold] in their paper [Phenological shifts in hoverflies (Diptera: Syrphidae): linking measurement and mechanism](https://doi.org/10.1111/ecog.02623) [^6]
* Hampton _et al_. cite and use [rentrez][rentrez] in their paper [The Tao of open science for ecology](https://doi.org/10.1890/es14-00402.1) [^7]
* Nguyen _et al_. cite and use [rentrez][rentrez] in their paper [Integrative Computational and Experimental Approaches to Establish a Post-Myocardial Infarction Knowledge Map](https://doi.org/10.1371/journal.pcbi.1003472) [^8]
* Killen _et al_. cite and use [rotl][rotl] in their paper [Do method and species lifestyle affect measures of maximum metabolic rate in fishes?](https://doi.org/10.1111/jfb.13195) [^9]
* Estrada-Peña & de la Fuente cite and use [rotl][rotl] in their paper [Species interactions in occurrence data for a community of tick-transmitted pathogens](https://doi.org/10.1038/sdata.2016.56) [^10]
* Bauer _et al_. cite [fulltext][fulltext] and [rcrossref][rcrossref] in their paper [The Quality of Citations: Towards Quantifying Qualitative Impact in Social Science Research](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=2874549) [^11]
* Mackinnon _et al_. cite and use [rsnps][rsnps] in their paper [Environmental Correlation Analysis for Genes Associated with Protection against Malaria](https://doi.org/10.1093/molbev/msw004) [^12]
* Söderholm cites and uses [rsnps][rsnps] in his thesis [The Complex Genetics of Multiple Sclerosis: A preliminary study of MS-associated SNPs prior to a larger genotyping project](http://www.diva-portal.org/smash/record.jsf?pid=diva2%3A939390&dswid=-3564) [^13]
* Kolb uses and cites [geonames][geonames] in his paper [Visualizing GeoData with R](https://doi.org/10.17713/ajs.v45i1.88) [^14]

<br><br>

-----------------------------

<br><br>

## Keep up with rOpenSci news

There are a number of ways to keep up with what rOpenSci is doing:

* Mailing list: Sign up with an email address to get new blog posts sent to your inbox -> [ropensci.org/#subscribe](https://ropensci.org/#subscribe)
* rOpenSci on Twitter: we're [@ropensci](https://twitter.com/ropensci)
* The rOpenSci blog at [ropensci.org/blog](https://ropensci.org/blog) - you can subscribe in any RSS aggregator, or manually via [https://ropensci.org/feed.xml](https://ropensci.org/feed.xml). We also announce new blog posts on our Twitter account.

[geojsonio]: https://github.com/ropensci/geojsonio
[finch]: https://github.com/ropensci/finch
[EML]: https://github.com/ropensci/EML
[clifro]: https://github.com/ropensci/clifro
[rbhl]: https://github.com/ropensci/rbhl
[stplanr]: https://github.com/ropensci/stplanr
[git2r]: https://github.com/ropensci/git2r
[crul]: https://github.com/ropensci/crul
[rAltmetric]: https://github.com/ropensci/rAltmetric
[rgbif]: https://github.com/ropensci/rgbif 
[taxize]: https://github.com/ropensci/taxize 
[bold]: https://github.com/ropensci/bold 
[rentrez]: https://github.com/ropensci/rentrez 
[rotl]: https://github.com/ropensci/rotl
[fulltext]: https://github.com/ropensci/fulltext
[rcrossref]: https://github.com/ropensci/rcrossref
[rsnps]: https://github.com/ropensci/rsnps
[geonames]: https://github.com/ropensci/geonames
[JSTORr]: https://github.com/benmarwick/JSTORr

<br><br>

### Footnotes

[^1]: Nishida, Kan. 2017. Making Maps for Canada’s Provisions and Census Divisions in R. <https://blog.exploratory.io/making-maps-for-canadas-provisions-and-census-divisions-in-r-c189b88ccd8a#.gzi5khrh6>
[^2]: Sonkkila, Tuja. 2017. Semi-automatic IT at work. <https://blogs.aalto.fi/suoritin/2017/01/02/impact/>
[^3]: Gilles, D., Zaiss, R., Blach-Overgaard, A., Catarino, L., Damen, T., Deblauwe, V., et al. (2016). RAINBIO: a mega-database of tropical African vascular plants distributions. PhytoKeys, 74, 1–18. <https://doi.org/10.3897/phytokeys.74.9723>
[^4]: Lundgren, M. R., & Christin, P.-A. (2016). Despite phylogenetic effects, C3-C4 lineages bridge the ecological gap to C4 photosynthesis. Journal of Experimental Botany, erw451. <https://doi.org/10.1093/jxb/erw451>
[^5]: Galata, V., Backes, C., Laczny, C. C., Hemmrich-Stanisak, G., Li, H., Smoot, L., et al. (2016). Comparing genome versus proteome-based identification of clinical bacterial isolates. Briefings in Bioinformatics, bbw122. <https://doi.org/10.1093/bib/bbw122>
[^6]: Hassall, C., Owen, J., & Gilbert, F. (2016). Phenological shifts in hoverflies (Diptera: Syrphidae): linking measurement and mechanism. Ecography. <https://doi.org/10.1111/ecog.02623>
[^7]: Hampton, S. E., Anderson, S. S., Bagby, S. C., Gries, C., Han, X., Hart, E. M., et al. (2015). The Tao of open science for ecology. Ecosphere, 6(7), art120. <https://doi.org/10.1890/es14-00402.1>
[^8]: Nguyen, N. T., Zhang, X., Wu, C., Lange, R. A., Chilton, R. J., Lindsey, M. L., & Jin, Y.-F. (2014). Integrative Computational and Experimental Approaches to Establish a Post-Myocardial Infarction Knowledge Map. PLoS Computational Biology, 10(3), e1003472. <https://doi.org/10.1371/journal.pcbi.1003472>
[^9]: Killen, S. S., Norin, T., & Halsey, L. G. (2016). Do method and species lifestyle affect measures of maximum metabolic rate in fishes? Journal of Fish Biology. <https://doi.org/10.1111/jfb.13195>
[^10]: Estrada-Peña, A., & de la Fuente, J. (2016). Species interactions in occurrence data for a community of tick-transmitted pathogens. Scientific Data, 3, 160056. <https://doi.org/10.1038/sdata.2016.56>
[^11]: Bauer, P. C., Barbera, P., & Munzert, S. (2016). The Quality of Citations: Towards Quantifying Qualitative Impact in Social Science Research. <https://papers.ssrn.com/sol3/papers.cfm?abstract_id=2874549>
[^12]: Mackinnon, M. J., Ndila, C., Uyoga, S., Macharia, A., Snow, R. W., Band, G., et al. (2016). Environmental Correlation Analysis for Genes Associated with Protection against Malaria. Molecular Biology and Evolution, 33(5), 1188–1204. <https://doi.org/10.1093/molbev/msw004>
[^13]: Söderholm, S. (2016). The Complex Genetics of Multiple Sclerosis: A preliminary study of MS-associated SNPs prior to a larger genotyping project. <http://www.diva-portal.org/smash/record.jsf?pid=diva2%3A939390&dswid=-3564>
[^14]: Kolb, J.-P. (2016). Visualizing GeoData with R. Austrian Journal of Statistics, 45(1), 45. <https://doi.org/10.17713/ajs.v45i1.88>
