---
date: "2020-04-13T00:00:00Z"
slug: "2020-04-13"
title: "2020-04-13"
---

## Software 📦

CRAN: <img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px">
GitHub: <img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px">

### New Versions

* A new version (`v0.9.0`) of `rplos` is on CRAN - Interface to the Search API for Public Library of Science journals. See the [release notes](https://github.com/ropensci/rplos/releases/tag/v0.9.0) for changes. Checkout the [vignettes](https://cran.rstudio.com/web/packages/rplos/vignettes/) to get started. {{ "rplos" | image_cran }} {{ "rplos" | image_github }}
* A new version (`v0.3.2`) of `crminer` is on CRAN - Fetch scholarly full text from Crossref. See the [release notes](https://github.com/ropensci/crminer/releases/tag/v0.3.2) for changes. Checkout the [README](https://github.com/ropensci/crminer) to get started. {{ "crminer" | image_cran }} {{ "crminer" | image_github }}
* A new version (`v0.9.2`) of `geojsonio` is on CRAN - Convert Data from and to GeoJSON or TopoJSON. See the [release notes](https://github.com/ropensci/geojsonio/releases/tag/v0.9.2) for changes. Checkout the [vignettes](https://cran.rstudio.com/web/packages/geojsonio/vignettes/) to get started. {{ "geojsonio" | image_cran }} {{ "geojsonio" | image_github }}
* A new version (`v0.9.6`) of `rnoaa` is on CRAN - NOAA Weather Data from R. See the [release notes](https://github.com/ropensci/rnoaa/releases/tag/v0.9.6) for changes. Checkout the [vignettes](https://cran.rstudio.com/web/packages/rnoaa/vignettes/) to get started. {{ "rnoaa" | image_cran }} {{ "rnoaa" | image_github }}
* A new version (`v0.3.2`) of `scrubr` is on CRAN - clean biological occurrence records. See the [release notes](https://github.com/ropensci/scrubr/releases/tag/v0.3.2) for changes. Checkout the [docs](https://docs.ropensci.org/scrubr/) to get started. {{ "scrubr" | image_cran }} {{ "scrubr" | image_github }}
* A new version (`v2.0.5`) of `ijtiff` is on CRAN - Comprehensive TIFF I/O with full support for ImageJ TIFF files. See the [release notes](https://github.com/ropensci/ijtiff/blob/master/NEWS.md) for changes. Checkout the [docs](https://ropensci.github.io/ijtiff) to get started. {{ "ijtiff" | image_cran }} {{ "ijtiff" | image_github }}
* A new version (`v0.3.8`) of `rdefra` is on CRAN - get data from DEFRA's UK-AIR website <https://uk-air.defra.gov.uk/>. See the [release notes](https://github.com/ropensci/rdefra/releases/tag/v0.3.8) for changes. Checkout the [rdefra vignette](https://cran.rstudio.com/web/packages/rdefra/vignettes/rdefra_vignette.html) to get started. {{ "rdefra" | image_cran }} {{ "rdefra" | image_github }}
* A new version (`v0.5.2`) of `stplanr` is on CRAN - sustainable transport planning. See the [release notes](https://github.com/ropensci/stplanr/blob/master/NEWS.md) for changes. Checkout the [docs](https://docs.ropensci.org/stplanr/) to get started. {{ "stplanr" | image_cran }} {{ "stplanr" | image_github }}
* A new version (`v4.9.2.1`) of `plotly` is on CRAN - Create Interactive Web Graphics via plotly.js. See the [release notes](https://github.com/ropensci/plotly/blob/master/NEWS.md) for changes. Checkout the [plotly book](https://plotly-book.cpsievert.me/) to get started. {{ "plotly" | image_cran }} {{ "plotly" | image_github }}
* A new version (`v0.4.1`) of `auk` is on CRAN - eBird data extraction and processing. See the [release notes](https://github.com/CornellLabofOrnithology/auk/blob/master/NEWS.md) for changes. Checkout the [docs](http://cornelllabofornithology.github.io/auk/) to get started. {{ "auk" | image_cran }} <a target="_blank" href="https://github.com/CornellLabofOrnithology/auk"><img src="../assets/img/github-alt.png" style="border-radius: 6px 6px 6px 6px" width="25"></a>
* A new version (`v0.3.8`) of `jstor` is on CRAN - read data from JSTOR/DfR. See the [release notes](https://github.com/ropensci/jstor/blob/master/NEWS.md) for changes. Checkout the [docs](https://ropensci.github.io/jstor) to get started. {{ "jstor" | image_cran }} {{ "jstor" | image_github }}
* A new version (`v1.0`) of `tradestatistics` is on CRAN - open trade statistics api wrapper and utility program. See the [release notes](https://github.com/ropensci/tradestatistics/blob/master/NEWS.md) for changes. Checkout the [docs](https://docs.ropensci.org/tradestatistics/) to get started. {{ "tradestatistics" | image_cran }} {{ "tradestatistics" | image_github }}
* A new version (`v1.4`) of `xslt` is on CRAN - extensible style-sheet language transformations. See the [release notes](https://github.com/ropensci/xslt/blob/master/NEWS) for changes. Checkout the [README](https://github.com/ropensci/xslt#readme) to get started. {{ "xslt" | image_cran }} {{ "xslt" | image_github }}
* A new version (`v1.1`) of `lightr` is on CRAN - read spectrometric data and metadata. See the [release notes](https://github.com/ropensci/lightr/releases/tag/v1.1) for changes. Checkout the [docs](https://docs.ropensci.org/lightr/) to get started. {{ "lightr" | image_cran }} {{ "lightr" | image_github }}
* A new version (`v0.9.94`) of `taxize` is on CRAN - taxonomic toolbelt for R. See the [release notes](https://github.com/ropensci/taxize/releases/tag/v0.9.94) for changes. Checkout the [taxize book](https://taxize.dev/) to get started. {{ "taxize" | image_cran }} {{ "taxize" | image_github }}
* A new version (`v0.5.4`) of `vcr` is on CRAN - record HTTP calls to disk for test caching. See the [release notes](https://github.com/ropensci/vcr/releases/tag/v0.5.4) for changes. Checkout the [HTTP testing book](https://books.ropensci.org/http-testing/) to get started. {{ "vcr" | image_cran }} {{ "vcr" | image_github }}

<br><br>

## Software Review ✔

In the interest of reducing load on reviewers and editors as we manage the COVID-19 crisis, rOpenSci is temporarily pausing new submissions for software peer review. Please check back at our [software review repository](https://github.com/ropensci/software-review) after 17 April for updates. In this period new submissions will not be handled, nor new reviewers assigned. Reviews and responses to reviews will be handled on a 'best effort' basis, but no follow-up reminders will be sent. Other rOpenSci community activities continue. We express our continued great appreciation for the work of our authors and reviewers.

<br><br>

## On the blog

[Stefanie Butland](https://ropensci.org/author/stefanie-butland/) and [Maëlle Salmon](https://ropensci.org/author/ma%C3%ABlle-salmon/) wrote a post on lessons learned from creating the new blog guide: [10 Things We Learned in Creating the Blog Guide with bookdown](https://ropensci.org/blog/2020/04/07/bookdown-learnings/)

<br><br>

## Use Cases

The following 21 works use/cite rOpenSci software:

* Cirtwill _et al_. used [taxize][] in their paper [Related plants tend to share pollinators and herbivores, but strength of phylogenetic signal varies among plant families](https://doi.org/10.1111/nph.16420) [^1]
* Léveillé-Bourret _et al_. used [rgbif][] and [CoordinateCleaner][] in their paper [RAD sequencing resolves the phylogeny, taxonomy and biogeography of Trichophoreae despite a recent rapid radiation (Cyperaceae](https://doi.org/10.1016/j.ympev.2019.106727) [^2]
* Moudrý & Devillers used [CoordinateCleaner][] in their paper [Quality and usability challenges of global marine biodiversity databases: An example for marine mammal data](https://doi.org/10.1016/j.ecoinf.2020.101051) [^3]
* Deng _et al_. used [hunspell][] in their paper [Sentiment analysis of real-world migraine tweets for population research](https://doi.org/10.1177/2515816319898867) [^4]
* Shan _et al_. used [rotl][] in their paper [Considerations in adapting CRISPR/Cas9 in nongenetic model plant systems](https://doi.org/10.1002/aps3.11314) [^5]
* Horne _et al_. used [rotl][] in their paper [Selection for increased male size predicts variation in sexual size dimorphism among fish species](https://doi.org/10.1098/rspb.2019.2640) [^6]
* Akpınar _et al_. used [taxize][] in their paper [Pathogenicity of human mtDNA variants is revealed by combining a novel phylogenetic analysis with machine learning](https://doi.org/10.1101/2020.01.10.902239) [^7]
* Milla _et al_. used [rgbif][] and [CoordinateCleaner][] in their paper [Phylogenetic patterns and phenotypic profiles of the species of plants and mammals farmed for food](https://doi.org/10.1038/s41559-018-0690-4) [^8]
* Scherer _et al_. used [NLMR][] and [landscapetools][] in their paper [Moving infections: individual movement decisions drive disease persistence in spatially structured landscapes](https://doi.org/10.1111/oik.07002) [^9]
* Burns & Bloom used [rfishbase][] in their paper [Migratory lineages rapidly evolve larger body sizes than non-migratory relatives in ray-finned fishes](https://doi.org/10.1098/rspb.2019.2615) [^10]
* Santos used [rentrez][] in their paper [Phylogenetic Analyses of Chemokine Receptors from Sequence Retrieval to Phylogenetic Trees](https://link.springer.com/protocol/10.1007/978-1-0716-0247-8_27) [^11]
* Walczyńska _et al_. used [rotl][] in their paper [Should I shrink or should I flow? – body size adjustment to thermo-oxygenic niche](https://doi.org/10.1101/2020.01.14.905901) [^12]
* Mathes used [rnoaa][] in their paper [The Dynamics of Residential Sorting and Health: Implications of Climate Change in the US](https://sophiemathescom.files.wordpress.com/2019/11/job_market_paper_sophie_mathes-1.pdf) [^13]
* Théroux-Rancourt _et al_. used [brranching][] in their paper [Maximum CO2 diffusion inside leaves is limited by the scaling of cell size and genome size](https://doi.org/10.1101/2020.01.16.904458) [^14]
* M. _et al_. used [rgbif][] in their paper [Creating Maps](NULL) [^15]
* Wraith _et al_. used [rgbif][] in their paper [Orchid conservation and research: An analysis of gaps and priorities for globally Red Listed species](https://doi.org/10.1007/s13280-019-01306-7) [^16]
* Vincent & Bowler used [rglobi][] in their paper [Diatoms Are Selective Segregators in Global Ocean Planktonic Communities](https://doi.org/10.1128/msystems.00444-19) [^17]
* Sawa _et al_. used [phylogram][] in their paper [Molecular epidemiology of clinically high‐risk Pseudomonas aeruginosa strains: Practical overview](https://doi.org/10.1111/1348-0421.12776) [^18]
* Yuan _et al_. used [MODISTools][] in their paper [Habitat use, interspecific competition and phylogenetic history shape the evolution of claw and toepad morphology in Lesser Antillean anoles](https://doi.org/10.1093/biolinnean/blz203) [^19]
* Gilbertson _et al_. used [spatsoc][] in their paper [Trade‐offs with telemetry‐derived contact networks for infectious disease studies in wildlife](https://doi.org/10.1111/2041-210x.13355) [^20]
* Bachman _et al_. used [taxize][] and [wicket][] and [rgbif][] in their paper [Rapid Least Concern: towards automating Red List assessments](https://doi.org/10.3897/bdj.8.e47018) [^21]

<br><br>

## From the Forum

We have a [discussion forum](https://discuss.ropensci.org) (using [Discourse](https://www.discourse.org/)) for the rOpenSci community. It's a really nice way to have conversations on the internet. From time to time we'll highlight recent discussions of interest.

<br>

A new use case was reported in the forum recently:

* [Emmanuel Ọlámíjùwọ́n](https://twitter.com/eolamijuwon) shared a use case for our pacakge [magick][]: [{magick}ally Visualize Historical Google Maps Traffic Data](https://discuss.ropensci.org/t/magick-ally-visualize-historical-google-maps-traffic-data/2026). [magick][] is a package for advanced graphics and image-processing in R

<br><br>

## Call For Maintainers

Part of the mission of rOpenSci is making sustainable software that users can rely on. Some software maintainers need to give up maintenance due to a variety of circumstances. When that happens we try to find new maintainers. Checkout our [Contributing Guide](https://devguide.ropensci.org/contributingguide.html) for why and how to contribute to rOpenSci.

We've had eight recent examples of maintainer transitions within rOpenSci:

- [RSelenium][]: now maintained by [Ju Kim](https://github.com/juyeongkim)
- [chromer][]: now maintained by [Paula Andrea](https://github.com/orchid00)
- [qualtRics][]: now maintained by [Julia Silge](https://github.com/juliasilge)
- [rsnps][]: now maintained by [Julia Gustavsen](https://github.com/jooolia) and [Sina Rüeger](https://github.com/sinarueeger)
- [rdpla][]: now maintained by [Alyssa Columbus](https://github.com/acolum)
- [webchem][]: now maintained by [Erik Sapper](https://github.com/eriksapper)
- [mregions][]: in process of moving to maintenance by [VLIZ](http://www.vliz.be)
- [rWBclimate][]: now maintained by [Sergio Ibarra-Espinosa](https://github.com/ibarraespinosa) and [Amanda Rehbein](https://github.com/salvatirehbein)

We don't have any packages looking for new maintainers - if you are a maintainer and want to hand over that role to someone else get in touch with us.

<br><br>

## Get involved with rOpenSci

We maintain a [Contributing Guide](https://devguide.ropensci.org/contributingguide.html) that can help direct you to the right place, whether you want to make code contributions, non-code contributions, or other things like sharing use cases.

<br><br>

<hr style="display: block; height: 1px; border: 0; border-top: 3px solid #7CCCC8; margin: 1em 0; padding: 0; ">

<br><br>

### Keep up with rOpenSci

* Mailing list: Sign up with an email address to get this newsletter sent to your inbox -> [ropensci.org/#subscribe](https://ropensci.org/#subscribe)
* Alternatively, you can subscribe to this newsletter via our XML feed at <https://news.ropensci.org/feed.xml> or our JSON feed at <https://news.ropensci.org/feed.json>
* rOpenSci on Twitter: [@ropensci](https://twitter.com/ropensci)
* The rOpenSci blog at [ropensci.org/blog](https://ropensci.org/blog) - you can subscribe in any RSS aggregator, or manually via <https://ropensci.org/feed.xml>. We also announce new blog posts on our Twitter account.

<br>

#### Footnotes

[^1]: Cirtwill, A. R., Dalla Riva, G. V., Baker, N. J., Ohlsson, M., Norström, I., Wohlfarth, I., … Stouffer, D. B. (2020). Related plants tend to share pollinators and herbivores, but strength of phylogenetic signal varies among plant families. New Phytologist. <https://doi.org/10.1111/nph.16420>
[^2]: Léveillé-Bourret, É., Chen, B.-H., Garon-Labrecque, M.-È., Ford, B. A., & Starr, J. R. (2020). RAD sequencing resolves the phylogeny, taxonomy and biogeography of Trichophoreae despite a recent rapid radiation (Cyperaceae). Molecular Phylogenetics and Evolution, 145, 106727. <https://doi.org/10.1016/j.ympev.2019.106727>
[^3]: Moudrý, V., & Devillers, R. (2020). Quality and usability challenges of global marine biodiversity databases: An example for marine mammal data. Ecological Informatics, 56, 101051. <https://doi.org/10.1016/j.ecoinf.2020.101051>
[^4]: Deng, H., Wang, Q., Turner, D. P., Sexton, K. E., Burns, S. M., Eikermann, M., … Houle, T. T. (2020). Sentiment analysis of real-world migraine tweets for population research. Cephalalgia Reports, 3, 251581631989886. <https://doi.org/10.1177/2515816319898867>
[^5]: Shan, S., Soltis, P. S., Soltis, D. E., & Yang, B. (2020). Considerations in adapting CRISPR/Cas9 in nongenetic model plant systems. Applications in Plant Sciences, 8(1). <https://doi.org/10.1002/aps3.11314>
[^6]: Horne, C. R., Hirst, A. G., & Atkinson, D. (2020). Selection for increased male size predicts variation in sexual size dimorphism among fish species. Proceedings of the Royal Society B: Biological Sciences, 287(1918), 20192640. <https://doi.org/10.1098/rspb.2019.2640>
[^7]: Akpınar, B. A., Carlson, P. O., Paavilainen, V. O., & Dunn, C. D. (2020). Pathogenicity of human mtDNA variants is revealed by combining a novel phylogenetic analysis with machine learning. <https://doi.org/10.1101/2020.01.10.902239>
[^8]: Milla, R., Bastida, J. M., Turcotte, M. M., Jones, G., Violle, C., Osborne, C. P., … Byun, C. (2018). Phylogenetic patterns and phenotypic profiles of the species of plants and mammals farmed for food. Nature Ecology & Evolution, 2(11), 1808–1817. <https://doi.org/10.1038/s41559-018-0690-4>
[^9]: Scherer, C., Radchuk, V., Franz, M., Thulke, H., Lange, M., Grimm, V., & Kramer‐Schadt, S. (2020). Moving infections: individual movement decisions drive disease persistence in spatially structured landscapes. Oikos. <https://doi.org/10.1111/oik.07002>
[^10]: Burns, M. D., & Bloom, D. D. (2020). Migratory lineages rapidly evolve larger body sizes than non-migratory relatives in ray-finned fishes. Proceedings of the Royal Society B: Biological Sciences, 287(1918), 20192615. <https://doi.org/10.1098/rspb.2019.2615>
[^11]: Santos, J. C. (2020). Phylogenetic Analyses of Chemokine Receptors from Sequence Retrieval to Phylogenetic Trees. In Immune Mediators in Cancer (pp. 313-343). Humana, New York, NY. <https://link.springer.com/protocol/10.1007/978-1-0716-0247-8_27>
[^12]: Walczyńska, A., Gudowska, A., & Sobczyk, Ł. (2020). Should I shrink or should I flow? – body size adjustment to thermo-oxygenic niche. <https://doi.org/10.1101/2020.01.14.905901>
[^13]: Mathes, S. (2019). The Dynamics of Residential Sorting and Health: Implications of Climate Change in the US. <https://sophiemathescom.files.wordpress.com/2019/11/job_market_paper_sophie_mathes-1.pdf>
[^14]: Théroux-Rancourt, G., Roddy, A. B., Earles, J. M., Gilbert, M. E., Zwieniecki, M. A., Boyce, C. K., … Brodersen, C. R. (2020). Maximum CO2 diffusion inside leaves is limited by the scaling of cell size and genome size. <https://doi.org/10.1101/2020.01.16.904458>
[^15]: Lakicevic M., Povak N., Reynolds K.M. (2020) Creating Maps. In: Introduction to R for Terrestrial Ecology. Springer, Cham <https://link.springer.com/chapter/10.1007/978-3-030-27603-4_3>
[^16]: Wraith, J., Norman, P., & Pickering, C. (2020). Orchid conservation and research: An analysis of gaps and priorities for globally Red Listed species. Ambio. <https://doi.org/10.1007/s13280-019-01306-7>
[^17]: Vincent, F., & Bowler, C. (2020). Diatoms Are Selective Segregators in Global Ocean Planktonic Communities. mSystems, 5(1). <https://doi.org/10.1128/msystems.00444-19>
[^18]: Sawa, T., Momiyama, K., Mihara, T., Kainuma, A., Kinoshita, M., & Moriyama, K. (2020). Molecular epidemiology of clinically high‐risk Pseudomonas aeruginosa strains: Practical overview. Microbiology and Immunology. <https://doi.org/10.1111/1348-0421.12776>
[^19]: Yuan, M. L., Jung, C., Wake, M. H., & Wang, I. J. (2020). Habitat use, interspecific competition and phylogenetic history shape the evolution of claw and toepad morphology in Lesser Antillean anoles. Biological Journal of the Linnean Society, 129(3), 630–643. <https://doi.org/10.1093/biolinnean/blz203>
[^20]: Gilbertson, M. L. J., White, L. A., & Craft, M. E. (2020). Trade‐offs with telemetry‐derived contact networks for infectious disease studies in wildlife. Methods in Ecology and Evolution. <https://doi.org/10.1111/2041-210x.13355>
[^21]: Bachman, S., Walker, B., Barrios, S., Copeland, A., & Moat, J. (2020). Rapid Least Concern: towards automating Red List assessments. Biodiversity Data Journal, 8. <https://doi.org/10.3897/bdj.8.e47018>


[RSelenium]: https://github.com/ropensci/RSelenium
[chromer]: https://github.com/ropensci/chromer
[qualtRics]: https://github.com/ropensci/qualtRics
[rsnps]: https://github.com/ropensci/rsnps
[rdpla]: https://github.com/ropensci/rdpla
[webchem]: https://github.com/ropensci/webchem
[stplanr]: https://github.com/ropensci/stplanr
[mregions]: https://github.com/ropensci/mregions
[rWBclimate]: https://github.com/ropensci/rWBclimate

[taxize]: https://github.com/ropensci/taxize
[rgbif]: https://github.com/ropensci/rgbif
[CoordinateCleaner]: https://github.com/ropensci/CoordinateCleaner
[hunspell]: https://github.com/ropensci/hunspell
[rotl]: https://github.com/ropensci/rotl
[NLMR]: https://github.com/ropensci/NLMR
[landscapetools]: https://github.com/ropensci/landscapetools
[rfishbase]: https://github.com/ropensci/rfishbase
[rentrez]: https://github.com/ropensci/rentrez
[rnoaa]: https://github.com/ropensci/rnoaa
[brranching]: https://github.com/ropensci/brranching
[rglobi]: https://github.com/ropensci/rglobi
[MODISTools]: https://github.com/ropensci/MODISTools
[spatsoc]: https://github.com/ropensci/spatsoc
[wicket]: https://github.com/ropensci/wicket
[phylogram]: https://github.com/ropensci/phylogram
[magick]: https://github.com/ropensci/magick
[phylogram]: https://github.com/ropensci/phylogram
