---
layout: post
title:  "Update 2017-09-25"
date:   2017-09-25
---

<table border="0" cellpadding="18" cellspacing="0" width="100%" style="min-width: 100% !important;background-color: #CBEAF2">
  <tbody>
  	<tr>
      <td valign="top" style="color: #FFFFFF;font-size: 14px;font-weight: bold;line-height: 250%">
          <div style="text-align: center;"><span style="font-size:27px"><span>rOpenSci HQ</span></span></div>
      </td>
  	</tr>
	</tbody>
</table>

### Community Calls

Video is now up for our latest community call on rOpenSci Software Review and Onboarding.

<https://github.com/ropensci/commcalls/issues/15>

<br><br>

<table border="0" cellpadding="18" cellspacing="0" width="100%" style="min-width: 100% !important;background-color: #B2E1ED">
  <tbody>
  	<tr>
      <td valign="top" style="color: #FFFFFF;font-size: 14px;font-weight: bold;line-height: 250%">
          <div style="text-align: center;"><span style="font-size:27px"><span>Discussion forum</span></span></div>
      </td>
  	</tr>
	</tbody>
</table>

Recently, on our discussion forum <https://discuss.ropensci.org>:

* [How to avoid space-hogging raster tempfiles?](https://discuss.ropensci.org/t/how-to-avoid-space-hogging-raster-tempfiles/864)
* [A research compendium and methylation raw data](https://discuss.ropensci.org/t/a-research-compendium-and-methylation-raw-data/867)
* [Resources for leaning C++/Rcpp?](https://discuss.ropensci.org/t/resources-for-leaning-c-rcpp/877)
* [How do you hire data people in a fair and unbiased way?](https://discuss.ropensci.org/t/how-do-you-hire-data-people-in-a-fair-and-unbiased-way/876)
* [traits::ncbi_byname to DNAbin objects](https://discuss.ropensci.org/t/traits-ncbi-byname-to-dnabin-objects/878)

<br><br>

<table border="0" cellpadding="18" cellspacing="0" width="100%" style="min-width: 100% !important;background-color: #7CCCC8">
  <tbody>
  	<tr>
      <td valign="top" style="color: #FFFFFF;font-size: 14px;font-weight: bold;line-height: 250%">
          <div style="text-align: center;"><span style="font-size:27px"><span>Software</span></span></div>
      </td>
  	</tr>
	</tbody>
</table>

### New packages

* A new package `microdemic` (`v0.1.0`) is [on CRAN](https://cran.rstudio.com/web/packages/jaod). `microdemic` is an R client for the [Microsoft Academic Knowledge](https://academic.microsoft.com/) [API](https://azure.microsoft.com/en-us/services/cognitive-services/academic-knowledge/). Check out the [microdemic README](https://github.com/ropenscilabs/microdemic#microdemic) to get started. [Repository on GitHub][microdemic]

### Releases

* A new version (`v0.5.2`) of `biomartr` is on [CRAN](https://cran.rstudio.com/web/packages/biomartr). See the [NEWS](https://cran.rstudio.com/web/packages/biomartr/news.html) for details. Check out the [biomartr vignettes](https://cran.rstudio.com/web/packages/biomartr/vignettes/) to get started. [Repository on GitHub][biomartr]
* A new version (`v0.3.0`) of `traits` is on [CRAN](https://cran.rstudio.com/web/packages/traits). This version adds new data sources and makes a suite of functions defunct (that were previsouly marked deprecated) that have moved to the [originr][] package. See the [release notes](https://github.com/ropensci/traits/releases/tag/v0.3.0) for details. Check out the [traits vignettes](https://cran.rstudio.com/web/packages/traits/vignettes/) to get started. [Repository on GitHub][traits]
* A new version (`v0.2.10`) of `rglobi` is on [CRAN](https://cran.rstudio.com/web/packages/rglobi). See the [release notes](https://github.com/ropensci/rglobi/releases/tag/v0.2.10) for details. Check out the [vignette](https://cran.rstudio.com/web/packages/rglobi/vignettes/rglobi_vignette.html) to get started. [Repository on GitHub][rglobi]
* A new version (`v1.0.6`) of `GSODR` is on [CRAN](https://cran.rstudio.com/web/packages/GSODR). See the [release notes](https://github.com/ropensci/GSODR/releases/tag/1.0.6) for details. Check out the [vignettes](https://cran.rstudio.com/web/packages/GSODR/vignettes/) to get started. [Repository on GitHub][GSODR]
* A new version (`v1.2`) of `xslt` is on [CRAN](https://cran.rstudio.com/web/packages/xslt). See the [release notes](https://github.com/ropensci/xslt/releases/tag/v1.2) for details. Check out [Jeroen's blog post](https://ropensci.org/blog/blog/2017/01/10/xslt-release) to get started. [Repository on GitHub][xslt]
* A new version (`v0.4.0`) of `lawn` is on [CRAN](https://cran.rstudio.com/web/packages/lawn). This version has updated versions of most of the JS libraries bundled with the package, and other improvements. See the [release notes](https://github.com/ropensci/lawn/releases/tag/v0.4.0) for details. Check out [the vignette](https://cran.rstudio.com/web/packages/lawn/vignettes/lawn_vignette.html) to get started. [Repository on GitHub][lawn]
* A new version (`v1.0.7`) of `lingtypology` is on [CRAN](https://cran.rstudio.com/web/packages/lingtypology). See the [release notes](https://github.com/ropensci/lingtypology/releases/tag/v1.0.7) for details. Check out [the vignette](https://cran.rstudio.com/web/packages/lingtypology/vignettes/lingtypology.html) to get started. [Repository on GitHub][lingtypology]
* A new version (`v0.8.0`) of `elastic` is on [CRAN](https://cran.rstudio.com/web/packages/lawelasticn). This version brings many changes to scrolling, adds new functions, and fixes some bugs. See the [release notes](https://github.com/ropensci/elastic/releases/tag/v0.8.0) for details. Check out [the README](https://github.com/ropensci/elastic#elastic) to get started. [Repository on GitHub][elastic]

<br><br>

<table border="0" cellpadding="18" cellspacing="0" width="100%" style="min-width: 100% !important;background-color: #64B7A9">
  <tbody>
  	<tr>
      <td valign="top" style="color: #FFFFFF;font-size: 14px;font-weight: bold;line-height: 250%">
          <div style="text-align: center;"><span style="font-size:27px"><span>Software Review</span></span></div>
      </td>
  	</tr>
	</tbody>
</table>
<br>
We accept community contributed packages via our onboarding system - a software review system, sorta like scholarly paper review, but way better. We'll highlight new packages here that have come through this system. A huge thanks to our reviewers, who do a lot of work reviewing (see the [blog post on our review system](https://ropensci.org/blog/2016/03/28/software-review)),
and the authors of the packages!

If you want to be a reviewer we have a new [form](https://ropensci.org/onboarding/) you can fill out, and we'll ping you when there's a submission that fits in your area of expertise.

The following package was recently submitted to our onboarding process:

* [tidyhydat][] > Extract and Tidy Canadian Hydrometric Data
    * Author: [Sam Albers](https://github.com/boshek)
    * Issue: [ropensci/onboarding#152](https://github.com/ropensci/onboarding/issues/152)
    * Reviewers: not assigned yet
* [EndoMineR][] > Functions to mine endoscopic and associated pathology datasets
    * Author: [Sebastian Zeki](https://github.com/sebastiz)
    * Issue: [ropensci/onboarding#153](https://github.com/ropensci/onboarding/issues/153)
    * Reviewers: not assigned yet


<br><br>


<table border="0" cellpadding="18" cellspacing="0" width="100%" style="min-width: 100% !important;background-color: #8CD4E0">
  <tbody>
  	<tr>
      <td valign="top" style="color: #FFFFFF;font-size: 14px;font-weight: bold;line-height: 250%">
          <div style="text-align: center;"><span style="font-size:27px"><span>On the blog</span></span></div>
      </td>
  	</tr>
	</tbody>
</table>
<br><br>
We've had one blog post on our main blog in the past two weeks:

* [Accessing patent data with the patentsview package](https://ropensci.org/blog/blog/2017/09/19/patentsview) by [Chris Baker](https://github.com/crew102)

<br><br>



<table border="0" cellpadding="18" cellspacing="0" width="100%" style="min-width: 100% !important;background-color: #79CAC7">
  <tbody>
  	<tr>
      <td valign="top" style="color: #FFFFFF;font-size: 14px;font-weight: bold;line-height: 250%">
          <div style="text-align: center;"><span style="font-size:27px"><span>Use cases</span></span></div>
      </td>
  	</tr>
	</tbody>
</table>
<br>

The following 3 works use/cite rOpenSci or rOpenSci software:

* Olson _et al_. used [taxize][] in their paper [Challenging a bioinformatic tool’s ability to detect microbial contaminants using in silico whole genome sequencing data](https://doi.org/10.7717/peerj.3729) [^1]
* Reznik _et al_. used [taxize][] in their paper [Genome-scale architecture of small molecule regulatory networks and the fundamental trade-off between regulation and enzymatic activity](https://doi.org/10.1016/j.celrep.2017.08.066) [^2]
*  Cardoso _et al_. used [rgbif][] in their paper [Amazon plant diversity revealed by a taxonomically verified species list](https://doi.org/10.1073/pnas.1706756114) [^3]


<br><br>

<hr style="display: block; height: 1px; border: 0; border-top: 3px solid #7CCCC8; margin: 1em 0; padding: 0; ">

<br><br>


## Submit news to rOpenSci

Reach out to us with your news suggestions by sending us an email <mailto:info@ropensci.org> or by
contacting us on Twitter [@ropensci](https://twitter.com/ropensci)

<br>

## Keep up with rOpenSci

There are a number of ways to keep up with what rOpenSci is doing:

* Mailing list: Sign up with an email address to get this newsletter sent to your inbox -> [ropensci.org/#subscribe](https://ropensci.org/#subscribe)
* Alternatively, you can subscribe to this newsletter via our XML feed at <https://ropensci.github.io/biweekly/feed.xml> or our JSON feed at <https://ropensci.github.io/biweekly/feed.json>
* rOpenSci on Twitter: we're [@ropensci](https://twitter.com/ropensci)
* The rOpenSci blog at [ropensci.org/blog](https://ropensci.org/blog) - you can subscribe in any RSS aggregator, or manually via [https://ropensci.org/feed.xml](https://ropensci.org/feed.xml). We also announce new blog posts on our Twitter account.

<br>

### Footnotes

[^1]: Olson, N. D., Zook, J. M., Morrow, J. B., & Lin, N. J. (2017). Challenging a bioinformatic tool’s ability to detect microbial contaminants using in silico whole genome sequencing data. PeerJ, 5, e3729. <https://doi.org/10.7717/peerj.3729>
[^2]: Reznik, E., Christodoulou, D., Goldford, J. E., Briars, E., Sauer, U., Segrè, D., & Noor, E. (2017). Genome-Scale Architecture of Small Molecule Regulatory Networks and the Fundamental Trade-Off between Regulation and Enzymatic Activity. Cell Reports, 20(11), 2666–2677. <https://doi.org/10.1016/j.celrep.2017.08.066>
[^3]: Cardoso, D., Särkinen, T., Alexander, S., Amorim, A. M., Bittrich, V., Celis, M., … Forzza, R. C. (2017). Amazon plant diversity revealed by a taxonomically verified species list. Proceedings of the National Academy of Sciences, 201706756. <https://doi.org/10.1073/pnas.1706756114>


[taxize]: https://github.com/ropensci/taxize
[microdemic]: https://github.com/ropenscilabs/microdemic
[geojsonio]: https://github.com/ropensci/geojsonio
[biomartr]: https://github.com/ropensci/biomartr
[traits]: https://github.com/ropenscilabs/traits
[writexl]: https://github.com/ropensci/writexl
[rglobi]: https://github.com/ropensci/rglobi
[GSODR]: https://github.com/ropensci/GSODR
[xslt]: https://github.com/ropensci/xslt
[lawn]: https://github.com/ropensci/lawn
[lingtypology]: https://github.com/ropensci/lingtypology
[elastic]: https://github.com/ropensci/elastic
[tidyhydat]: https://github.com/bcgov/tidyhydat
[EndoMineR]: https://github.com/sebastiz/EndoMineR
[rgbif]: https://github.com/ropensci/rgbif
