---
layout: post
title:  "2019-07-22"
date:   2019-07-22
slug:   "2019-07-22"
---

## rOpenSci HQ

* [Recording and notes](https://ropensci.org/commcalls/2019-06-28/) and [public discussion](https://discuss.ropensci.org/t/community-call-managing-multilingual-communities/1722) are up from our latest [community call on Involving Multilingual Communities](https://ropensci.org/blog/2019/06/13/commcall-jun2019/).
* Join us for our next community call [Reproducible Research with R](https://ropensci.org/blog/2019/07/11/commcall-jul2019/) on July 30th, 2019. Speakers are [Ben Marwick](https://ropensci.org/authors/ben-marwick/), [Anna Krystalli](https://ropensci.org/authors/anna-krystalli/), and [Karthik Ram](https://ropensci.org/authors/karthik-ram/).
* [rOpenSci Announces $678K Award from the Sloan Foundation to Expand Software Peer Review](https://ropensci.org/blog/2019/07/15/expanding-software-review/) & [rOpenSci Hiring for New Position in Statistical Software Testing and Peer Review](https://ropensci.org/blog/2019/07/18/ropensci-hiring/)

<br>

## Software 📦

CRAN: <img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px">
GitHub: <img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px">

### New Versions

* A new version (`v7.5.2`) of `drake` is on CRAN - a pipeline toolkit for reproducible computation at scale. See the [release notes](https://github.com/ropensci/drake/releases/tag/v7.5.2) for changes. Checkout the [docs](https://ropensci.github.io/drake/) to get started. {{ "drake" | image_cran }} {{ "drake" | image_github }}
> remove accidental creation of a dot file
* A new version (`v0.6.4`/`v0.6.5`) of `rerddap` is on CRAN - General Purpose Client for ERDDAP Servers. See the [release notes](https://github.com/ropensci/rerddap/releases/tag/v0.6.5) for changes. Checkout the [vignettes](https://cran.r-project.org/web/packages/rerddap/vignettes/) to get started. {{ "rerddap" | image_cran }} {{ "rerddap" | image_github }}
> fix internal http request error handler; fix a test
* A new version (`v1.0.4`) of `available` is on CRAN - check if the title of a package is available. See the [release notes](https://github.com/ropenscilabs/available/blob/master/NEWS.md) for changes. Checkout the [README](https://github.com/ropenscilabs/available#available) to get started. {{ "available" | image_cran }} {{ "available" | image_github: 'ropenscilabs' }}
> tests requiring network access are not skipped on CRAN
* A new version (`v1.2`) of `cld3` is on CRAN - Bindings to Google's Compact Language Detector 3. See the [release notes](https://github.com/ropensci/cld3/blob/master/NEWS) for changes. Checkout the [docs](https://docs.ropensci.org/cld3/) to get started. {{ "cld3" | image_cran }} {{ "cld3" | image_github }}
> update to protobuf 3.8 for windows; suppress warnings about pragmas in autogen protobuf headers
* A new version (`v0.2.7`) of `fingertipsR` is on CRAN - Fingertips data for public health <http://fingertips.phe.org.uk/>. See the [release notes](https://github.com/ropensci/fingertipsR/blob/master/NEWS.md) for changes. Checkout the [vignettes](https://cran.rstudio.com/web/packages/fingertipsR/vignettes/) to get started. {{ "fingertipsR" | image_cran }} {{ "fingertipsR" | image_github }}
> add option `AreaTypeId="All` in `fingertips_data()`; added param to `area_types()`; change to `category_types()`; add http request retry
* A new version (`v0.2.0`) of `vitae` is on CRAN - R Markdown Resumes and CVs. See the [release notes](https://github.com/ropenscilabs/vitae/releases/tag/v0.2.0) for changes. Checkout the [docs](https://docs.ropensci.org/vitae/) to get started. {{ "vitae" | image_cran }} {{ "vitae" | image_github: 'ropenscilabs' }}
> theme support for moderncv; other template changes; breaking change to `bibliography_entries`
* A new version (`v2.0.2`) of `ijtiff` is on CRAN - Comprehensive TIFF I/O with full support for ImageJ TIFF files. See the [release notes](https://github.com/ropensci/ijtiff/blob/master/NEWS.md) for changes. Checkout the [docs](https://ropensci.github.io/ijtiff) to get started. {{ "ijtiff" | image_cran }} {{ "ijtiff" | image_github }}
> bug fixes
* A new version (`v0.6.0`) of `bomrang` is on CRAN - Australian government Bureau of Meteorology (BOM) data. See the [release notes](https://github.com/ropensci/bomrang/blob/master/NEWS.md) for changes. Checkout the [docs](https://docs.ropensci.org/bomrang/) to get started. {{ "bomrang" | image_cran }} {{ "bomrang" | image_github }}
> many bug fixes
* A new version (`v0.26.1`) of `git2r` is on CRAN - an interface to the libgit2 library, a pure C implementation of the Git core methods. See the [release notes](https://github.com/ropensci/git2r/releases/tag/v0.26.1) for changes. Checkout the [README](https://github.com/ropensci/git2r#introduction) to get started. {{ "git2r" | image_cran }} {{ "git2r" | image_github }}
> fixed the broken build on Solaris on CRAN
* A new version (`v0.4.2`) of `traits` is on CRAN - species trait data from around the web. See the [release notes](https://github.com/ropensci/traits/releases/tag/v0.4.2) for changes. Checkout the [README](https://github.com/ropensci/traits#traits) to get started. {{ "traits" | image_cran }} {{ "traits" | image_github }}
> minor betydb change; `taxa_search()` breaking change (Traitbank service completely changed)
* A new version (`v0.8.0`) of `crul` is on CRAN - an R6 based HTTP client for R made for developers. See the [release notes](https://github.com/ropensci/crul/releases/tag/v0.8.0) for changes. Checkout the [docs](https://ropensci.github.io/crul/) to get started. {{ "crul" | image_cran }} {{ "crul" | image_github }}
> can now pass params through `parse()` to `iconv()`; use httpcode pkg instead of fauxpas inside `ok()`
* A new version (`v0.2.0`) of `grainchanger` is on CRAN - Data aggregation methods for raster data. See the [release notes](https://github.com/ropensci/grainchanger/releases/tag/v0.2.0) for changes. Checkout the [docs](https://docs.ropensci.org/grainchanger/) to get started. {{ "grainchanger" | image_cran }} {{ "grainchanger" | image_github }}
* A new version (`v0.9.8`) of `taxize` is on CRAN - taxonomic toolbelt for R. See the [release notes](https://github.com/ropensci/taxize/releases/tag/v0.9.8) for changes. Checkout the [taxize book](https://ropenscilabs.github.io/taxize-book/) to get started. {{ "taxize" | image_cran }} {{ "taxize" | image_github }}
> new fxn `taxize_options`; all `get_*` fxns gain ability to track progress and restart in case of interruption - & prettier progress and summary info; bug fixes
* A new version (`v0.4.0`) of `worrms` is on CRAN - client for [World Register of Marine Species](http://www.marinespecies.org/). See the [release notes](https://github.com/ropensci/worrms/releases/tag/v0.4.0) for changes. Checkout the [vignette](https://cran.rstudio.com/web/packages/worrms/vignettes/worrms.html) to get started. {{ "worrms" | image_cran }} {{ "worrms" | image_github }}
> new fxns `wm_ranks_id()`/`wm_ranks_name()` for getting taxonomic ranks by rank identifier or rank name; new fxn `wm_records_rank()` for getting AphiaRecords for a given rank id
* A new version (`v0.9.0`) of `bold` is on CRAN - interface to Bold Systems (<http://www.boldsystems.org/>) API. See the [release notes](https://github.com/ropensci/bold/releases/tag/v0.9.0) for changes. Checkout the [vignette](https://cran.rstudio.com/web/packages/bold/vignettes/bold.html) to get started. {{ "bold" | image_cran }} {{ "bold" | image_github }}
> allow curl options to be passed into `bold_identify_parents()`; bug fix in `bold_identify()`
* A new version (`v1.3.0`) of `fulltext` is on CRAN - full text of scholarly articles across many data sources. See the [release notes](https://github.com/ropensci/fulltext/releases/tag/v1.3.0) for changes. Checkout the [fulltext book](https://ropenscilabs.github.io/fulltext-book/) to get started. {{ "fulltext" | image_cran }} {{ "fulltext" | image_github }}
> `ft_get()` gains new data source: ScienceDirect; `ft_get()` performance improvements; curl options can now be passed through nearly all fxns
* A new version (`v3.0.4`) of `rfishbase` is on CRAN - interface to Fishbase data. See the [release notes](https://github.com/ropensci/rfishbase/blob/master/NEWS.md) for changes. Checkout the [docs](http://ropensci.github.io/rfishbase/) to get started. {{ "rfishbase" | image_cran }} {{ "rfishbase" | image_github }}
> use latest data version
* A new version (`v1.1.0`) of `jsonvalidate` is on CRAN - Validate JSON. See the [release notes](https://github.com/ropensci/jsonvalidate/releases/tag/v1.1.0) for changes. Checkout the [docs](http://ropensci.github.io/jsonvalidate/) to get started. {{ "jsonvalidate" | image_cran }} {{ "jsonvalidate" | image_github }}

<br><br>

## Software Review ✔

We accept community contributed packages via our software review system - an open software review system, sorta like scholarly paper review, but way better. We'll highlight newly onboarded packages here. A huge thanks to our reviewers, who do a lot of work reviewing (see the [blog post on our review system](https://ropensci.org/blog/2016/03/28/software-review)),
and the authors of the packages!

If you want to be a reviewer fill out [this short form](https://ropensci.org/onboarding/), and we'll ping you when there's a submission that fits in your area of expertise.

The following two packages recently went through our software-review process and have been approved:

* [grainchanger][] > Moving-Window and Direct Data Aggregation
    * Author: [Laura Jane Graham](https://github.com/laurajanegraham)
    * Issue: [ropensci/onboarding#289](https://github.com/ropensci/onboarding/issues/289)
    * Reviewers: not yet assigned
        * [Max Joseph](https://github.com/mbjoseph)
        * [John Baumgartner](https://github.com/johnbaums)
* [popler][] > Browse and query the popler database
    * Author: [Aldo Compagnoni](https://github.com/AldoCompagnoni)
    * Issue: [ropensci/software-review#254](https://github.com/ropensci/software-review/issues/254)
    * Reviewers:
        * [Corinna Gries](https://github.com/cgries)
        * [Ben Bond-Lamberty](https://github.com/bpbond)


<br><br>

## On the blog

[Stefanie Butland](https://ropensci.org/authors/stefanie-butland/) wrote about [folks talking about rOpenSci projects at useR in Toulouse, France](https://ropensci.org/blog/2019/07/08/user2019/). The conference already happened; slides are linked from the [talk schedule page](http://www.user2019.fr/talk_schedule/), and video of talks may be up later.

<br>

[Laura Graham](https://ropensci.org/authors/laura-graham/) wrote about her R package that recently went through software review [Aggregating spatial data with the grainchanger package](https://ropensci.org/blog/2019/07/16/grainchanger/). [grainchanger](https://docs.ropensci.org/grainchanger/) facilitates spatial data aggregation.

<img src="../assets/img/grainchainger-logo.png" width="200">


<br><br>


## Use Cases

The following 20 works use/cite rOpenSci software:

* Steidinger et al. cited [taxize][] in their paper [Climatic controls of decomposition drive the global biogeography of forest-tree symbioses](https://doi.org/10.1038/s41586-019-1128-0) [^1]
* Mitchell used [RSelenium][] in their thesis [Computational Tools for the Untargeted Assignment of FT-MS Metabolomics Datasets](https://uknowledge.uky.edu/cgi/viewcontent.cgi?article=1042&context=biochem_etds) [^2]
* Bagley et al. used [taxize][] in their paper [High-throughput environmental DNA analysis informs a biological assessment of an urban stream](https://doi.org/10.1016/j.ecolind.2019.04.088) [^3]
* Foisy et al. used [taxize][] in their paper [Do latex and resin canals spur plant diversification? Re‐examining a classic example of escape and radiate coevolution](https://doi.org/10.1111/1365-2745.13203) [^4]
* Ogłuszka et al. used [plotly][] in their paper [Evaluate Cutpoints: Adaptable continuous data distribution system for determining survival in Kaplan-Meier estimator](https://doi.org/10.1016/j.cmpb.2019.05.023) [^5]
* Westmeijer et al. used [rgbif][] in their paper [Mechanistic niche modelling to identify favorable growth sites of temperate macroalgae](https://doi.org/10.1016/j.algal.2019.101529) [^6]
* Kortz et al. used [plotly][] in their paper [Increases in local richness (α-diversity) following invasion are offset by biotic homogenization in a biodiversity hotspot](https://doi.org/10.1098/rsbl.2019.0133) [^7]
* Pérez-Palma et al. used [plotly][] in their paper [Simple ClinVar: an interactive web server to explore and retrieve gene and disease variants aggregated in ClinVar database](https://doi.org/10.1093/nar/gkz411) [^8]
* Green et al. used [hunspell][] and [cld2][] in their paper [What is the best method of family planning for me?: a text mining analysis of messages between users and agents of a digital health service in Kenya](https://doi.org/10.12688/gatesopenres.12999.1) [^9]
* Mihalitsis & Bellwood used [rotl][] in their research [Morphological and functional diversity of piscivorous fishes on coral reefs](https://doi.org/10.1007/s00338-019-01820-w) [^10]
* Kannenberg et al. used [MODIStools][] in their paper [Linking drought legacy effects across scales: From leaves to tree rings to ecosystems](https://doi.org/10.1111/gcb.14710) [^11]
* Aspillaga et al. used [plotly][] in their paper [Modelling the three‐dimensional space use of aquatic animals combining topography and Eulerian telemetry data](https://doi.org/10.1111/2041-210x.13232) [^12]
* Gibson et al. used [plotly][] in their paper [Gut microbiome differences between wild and captive black rhinoceros – implications for rhino health](https://doi.org/10.1038/s41598-019-43875-3) [^13]
* Sinval et al. used [skimr][] in their paper [The Quality of Work Life Scale: Validity Evidence from Brazil and Portugal](https://doi.org/10.1007/s11482-019-09730-3) [^14]
* Iserman et al. used [hunspell][] in their conference paper [Dictionaries and Decision Trees for the 2019 CLPsych Shared Task](https://www.aclweb.org/anthology/W19-3025) [^15]
* Goodman et al. used [rfishbase][] in their paper [The relationship between geographic range extent, sea surface temperature and adult traits in coastal temperate fishes](https://doi.org/10.1111/jbi.13595) [^16]
* Fioravanti et al. used [RSelenium][] in their paper [A new homogenized daily data set for temperature variability assessment in Italy](https://doi.org/10.1002/joc.6177) [^17]
* Logemann et al. used [magick][] in their paper [Using the HPTLC-bioluminescence bacteria assay for the determination of acute toxicities in marine sediments and its eligibility as a monitoring assessment tool](https://doi.org/10.1016/j.chemosphere.2019.05.246) [^18]
* Al-Mustanjid used [rentrez][] in their paper [Design of a common pathway drug for all types of cardiovascular diseases: A network biology approach](http://www.iaees.org/publications/journals/nb/articles/2019-9(2)/design-of-a-common-pathway-drug-for-cardiovascular-diseases.pdf) [^19]
* Wambui & Musenge used [MODIStools][] in their paper [A space-time analysis of recurrent malnutrition-related hospitalisations in Kilifi, Kenya for children under-5 years](https://doi.org/10.1186/s40795-019-0296-5) [^20]

<br><br>

## Call For Contributors

Part of the mission of rOpenSci is making sustainable software that users can rely on. Some software maintainers need to give up maintenance due to a variety of circumstances. When that happens we try to find new maintainers. Checkout our [Contributing Guide](https://devguide.ropensci.org/contributingguide.html) for why and how to contribute to rOpenSci.

We've had six recent examples of maintainer transitions within rOpenSci:

- [RSelenium][]: now maintained by [Ju Kim](https://github.com/juyeongkim)
- [chromer][]: now maintained by [Paula Andrea](https://github.com/orchid00)
- [qualtRics][]: now maintained by [Julia Silge](https://github.com/juliasilge)
- [rsnps][]: now maintained by [Julia Gustavsen](https://github.com/jooolia) and [Sina Rüeger](https://github.com/sinarueeger)
- [rdpla][]: now maintained by [Alyssa Columbus](https://github.com/acolum)
- [webchem][]: now maintained by [Erik Sapper](https://github.com/eriksapper)

We don't have any packages looking for new maintainers - if you are a maintainer and want to hand over that role to someone else get in touch with us.

<br><br>

<hr style="display: block; height: 1px; border: 0; border-top: 3px solid #7CCCC8; margin: 1em 0; padding: 0; ">

<br><br>

### Keep up with rOpenSci

* Mailing list: Sign up with an email address to get this newsletter sent to your inbox -> [ropensci.org/#subscribe](https://ropensci.org/#subscribe)
* Alternatively, you can subscribe to this newsletter via our XML feed at <https://news.ropensci.org/feed.xml> or our JSON feed at <https://news.ropensci.org/feed.json>
* rOpenSci on Twitter: [@ropensci](https://twitter.com/ropensci)
* The rOpenSci blog at [ropensci.org/blog](https://ropensci.org/blog) - you can subscribe in any RSS aggregator, or manually via <https://ropensci.org/feed.xml>. We also announce new blog posts on our Twitter account.

<br>

#### Footnotes

[^1]: Steidinger, B. S., Crowther, T. W., Liang, J., Van Nuland, M. E., Werner, G. D. A., … Peay, K. G. (2019). Climatic controls of decomposition drive the global biogeography of forest-tree symbioses. Nature, 569(7756), 404–408. <https://doi.org/10.1038/s41586-019-1128-0>
[^2]: Mitchell, J. M. (2019). Computational Tools for the Untargeted Assignment of FT-MS Metabolomics Datasets. Dissertation. <https://uknowledge.uky.edu/cgi/viewcontent.cgi?article=1042&context=biochem_etds>
[^3]: Bagley, M., Pilgrim, E., Knapp, M., Yoder, C., Santo Domingo, J., & Banerji, A. (2019). High-throughput environmental DNA analysis informs a biological assessment of an urban stream. Ecological Indicators, 104, 378–389. <https://doi.org/10.1016/j.ecolind.2019.04.088>
[^4]: Foisy, M. R., Albert, L. P., Hughes, D. W. W., & Weber, M. G. (2019). Do latex and resin canals spur plant diversification? Re‐examining a classic example of escape and radiate coevolution. Journal of Ecology. <https://doi.org/10.1111/1365-2745.13203>
[^5]: Ogłuszka, M., Orzechowska, M., Jędroszka, D., Witas, P., & Bednarek, A. K. (2019). Evaluate Cutpoints: Adaptable continuous data distribution system for determining survival in Kaplan-Meier estimator. Computer Methods and Programs in Biomedicine, 177, 133–139. <https://doi.org/10.1016/j.cmpb.2019.05.023>
[^6]: Westmeijer, G., Everaert, G., Pirlet, H., De Clerck, O., & Vandegehuchte, M. B. (2019). Mechanistic niche modelling to identify favorable growth sites of temperate macroalgae. Algal Research, 41, 101529. <https://doi.org/10.1016/j.algal.2019.101529>
[^7]: Kortz, A. R., & Magurran, A. E. (2019). Increases in local richness (α-diversity) following invasion are offset by biotic homogenization in a biodiversity hotspot. Biology Letters, 15(5), 20190133. <https://doi.org/10.1098/rsbl.2019.0133>
[^8]: Pérez-Palma, E., Gramm, M., Nürnberg, P., May, P., & Lal, D. (2019). Simple ClinVar: an interactive web server to explore and retrieve gene and disease variants aggregated in ClinVar database. Nucleic Acids Research. <https://doi.org/10.1093/nar/gkz411>
[^9]: Green, E. P., Whitcomb, A., Kahumbura, C., Rosen, J. G., Goyal, S., Achieng, D., & Bellows, B. (2019). What is the best method of family planning for me?: a text mining analysis of messages between users and agents of a digital health service in Kenya. Gates Open Research, 3, 1475. <https://doi.org/10.12688/gatesopenres.12999.1>
[^10]: Mihalitsis, M., & Bellwood, D. R. (2019). Morphological and functional diversity of piscivorous fishes on coral reefs. Coral Reefs. <https://doi.org/10.1007/s00338-019-01820-w>
[^11]: Kannenberg, S. A., Novick, K. A., Alexander, M. R., Maxwell, J. T., Moore, D. J. P., Phillips, R. P., & Anderegg, W. R. L. (2019). Linking drought legacy effects across scales: From leaves to tree rings to ecosystems. Global Change Biology. <https://doi.org/10.1111/gcb.14710>
[^12]: Aspillaga, E., Safi, K., Hereu, B., & Bartumeus, F. (2019). Modelling the three‐dimensional space use of aquatic animals combining topography and Eulerian telemetry data. Methods in Ecology and Evolution. <https://doi.org/10.1111/2041-210x.13232>
[^13]: Gibson, K. M., Nguyen, B. N., Neumann, L. M., Miller, M., Buss, P., Daniels, S., … Pukazhenthi, B. (2019). Gut microbiome differences between wild and captive black rhinoceros – implications for rhino health. Scientific Reports, 9(1). <https://doi.org/10.1038/s41598-019-43875-3>
[^14]: Sinval, J., Sirgy, M. J., Lee, D.-J., & Marôco, J. (2019). The Quality of Work Life Scale: Validity Evidence from Brazil and Portugal. Applied Research in Quality of Life. <https://doi.org/10.1007/s11482-019-09730-3>
[^15]: Iserman, M., Nalabandian, T., & Ireland, M. (2019, June). Dictionaries and Decision Trees for the 2019 CLPsych Shared Task. In Proceedings of the Sixth Workshop on Computational Linguistics and Clinical Psychology (pp. 188-194). <https://www.aclweb.org/anthology/W19-3025>
[^16]: Goodman, M. C., Hannah, S. M., & Ruttenberg, B. I. (2019). The relationship between geographic range extent, sea surface temperature and adult traits in coastal temperate fishes. Journal of Biogeography. <https://doi.org/10.1111/jbi.13595>
[^17]: Fioravanti, G., Piervitali, E., & Desiato, F. (2019). A new homogenized daily data set for temperature variability assessment in Italy. International Journal of Climatology. <https://doi.org/10.1002/joc.6177>
[^18]: Logemann, A., Schafberg, M., & Brockmeyer, B. (2019). Using the HPTLC-bioluminescence bacteria assay for the determination of acute toxicities in marine sediments and its eligibility as a monitoring assessment tool. Chemosphere. <https://doi.org/10.1016/j.chemosphere.2019.05.246>
[^19]: Al-Mustanjid, A. (2019). Design of a common pathway drug for all types of cardiovascular diseases: A network biology approach. Network Biology, 9(2), 28. <http://www.iaees.org/publications/journals/nb/articles/2019-9(2)/design-of-a-common-pathway-drug-for-cardiovascular-diseases.pdf>
[^20]: Wambui, K. M., & Musenge, E. (2019). A space-time analysis of recurrent malnutrition-related hospitalisations in Kilifi, Kenya for children under-5 years. BMC Nutrition, 5(1). <https://doi.org/10.1186/s40795-019-0296-5>



[RSelenium]: https://github.com/ropensci/RSelenium
[chromer]: https://github.com/ropensci/chromer
[qualtRics]: https://github.com/ropensci/qualtRics
[rsnps]: https://github.com/ropensci/rsnps
[rdpla]: https://github.com/ropensci/rdpla
[webchem]: https://github.com/ropensci/webchem
[taxize]: https://github.com/ropensci/taxize
[tokenizers]: https://github.com/ropensci/tokenizers
[clifro]: https://github.com/ropensci/clifro
[writexl]: https://github.com/ropensci/writexl
[magick]: https://github.com/ropensci/magick
[nlrx]: https://github.com/ropensci/nlrx
[hydroscoper]: https://github.com/ropensci/hydroscoper
[lingtypology]: https://github.com/ropensci/lingtypology
[grainchanger]: https://github.com/ropensci/grainchanger
[popler]: https://github.com/AldoCompagnoni/popler
[plotly]: https://github.com/ropensci/plotly
[rgbif]: https://github.com/ropensci/rgbif
[hunspell]: https://github.com/ropensci/hunspell
[cld2]: https://github.com/ropensci/cld2
[rotl]: https://github.com/ropensci/rotl
[MODIStools]: https://github.com/ropensci/MODIStools
[rfishbase]: https://github.com/ropensci/rfishbase
[skimr]: https://github.com/ropensci/skimr
[rentrez]: https://github.com/ropensci/rentrez

