---
layout: post
title:  "2019-02-19"
date:   2019-02-19
slug:   "2019-02-19"
---

## Software üì¶

CRAN: <img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px">
GitHub: <img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px">

### New packages

* The first version (`v0.1.0`) of `rcitoid` is on CRAN - Client for Citoid, Wikipedia's citation data. Checkout the [README](https://github.com/ropenscilabs/rcitoid#rcitoid) to get started. {{ "rcitoid" | image_cran }} {{ "rcitoid" | image_github: 'ropenscilabs' }}

### New Versions

* A new version (`v1.0.7`) of `nasapower` is on CRAN - NASA POWER API Client. See the [release notes](https://github.com/ropensci/nasapower/blob/master/NEWS.md) for changes. Checkout the [docs](https://ropensci.github.io/nasapower/) to get started. {{ "nasapower" | image_cran }} {{ "nasapower" | image_github }}
> minor changes
* A new version (`v0.2-7`) of `dbhydroR` is on CRAN - DBHYDRO hydrologic and water quality data. See the [release notes](https://github.com/ropensci/dbhydroR/releases/tag/v0.2-7) for changes. Checkout the [docs](https://ropensci.github.io/dbhydroR/) to get started. {{ "dbhydroR" | image_cran }} {{ "dbhydroR" | image_github }}
> fixed critical bug in `get_hydro` causing data parsing failure in all cases
* A new version (`v3.0.7`) of `rotl` is on CRAN - an interface to the Open Tree of Life API. See the [release notes](https://github.com/ropensci/rotl/releases/tag/v3.0.7) for changes. Checkout the [vignettes](https://cran.rstudio.com/web/packages/rotl/vignettes/) to get started. {{ "rotl" | image_cran }} {{ "rotl" | image_github }}
> update vignette to address change to OTL TNRS endpoint
* A new version (`v0.2.0`) of `stats19` is on CRAN - Work with Open Road Traffic Casualty Data from Great Britain. See the [release notes](https://github.com/ropensci/stats19/blob/master/NEWS.md) for changes. Checkout the [docs](https://itsleeds.github.io/stats19/) to get started. {{ "stats19" | image_cran }} {{ "stats19" | image_github }}
* A new version (`v0.5.3`) of `visdat` is on CRAN - create preliminary exploratory data visualisations of an entire dataset to identify problems or unexpected features using ggplot2. See the [release notes](https://github.com/ropensci/visdat/blob/master/NEWS.md) for changes. Checkout the [docs](http://visdat.njtierney.com/) to get started. {{ "visdat" | image_cran }} {{ "visdat" | image_github }}
> updates to `vis_cor()`; use `glue` instead of `paste`
* A new version (`v0.2.6`) of `vcr` is on CRAN - record HTTP calls to disk for test caching. See the [release notes](https://github.com/ropensci/vcr/releases/tag/v0.2.6) for changes. Checkout the [HTTP testing book](https://ropensci.github.io/http-testing-book/) to get started. {{ "vcr" | image_cran }} {{ "vcr" | image_github }}
> new vcr setup fxn `use_vcr()`; support all recording modes!; many fixes to support httr integration; `use_cassette()` error blocks no longer include secrets
* A new version (`v0.3.4`) of `webmockr` is on CRAN - stubbing and setting expectations on HTTP requests. See the [release notes](https://github.com/ropensci/webmockr/releases/tag/v0.3.4) for changes. Checkout the [HTTP testing book](https://ropensci.github.io/http-testing-book/) to get started. {{ "webmockr" | image_cran }} {{ "webmockr" | image_github }}
> `to_return_()`/`wi_th_()` defunct; `to_return()` gains `.list` param; bug fixes üêû
* A new version (`v0.3.0`) of `geojsonlint` is on CRAN - Tools for Validating GeoJSON. See the [release notes](https://github.com/ropensci/geojsonlint/releases/tag/v0.3.0) for changes. Checkout the [vignette](https://cran.r-project.org/web/packages/geojsonlint/vignettes/geojsonlint_vignette.html) to get started. {{ "geojsonlint" | image_cran }} {{ "geojsonlint" | image_github }}
> update JS library geojsonhint to `v2.1`; use `crul` for HTTP
* A new version (`v0.0.10`) of `piggyback` is on CRAN - Managing larger data on a GitHub repository. See the [release notes](https://github.com/ropensci/piggyback/releases/tag/v0.0.10) for changes. Checkout the [docs](https://ropensci.github.io/piggyback/) to get started. {{ "piggyback" | image_cran }} {{ "piggyback" | image_github }}
> improve interface for overwrite files on upload; bug fixes
* A new version (`v2.1`) of `jsonld` is on CRAN - JSON for linking data. See the [release notes](https://github.com/ropensci/jsonld/blob/master/NEWS) for changes. Checkout the [README](https://github.com/ropensci/jsonld/) to get started. {{ "jsonld" | image_cran }} {{ "jsonld" | image_github }}
> refactor async code to work with V8 package changes

<br><br>

## Software Review ‚úî

We accept community contributed packages via our onboarding system - an open software review system, sorta like scholarly paper review, but way better. We'll highlight newly onboarded packages here. A huge thanks to our reviewers, who do a lot of work reviewing (see the [blog post on our review system](https://ropensci.org/blog/2016/03/28/software-review)),
and the authors of the packages!

If you want to be a reviewer fill out [this short form](https://ropensci.org/onboarding/), and we'll ping you when there's a submission that fits in your area of expertise.

The following package recently went through our onboarding process and has been approved:

* [emld][] > Ecological Metadata as Linked Data
    * Author: [Carl Boettiger](https://github.com/cboettig)
    * Issue: [ropensci/onboarding#269](https://github.com/ropensci/onboarding/issues/269)
     * Reviewers:
        * [Kelly Hondula](https://github.com/khondula)
        * [Peter Slaughter](https://github.com/gothub)

The following two packagess were recently submitted:

* [CytoRSuite][] > Compensation, Gating & Visualisation Toolkit for Analysis of Flow Cytometry Data
    * Author: [Dillon Hammill](https://github.com/DillonHammill)
    * Issue: [ropensci/onboarding#281](https://github.com/ropensci/onboarding/issues/281)
    * Reviewers: not yet assigned
* [outsider][] > Install and run programs, outside of R, inside of R
    * Author: [Dom Bennett](https://github.com/DomBennett)
    * Issue: [ropensci/onboarding#282](https://github.com/ropensci/onboarding/issues/282)
    * Reviewers: not yet assigned

<br><br>


## On the blog

### Community

[Dan Sholler](https://ropensci.org/authors/dan-sholler/) wrote a summary of his recent [rOpenSci Community Call](https://ropensci.org/commcalls/2018-12-18/): [Community Call Follow-up - Governance of Open Source Research Software Organizations](https://ropensci.org/blog/2019/02/12/governance/).

### Tech notes

[Jeroen Ooms](https://ropensci.org/authors/jeroen-ooms/) wrote about [A Major Upgrade of the V8 package](https://ropensci.org/technotes/2019/02/11/v8-20/) in his recent rOpenSci tech note. The [V8][] package provides an embedded JavaScript engine that can be used inside of R.

<br><br>


## Use Cases

The following 14 works use/cite rOpenSci software:

* Lee, et al. used [RSelenium][] and [hunspell][] in their paper [Investigating the association between mating-relevant self-concepts and mate preferences through a data-driven analysis of online personal descriptions](https://doi.org/10.31234/osf.io/38zef) [^1]
* Lu & Hedin used [rgbif][] in their paper [Global plant‚Äìsymbiont organization and emergence of biogeochemical cycles resolved by evolution-based trait modelling](https://doi.org/10.1038/s41559-018-0759-0) [^2]
* Zizka et al. referenced [opencage][], [rgbif][], [rnaturalearth][], and [scrubr][] in their paper [CoordinateCleaner: standardized cleaning of occurrence records from biological collection databases](https://doi.org/10.1111/2041-210x.13152) [^3]
* Rice et al. used [rgbif][], [rnaturalearthdata][] and [CoordinateCleaner][] in their paper [The global biogeography of polyploid plants](https://doi.org/10.1038/s41559-018-0787-9) [^4]
* Sellgren et al. used [plotly][] in their paper [Peripheral and central levels of kynurenic acid in bipolar disorder subjects and healthy controls](https://doi.org/10.1038/s41398-019-0378-9) [^5]
* Gill et al. used [brranching][] [Plant DNA-barcode library and community phylogeny for a semi-arid East African savanna](https://doi.org/10.1111/1755-0998.13001) [^6]
* Hicks et al. used [rcrossref][] in their paper [Network analysis to evaluate the impact of research funding on research community consolidation](https://doi.org/10.1101/534495) [^7]
* Jovanoviƒá et al. used [plotly][] [Introducing of modeling techniques in the research of POPs in breast milk ‚Äì A pilot study](https://doi.org/10.1016/j.ecoenv.2019.01.087) [^8]
* Kay et al. used [plotly][] in their paper [Agroforestry is paying off ‚Äì Economic evaluation of ecosystem services in European landscapes with and without agroforestry systems](https://doi.org/10.1016/j.ecoser.2019.100896) [^9]
* Marcos et al. used [MODIStsp][] in their paper [Improving the detection of wildfire disturbances in space and time based on indicators extracted from MODIS data: a case study in northern Portugal](https://doi.org/10.1016/j.jag.2018.12.003) [^10]
* Martins et al. used [plotly][] in their paper [HABIT - a webserver for interactive T cell neoepitope discovery](https://doi.org/10.1101/535716) [^11]
* Nadal-Ribelles et al. used [plotly][] in their paper [Sensitive high-throughput single-cell RNA-seq reveals within-clonal transcript correlations in yeast populations](https://doi.org/10.1038/s41564-018-0346-9) [^12]
* Rigg et al. used [MODIStsp][] in their paper [Malaria infection rates in Anopheles albimanus (Diptera: Culicidae) at Ipet√≠-Guna, a village within a region targeted for malaria elimination in Panam√°](https://doi.org/10.1016/j.meegid.2019.02.003) [^13]
* Briz-Red√≥n used [osmdata][] in their paper [SpNetPrep: An R package using Shiny to facilitate spatial statistics on road networks](https://doi.org/10.3897/rio.5.e33521) [^14]


<br><br>

## In The News

Some nice words about [bib2df][] from community member [Philipp Ottolinger](https://github.com/ottlngr/).
<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Thank you <a href="https://twitter.com/0ttlngr?ref_src=twsrc%5Etfw">@0ttlngr</a> and collaborators for making the bib2df-package for R!<br>So practical to be able to read in <a href="https://twitter.com/hashtag/bibtex?src=hash&amp;ref_src=twsrc%5Etfw">#bibtex</a> syntax into neat complete dataframes, including quirky <a href="https://twitter.com/clld_dot_org?ref_src=twsrc%5Etfw">@clld_dot_org</a> <a href="https://twitter.com/hashtag/Glottolog?src=hash&amp;ref_src=twsrc%5Etfw">#Glottolog</a> specific field types without complaints of non-complete entries<a href="https://twitter.com/hashtag/rstats?src=hash&amp;ref_src=twsrc%5Etfw">#rstats</a></p>&mdash; Hedvig (@Laserhedvig) <a href="https://twitter.com/Laserhedvig/status/1093686544857088001?ref_src=twsrc%5Etfw">February 8, 2019</a></blockquote>
<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>

<br>

Someone found [NLMR][] and [landscapetools][] very useful! From community member [Marco Sciaini](https://github.com/marcosci/)
<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Using the excellent <a href="https://twitter.com/rOpenSci?ref_src=twsrc%5Etfw">@rOpenSci</a> NLMR and landscapetools üì¶s  by <a href="https://twitter.com/msciain?ref_src=twsrc%5Etfw">@msciain</a> et al. to create correlated spatial features for model simulation. Incredibly useful! <a href="https://t.co/FdK0dMf3Mm">pic.twitter.com/FdK0dMf3Mm</a></p>&mdash; Matt Harris (@Md_Harris) <a href="https://twitter.com/Md_Harris/status/1090469625907892224?ref_src=twsrc%5Etfw">January 30, 2019</a></blockquote>

<br>

[Brodie Gaslam](https://www.brodieg.com/) wrote a blog post using our [magick][] package (by [Jeroen Ooms](https://ropensci.org/about/#team)) [Favicons From the Comfort of Your R Session](https://www.brodieg.com/2019/02/09/favicons-from-the-comfort-of-your-r-session/)

<br><br>

## Call For Contributors

Part of the mission of rOpenSci is making sustainable software that users can rely on. Some software maintainers need to give up maintenance due to a variety of circumstances. When that happens we try to find new maintainers. We've had three recent examples of maintainer transitions within rOpenSci:

- [RSelenium][]: now maintained by [Ju Kim](https://github.com/juyeongkim)
- [chromer][]: now maintained by [Paula Andrea](https://github.com/orchid00)
- [qualtRics][]: now maintained by [Julia Silge](https://github.com/juliasilge)

We've still got two packages in need of new maintainers:

- [webchem][]: The current maintainer of webchem, [Eduard Sz√∂cs](https://github.com/EDiLD), is looking for a new maintainer. See [issue #155](https://github.com/ropensci/webchem/issues/155). Do ping Eduard in that issue and/or email us at [info@ropensci.org](mailto:info@ropensci.org)
- [rsnps][]: The current maintainer would like to hand it over to someone with more expertise in the domain. Please get in touch with [Scott](mailto:myrmecocystus@gmail.com) if you're interested.

<br><br>

## Related News

- The Red Hat Women in Open Source Award vote ends on February 25th. [Gabriela de Gueiroz](https://twitter.com/gdequeiroz) is one of the options: she's the founder of [R-Ladies](https://rladies.org/). You can vote here: <https://www.redhat.com/en/about/women-in-open-source>


<br><br>

<hr style="display: block; height: 1px; border: 0; border-top: 3px solid #7CCCC8; margin: 1em 0; padding: 0; ">

<br><br>

### Keep up with rOpenSci

* Mailing list: Sign up with an email address to get this newsletter sent to your inbox -> [ropensci.org/#subscribe](https://ropensci.org/#subscribe)
* Alternatively, you can subscribe to this newsletter via our XML feed at <https://news.ropensci.org/feed.xml> or our JSON feed at <https://news.ropensci.org/feed.json>
* rOpenSci on Twitter: [@ropensci](https://twitter.com/ropensci)
* The rOpenSci blog at [ropensci.org/blog](https://ropensci.org/blog) - you can subscribe in any RSS aggregator, or manually via <https://ropensci.org/feed.xml>. We also announce new blog posts on our Twitter account.

<br>

#### Footnotes

[^1]: Lee, A. J., Jones, B. C., & DeBruine, L. M. (2019, January 21). Investigating the association between mating-relevant self-concepts and mate preferences through a data-driven analysis of online personal descriptions. <https://doi.org/10.31234/osf.io/38zef>
[^2]: Lu, M., & Hedin, L. O. (2019). Global plant‚Äìsymbiont organization and emergence of biogeochemical cycles resolved by evolution-based trait modelling. Nature Ecology & Evolution, 3(2), 239‚Äì250. <https://doi.org/10.1038/s41559-018-0759-0>
[^3]: Zizka, A., Silvestro, D., Andermann, T., Azevedo, J., Duarte Ritter, C., Edler, D., ‚Ä¶ Antonelli, A. (2019). CoordinateCleaner: standardized cleaning of occurrence records from biological collection databases. Methods in Ecology and Evolution. <https://doi.org/10.1111/2041-210x.13152>
[^4]: Rice, A., ≈†marda, P., Novosolov, M., Drori, M., Glick, L., Sabath, N., ‚Ä¶ Mayrose, I. (2019). The global biogeography of polyploid plants. Nature Ecology & Evolution, 3(2), 265‚Äì273. <https://doi.org/10.1038/s41559-018-0787-9>
[^5]: Sellgren, C. M., Gracias, J., Jungholm, O., Perlis, R. H., Engberg, G., Schwieler, L., ‚Ä¶ Erhardt, S. (2019). Peripheral and central levels of kynurenic acid in bipolar disorder subjects and healthy controls. Translational Psychiatry, 9(1). <https://doi.org/10.1038/s41398-019-0378-9>
[^6]: Gill, B. A., Musili, P. M., Kurukura, S., Hassan, A. A., Goheen, J. R., Kress, W. J., ‚Ä¶ Kartzinel, T. R. (2019). Plant DNA-barcode library and community phylogeny for a semi-arid East African savanna. Molecular Ecology Resources. <https://doi.org/10.1111/1755-0998.13001>
[^7]: Hicks, D. J., Coil, D. A., Stahmer, C. G., & Eisen, J. A. (2019). Network analysis to evaluate the impact of research funding on research community consolidation. <https://doi.org/10.1101/534495>
[^8]: Jovanoviƒá, G., Romaniƒá, S. H., Stojiƒá, A., Klinƒçiƒá, D., Sariƒá, M. M., Letiniƒá, J. G., & Popoviƒá, A. (2019). Introducing of modeling techniques in the research of POPs in breast milk ‚Äì A pilot study. Ecotoxicology and Environmental Safety, 172, 341‚Äì347. <https://doi.org/10.1016/j.ecoenv.2019.01.087>
[^9]: Kay, S., Graves, A., Palma, J. H. N., Moreno, G., Roces-D√≠az, J. V., Aviron, S., ‚Ä¶ Herzog, F. (2019). Agroforestry is paying off ‚Äì Economic evaluation of ecosystem services in European landscapes with and without agroforestry systems. Ecosystem Services, 36, 100896. <https://doi.org/10.1016/j.ecoser.2019.100896>
[^10]: Marcos, B., Gon√ßalves, J., Alcaraz-Segura, D., Cunha, M., & Honrado, J. P. (2019). Improving the detection of wildfire disturbances in space and time based on indicators extracted from MODIS data: a case study in northern Portugal. International Journal of Applied Earth Observation and Geoinformation, 78, 77‚Äì85. <https://doi.org/10.1016/j.jag.2018.12.003>
[^11]: Martins, J., Magalhaes, C., Vieira, V., Rocha, M., & Osorio, N. S. (2019). HABIT - a webserver for interactive T cell neoepitope discovery. <https://doi.org/10.1101/535716>
[^12]: Nadal-Ribelles, M., Islam, S., Wei, W., Latorre, P., Nguyen, M., de Nadal, E., ‚Ä¶ Steinmetz, L. M. (2019). Sensitive high-throughput single-cell RNA-seq reveals within-clonal transcript correlations in yeast populations. Nature Microbiology. <https://doi.org/10.1038/s41564-018-0346-9>
[^13]: Rigg, C. A., Hurtado, L. A., Calzada, J. E., & Chaves, L. F. (2019). Malaria infection rates in Anopheles albimanus (Diptera: Culicidae) at Ipet√≠-Guna, a village within a region targeted for malaria elimination in Panam√°. Infection, Genetics and Evolution, 69, 216‚Äì223. <https://doi.org/10.1016/j.meegid.2019.02.003>
[^14]: Briz-Red√≥n, √Å. (2019). SpNetPrep: An R package using Shiny to facilitate spatial statistics on road networks. Research Ideas and Outcomes, 5. <https://doi.org/10.3897/rio.5.e33521>


[taxize]: https://github.com/ropensci/taxize
[rgbif]: https://github.com/ropensci/rgbif
[rdhs]: https://github.com/ropensci/rdhs
[RSelenium]: https://github.com/ropensci/RSelenium
[chromer]: https://github.com/ropensci/chromer
[qualtRics]: https://github.com/ropensci/qualtRics
[webchem]: https://github.com/ropensci/webchem
[rsnps]: https://github.com/ropensci/rsnps
[rnoaa]: https://github.com/ropensci/rnoaa
[rcrossref]: https://github.com/ropensci/rcrossref
[rnaturalearth]: https://github.com/ropensci/rnaturalearth
[plotly]: https://github.com/ropensci/plotly
[spocc]: https://github.com/ropensci/spocc
[rotl]: https://github.com/ropensci/rotl
[crul]: https://github.com/ropensci/crul
[emld]: https://github.com/cboettig/emld
[CytoRSuite]: https://github.com/DillonHammill/CytoRSuite
[outsider]: https://github.com/AntonelliLab/outsider
[V8]: https://github.com/jeroen/V8
[rnaturalearthdata]: https://github.com/ropensci/rnaturalearthdata
[hunspell]: https://github.com/ropensci/hunspell
[opencage]: https://github.com/ropensci/opencage
[CoordinateCleaner]: https://github.com/ropensci/CoordinateCleaner
[brranching]: https://github.com/ropensci/brranching
[MODIStsp]: https://github.com/ropensci/MODIStsp
[osmdata]: https://github.com/ropensci/osmdata
[bib2df]: https://github.com/ropensci/bib2df
[NLMR]: https://github.com/ropensci/NLMR
[landscapetools]: https://github.com/ropensci/landscapetools
[magick]: https://github.com/ropensci/magick
[scrubr]: https://github.com/ropensci/scrubr
