---
layout: post
title:  "2019-08-05"
date:   2019-08-05
slug:   "2019-08-05"
---

## rOpenSci HQ

* [Recording and notes](https://ropensci.org/commcalls/2019-07-30/) and [public discussion](https://discuss.ropensci.org/t/community-call-reproducible-research-with-r/1765) are up from our latest [community call on Reproducible Research with R](https://ropensci.org/blog/2019/07/11/commcall-jul2019/).

<br>

## Software üì¶

CRAN: <img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px">
GitHub: <img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px">

### New packages

* The first version (`v1.7`) of `tidypmc` is on CRAN - parse full text XML documents from PubMed Central. Checkout the [vignettes](https://cran.rstudio.com/web/packages/tidypmc/vignettes/) to get started. {{ "tidypmc" | image_cran }} {{ "tidypmc" | image_github }}
* The first version (`v1.2.5`) of `UCSCXenaTools` is on CRAN - download and explore datasets from UCSC Xena data hubs. Checkout the [docs](https://shixiangwang.github.io/home/en/tools/ucscxenatools-intro/) to get started. {{ "UCSCXenaTools" | image_cran }} {{ "UCSCXenaTools" | image_github }}

### New Versions

* A new version (`v0.8.4`) of `crul` is on CRAN - an R6 based HTTP client for R made for developers. See the [release notes](https://github.com/ropensci/crul/releases/tag/v0.8.4) for changes. Checkout the [docs](https://ropensci.github.io/crul/) to get started. {{ "crul" | image_cran }} {{ "crul" | image_github }}
> (Windows users please do upgrade: no longer set cainfo curl option); improve parse method in HttpResponse class
* A new version (`v1.22`) of `rtika` is on CRAN - R Interface to Apache Tika. See the [release notes](https://github.com/ropensci/rtika/blob/master/NEWS.md) for changes. Checkout the [docs](https://ropensci.github.io/rtika/) to get started. {{ "rtika" | image_cran }} {{ "rtika" | image_github }}
> update Tika to 1.22
* A new version (`v0.3.0`) of `stplanr` is on CRAN - sustainable transport planning. See the [release notes](https://github.com/ropensci/stplanr/blob/master/NEWS.md) for changes. Checkout the [docs](https://ropensci.github.io/stplanr/) to get started. {{ "stplanr" | image_cran }} {{ "stplanr" | image_github }}
> new fxns: `od_to_odmatrix()`, `odmatrix_to_od()`, `od_oneway()`, and many new `od_id*()` fxns; new vignette
* A new version (`v0.1.3`) of `gitignore` is on CRAN - create useful .gitignore files for your project. See the [release notes](https://github.com/ropensci/gitignore/releases/tag/v0.1.3) for changes. Checkout the [docs](https://docs.ropensci.org/gitignore/) to get started. {{ "gitignore" | image_cran }} {{ "gitignore" | image_github }}
> fix cran check problems
* A new version (`v1.0.0`) of `stats19` is on CRAN - Work with Open Road Traffic Casualty Data from Great Britain. See the [release notes](https://github.com/ropensci/stats19/blob/master/NEWS.md) for changes. Checkout the [docs](https://itsleeds.github.io/stats19/) to get started. {{ "stats19" | image_cran }} {{ "stats19" | image_github }}
> now much easier to download STATS19 data w/ `dl_stats19()`
* A new version (`v0.5.0`) of `brranching` is on CRAN - Fetch Phylogenies from Many Sources. See the [release notes](https://github.com/ropensci/brranching/releases/tag/v0.5.0) for changes. Checkout the [vignette](https://cran.rstudio.com/web/packages/brranching/vignettes/brranching.html) to get started. {{ "brranching" | image_cran }} {{ "brranching" | image_github }}
> now using pkg [conditionz][] to make messages less frequent when user does not have an API key for NCBI
* A new version (`v0.1.2`) of `geoparser` is on CRAN - interface to the Geoparser.io API. See the [release notes](https://github.com/ropensci/geoparser/releases/tag/v0.1.2) for changes. Checkout the [docs](https://docs.ropensci.org/geoparser) to get started. {{ "geoparser" | image_cran }} {{ "geoparser" | image_github }}
> fix error caused by namespace issue; `geoparser_key()` fails better when no key avail.
* A new version (`v0.0.3`) of `plotdap` is on CRAN - easily visualize data from ERDDAP servers via the rerddap package. See the [release notes](https://github.com/ropensci/plotdap/releases/tag/v0.0.3) for changes. Checkout the [vignette](https://cran.rstudio.com/web/packages/plotdap/vignettes/using_plotdap.html) to get started. {{ "plotdap" | image_cran }} {{ "plotdap" | image_github }}
* A new version (`v2.1`) of `magick` is on CRAN - Advanced graphics and Image-Processing in R. See the [release notes](https://github.com/ropensci/magick/releases/tag/v2.1) for changes. Checkout the [vignette](https://cran.rstudio.com/web/packages/magick/vignettes/intro.html) to get started. {{ "magick" | image_cran }} {{ "magick" | image_github }}
> new docs site; many fixes
* A new version (`v4.1`) of `tesseract` is on CRAN - bindings to Tesseract, an OCR engine <https://opensource.google.com/projects/tesseract>. See the [release notes](https://github.com/ropensci/tesseract/releases/tag/v4.1) for changes. Checkout the [vignette](https://cran.rstudio.com/web/packages/tesseract/vignettes/intro.html) to get started. {{ "tesseract" | image_cran }} {{ "tesseract" | image_github }}
> fix memory leak in `ocr_data`; update to libtesseract 4.1.0
* A new version (`v0.1.4`) of `phylocomr` is on CRAN - Interface to Phylocom for analysis of phylogenetic community structure and character evolution. See the [release notes](https://github.com/ropensci/phylocomr/releases/tag/v0.1.4) for changes. Checkout the [vignette](https://cran.rstudio.com/web/packages/phylocomr/vignettes/phylocomr.html) to get started. {{ "phylocomr" | image_cran }} {{ "phylocomr" | image_github }}
> fix some non-working examples; fixes to internal type checking; fix C lib error
* A new version (`v0.3.0`) of `ckanr` is on CRAN - R client for the CKAN API. See the [release notes](https://github.com/ropensci/ckanr/releases/tag/v0.3.0) for changes. Checkout the [docs](https://docs.ropensci.org/ckanr/) to get started. {{ "ckanr" | image_cran }} {{ "ckanr" | image_github }}
> big release, first in 4 years: new pkg maintainer; many new fxns; many fxns w/ changed params; fxn name changes

<br><br>

## Software Review ‚úî

We accept community contributed packages via our software review system - an open software review system, sorta like scholarly paper review, but way better. We'll highlight newly onboarded packages here. A huge thanks to our reviewers, who do a lot of work reviewing (see the [blog post on our review system](https://ropensci.org/blog/2016/03/28/software-review)),
and the authors of the packages!

If you want to be a reviewer fill out [this short form](https://ropensci.org/onboarding/), and we'll ping you when there's a submission that fits in your area of expertise.

The following five packages were recently submitted:

* [photosearcher][] > Queries the Flick API for photo images and metadata
    * Author: [Nathan Fox](https://github.com/laurajanegraham)
    * Issue: [ropensci/onboarding#325](https://github.com/ropensci/onboarding/issues/325)
    * Reviewers:
        * [Monica Gerber](https://github.com/monicagerber)
        * other not assigned yet
* [Rclean][] > A Tool for Writing Cleaner, More Transparent Code
    * Author: [Matthew Lau](https://github.com/MKLau)
    * Issue: [ropensci/onboarding#327](https://github.com/ropensci/onboarding/issues/327)
    * Reviewers: not yet assigned
* [mapscanner][] > Print Maps, Draw on Them, Scan Them Back In
    * Author: [Mark Padgham](https://github.com/mpadge)
    * Issue: [ropensci/onboarding#330](https://github.com/ropensci/onboarding/issues/330)
    * Reviewers:
        * [Nicholas Potter](https://github.com/potterzot)
        * other not assigned yet
* [workloopR][] > Analysis of Work Loops and Other Data from Muscle Physiology Experiments
    * Author: [Vikram Baliga](https://github.com/vbaliga)
    * Issue: [ropensci/onboarding#326](https://github.com/ropensci/onboarding/issues/326)
    * Reviewers: not yet assigned
* [rmangal][] > An interface to the Mangal database <https://mangal.io/#/>
    * Author: [Steve Vissault](https://github.com/SteveViss)
    * Issue: [ropensci/onboarding#332](https://github.com/ropensci/onboarding/issues/332)
    * Reviewers: not yet assigned

The following package recently went through our software-review process and has been approved:

* [UCSCXenaTools][] > Download and Explore Datasets from UCSC Xena Data Hubs
    * Author: [Shixiang Wang](https://github.com/ShixiangWang)
    * Issue: [ropensci/software-review#315](https://github.com/ropensci/software-review/issues/315)
    * Reviewers:
        * [Christine Stawitz](https://github.com/ChristineStawitz-NOAA)
        * [Carl Ganz](https://github.com/carlganz)


<br><br>

## Use Cases

The following 15 works use/cite rOpenSci software:

* Wright et al. cite [plotly][] in their paper [Understanding microbial community dynamics to improve optimal microbiome selection](https://doi.org/10.1186/s40168-019-0702-x) [^1]
* Meisner et al. used [FedData][] in their paper [Soil and climactic predictors of canine coccidioidomycosis seroprevalence in Washington State: an ecological cross‚Äêsectional study](https://doi.org/10.1111/tbed.13265)> [^2]
* Tetzlaff et al. used [rotl][] in their paper [Effects of antipredator training, environmental enrichment, and soft release on wildlife translocations: A review and meta-analysis](https://doi.org/10.1016/j.biocon.2019.05.054) [^3]
* Alhajeri & Fourcade used [rredlist][] and [rgbif][] in their paper [High correlation between species‚Äêlevel environmental data estimates extracted from IUCN expert range maps and from GBIF occurrence data](https://doi.org/10.1111/jbi.13619) [^4]
*  Boggs et al. used [taxize][] in their paper [Single Fragment or Bulk Soil DNA Metabarcoding: Which is Better for Characterizing Biological Taxa Found in Surface Soils for Sample Separation?](https://doi.org/10.3390/genes10060431) [^5]
* Ros-Candeira et al. used [rgbif][] in their paper [Dataset of occurrence and incidence of pine processionary moth in Andalusia, south Spain](https://doi.org/10.3897/zookeys.852.28567) [^6]
* Van Denderen et al. used [rfishbase][] in their pre-print [Little difference in average fish growth and maximum size across temperatures](https://doi.org/10.32942/osf.io/8cu4y) [^7]
* Bailey et al. used [plotly][] in their paper [Habitat selection can reduce effects of extreme climatic events in a long‚Äêlived shorebird](https://doi.org/10.1111/1365-2656.13041) [^8]
* Palacios-Abrantes et al. used [taxize][] in their paper [A metadata approach to evaluate the state of ocean knowledge: Strengths, limitations, and application to Mexico](https://doi.org/10.1371/journal.pone.0216723) [^9]
* McTavish cited [rotl][] and [rgbif][] in her abstract for the upcoming [Biodiversity Next conference](https://biodiversitynext.org/): [Linking Biodiversity Data Using Evolutionary History](https://doi.org/10.3897/biss.3.36207) [^10]
* Nyboer et al. used [rfishbase][] and [rredlist][] in their paper [Assessing the vulnerability of Africa‚Äôs freshwater fishes to climate change: A continent-wide trait-based analysis](https://doi.org/10.1016/j.biocon.2019.05.003) [^11]
* Krah & Heibl used [phylotar][] in their paper [rGUIDANCE ‚Äì alignment confidence score computation in R](https://doi.org/10.21105/joss.01350) [^12]
*  Morelle et al. used [osmdata][] in their paper [Deathbed choice by ASF‚Äêinfected wild boar can help find carcasses](https://doi.org/10.1111/tbed.13267) [^13]
*  Grattarola et al. used [rredlist][] and [taxize][] in their paper [Biodiversidata: An Open-Access Biodiversity Database for Uruguay](https://doi.org/10.3897/bdj.7.e36226) [^14]
* Kravchuk et al. used [rotl][] in their paper [Characterization of the 20S proteasome of the lepidopteran, Spodoptera frugiperda](https://doi.org/10.1016/j.bbapap.2019.06.010) [^15]

<br><br>

## In The News

[Dan Quintana](https://twitter.com/dsquintana) gives a short video demonstration of using [europepmc][], a package maintained by [Najko Jahn](https://github.com/njahn82)
<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Ever wanted to visualize the popularity of a research topic over time? Here&#39;s how, using the &#39;europepmc&#39; <a href="https://twitter.com/hashtag/Rstats?src=hash&amp;ref_src=twsrc%5Etfw">#Rstats</a> package<br><br>YouTube link: <a href="https://t.co/fI5B6PLDco">https://t.co/fI5B6PLDco</a><br>Code: <a href="https://t.co/hSnYiBrUh8">https://t.co/hSnYiBrUh8</a><br>Thread üëá <a href="https://t.co/Ezsh9jUJul">pic.twitter.com/Ezsh9jUJul</a></p>&mdash; Dan Quintana (@dsquintana) <a href="https://twitter.com/dsquintana/status/1153765004606394372?ref_src=twsrc%5Etfw">July 23, 2019</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script> 


<br><br>

## Call For Contributors

Part of the mission of rOpenSci is making sustainable software that users can rely on. Some software maintainers need to give up maintenance due to a variety of circumstances. When that happens we try to find new maintainers. Checkout our [Contributing Guide](https://devguide.ropensci.org/contributingguide.html) for why and how to contribute to rOpenSci.

We've had six recent examples of maintainer transitions within rOpenSci:

- [RSelenium][]: now maintained by [Ju Kim](https://github.com/juyeongkim)
- [chromer][]: now maintained by [Paula Andrea](https://github.com/orchid00)
- [qualtRics][]: now maintained by [Julia Silge](https://github.com/juliasilge)
- [rsnps][]: now maintained by [Julia Gustavsen](https://github.com/jooolia) and [Sina R√ºeger](https://github.com/sinarueeger)
- [rdpla][]: now maintained by [Alyssa Columbus](https://github.com/acolum)
- [webchem][]: now maintained by [Erik Sapper](https://github.com/eriksapper)

We don't have any packages looking for new maintainers - if you are a maintainer and want to hand over that role to someone else get in touch with us.

<br><br>

<hr style="display: block; height: 1px; border: 0; border-top: 3px solid #7CCCC8; margin: 1em 0; padding: 0; ">

<br><br>

### Keep up with rOpenSci

* Mailing list: Sign up with an email address to get this newsletter sent to your inbox -> [ropensci.org/#subscribe](https://ropensci.org/#subscribe)
* Alternatively, you can subscribe to this newsletter via our XML feed at <https://news.ropensci.org/feed.xml> or our JSON feed at <https://news.ropensci.org/feed.json>
* rOpenSci on Twitter: [@ropensci](https://twitter.com/ropensci)
* The rOpenSci blog at [ropensci.org/blog](https://ropensci.org/blog) - you can subscribe in any RSS aggregator, or manually via <https://ropensci.org/feed.xml>. We also announce new blog posts on our Twitter account.

<br>

#### Footnotes

[^1]: Wright, R. J., Gibson, M. I., & Christie-Oleza, J. A. (2019). Understanding microbial community dynamics to improve optimal microbiome selection. Microbiome, 7(1). <https://doi.org/10.1186/s40168-019-0702-x>
[^2]: Meisner, J., Clifford, W. R., Wohrle, R. D., Kangiser, D., & Rabinowitz, P. (2019). Soil and climactic predictors of canine coccidioidomycosis seroprevalence in Washington State: an ecological cross‚Äêsectional study. Transboundary and Emerging Diseases. <https://doi.org/10.1111/tbed.13265>
[^3]: Tetzlaff, S. J., Sperry, J. H., & DeGregorio, B. A. (2019). Effects of antipredator training, environmental enrichment, and soft release on wildlife translocations: A review and meta-analysis. Biological Conservation, 236, 324‚Äì331. <https://doi.org/10.1016/j.biocon.2019.05.054>
[^4]: Alhajeri, B. H., & Fourcade, Y. (2019). High correlation between species‚Äêlevel environmental data estimates extracted from IUCN expert range maps and from GBIF occurrence data. Journal of Biogeography. <https://doi.org/10.1111/jbi.13619>
[^5]: Boggs, Scheible, Machado, & Meiklejohn. (2019). Single Fragment or Bulk Soil DNA Metabarcoding: Which is Better for Characterizing Biological Taxa Found in Surface Soils for Sample Separation? Genes, 10(6), 431. <https://doi.org/10.3390/genes10060431>
[^6]: Ros-Candeira, A., P√©rez-Luque, A. J., Su√°rez-Mu√±oz, M., Bonet-Garc√≠a, F. J., H√≥dar, J. A., Gim√©nez de Azc√°rate, F., & Ortega-D√≠az, E. (2019). Dataset of occurrence and incidence of pine processionary moth in Andalusia, south Spain. ZooKeys, 852, 125‚Äì136. <https://doi.org/10.3897/zookeys.852.28567>
[^7]: Van Denderen, D., Gislason, H., & Andersen, K. H. (2019). Little difference in average fish growth and maximum size across temperatures. EcoEvoRxiv. <https://doi.org/10.32942/osf.io/8cu4y>
[^8]: Bailey, L. D., Ens, B. J., Both, C., Heg, D., Oosterbeek, K., & van de Pol, M. (2019). Habitat selection can reduce effects of extreme climatic events in a long‚Äêlived shorebird. Journal of Animal Ecology. <https://doi.org/10.1111/1365-2656.13041>
[^9]: Palacios-Abrantes, J., Cisneros-Montemayor, A. M., Cisneros-Mata, M. A., Rodr√≠guez, L., Arregu√≠n-S√°nchez, F., Aguilar, V., ‚Ä¶ Cheung, W. W. L. (2019). A metadata approach to evaluate the state of ocean knowledge: Strengths, limitations, and application to Mexico. PLOS ONE, 14(6), e0216723. <https://doi.org/10.1371/journal.pone.0216723>
[^10]: McTavish, E. J. (2019). Linking Biodiversity Data Using Evolutionary History. Biodiversity Information Science and Standards, 3. <https://doi.org/10.3897/biss.3.36207>
[^11]: Nyboer, E. A., Liang, C., & Chapman, L. J. (2019). Assessing the vulnerability of Africa‚Äôs freshwater fishes to climate change: A continent-wide trait-based analysis. Biological Conservation, 236, 505‚Äì520. <https://doi.org/10.1016/j.biocon.2019.05.003>
[^12]: Krah, F.-S., & Heibl, C. (2019). rGUIDANCE ‚Äì alignment confidence score computation in R. Journal of Open Source Software, 4(36), 1350. <https://doi.org/10.21105/joss.01350>
[^13]: Morelle, K., Jezek, M., Licoppe, A., & Podgorski, T. (2019). Deathbed choice by ASF‚Äêinfected wild boar can help find carcasses. Transboundary and Emerging Diseases. <https://doi.org/10.1111/tbed.13267>
[^14]: Grattarola, F., Botto, G., da Rosa, I., Gobel, N., Gonz√°lez, E., Gonz√°lez, J., ‚Ä¶ Pincheira-Donoso, D. (2019). Biodiversidata: An Open-Access Biodiversity Database for Uruguay. Biodiversity Data Journal, 7. <https://doi.org/10.3897/bdj.7.e36226>
[^15]: Kravchuk, O. I., Lyupina, Y. V., Erokhov, P. A., Finoshin, A. D., Adameyko, K. I., Mishyna, M. Y., ‚Ä¶ Mikhailov, V. S. (2019). Characterization of the 20S proteasome of the lepidopteran, Spodoptera frugiperda. Biochimica et Biophysica Acta (BBA) - Proteins and Proteomics. <https://doi.org/10.1016/j.bbapap.2019.06.010>


[RSelenium]: https://github.com/ropensci/RSelenium
[chromer]: https://github.com/ropensci/chromer
[qualtRics]: https://github.com/ropensci/qualtRics
[rsnps]: https://github.com/ropensci/rsnps
[rdpla]: https://github.com/ropensci/rdpla
[webchem]: https://github.com/ropensci/webchem
[UCSCXenaTools]: https://github.com/ropensci/UCSCXenaTools
[photosearcher]: https://github.com/nfox29/photosearcher
[Rclean]: https://github.com/provtools/rclean
[mapscanner]: https://github.com/mpadge/mapscanner
[rmangal]: https://github.com/mangal-wg/rmangal
[workloopR]: https://github.com/vbaliga/workloopR
[europepmc]: https://github.com/ropensci/europepmc/
[FedData]: https://github.com/ropensci/FedData/
[taxize]: https://github.com/ropensci/taxize/
[rgbif]: https://github.com/ropensci/rgbif/
[rredlist]: https://github.com/ropensci/rredlist/
[rfishbase]: https://github.com/ropensci/rfishbase/
[phylotar]: https://github.com/ropensci/phylotar/
[osmdata]: https://github.com/ropensci/osmdata/
[rotl]: https://github.com/ropensci/rotl/
[plotly]: https://github.com/ropensci/plotly/
[conditionz]: https://github.com/ropenscilabs/conditionz/
[x]: https://github.com/ropensci/x/
[x]: https://github.com/ropensci/x/
[x]: https://github.com/ropensci/x/




