---
layout: post
title:  "2020-05-11"
date:   2020-05-11
slug:   "2020-05-11"
---

## Software üì¶

CRAN: <img src="../assets/img/octicon-package.png" width="25" style="border-radius: 6px 6px 6px 6px">
GitHub: <img src="../assets/img/github-alt.png" width="25" style="border-radius: 6px 6px 6px 6px">

### New Versions

* A new version (`v1.1.1`) of `dbparser` is on CRAN - DrugBank database XML parser. See the [release notes](https://github.com/ropensci/dbparser/blob/master/NEWS.md) for changes. Checkout the [docs](https://docs.ropensci.org/dbparser/) to get started. {{ "dbparser" | image_cran }} {{ "dbparser" | image_github }}
* A new version (`v1.4.0`) of `MODIStsp` is on CRAN - Automate Download and Preprocessing of MODIS Land Products Data. See the [release notes](https://github.com/ropensci/MODIStsp/releases/tag/v1.4.0) for changes. Checkout the [docs](https://docs.ropensci.org/MODIStsp/) to get started. {{ "MODIStsp" | image_cran }} {{ "MODIStsp" | image_github }}
* A new version (`v2.4.4`) of `RNeXML` is on CRAN - Semantically Rich I/O for the NeXML Format. See the [release notes](https://github.com/ropensci/RNeXML/blob/master/NEWS.md) for changes. Checkout the [docs](https://docs.ropensci.org/RNeXML/) to get started. {{ "RNeXML" | image_cran }} {{ "RNeXML" | image_github }}
* A new version (`v0.1.1`) of `colocr` is on CRAN - conduct co-localization analysis of fluorescence microscopy images. See the [release notes](https://github.com/ropensci/colocr/blob/master/NEWS.md) for changes. Checkout the [docs](https://docs.ropensci.org/colocr) to get started. {{ "colocr" | image_cran }} {{ "colocr" | image_github }}
* A new version (`v0.3.2`) of `cRegulome` is on CRAN - access, manage and visualize regulome (microRNA/transcription factors)-gene correlations in cancer. See the [release notes](https://github.com/ropensci/cRegulome/blob/master/NEWS.md) for changes. Checkout the [docs](https://docs.ropensci.org/cRegulome/) to get started. {{ "cRegulome" | image_cran }} {{ "cRegulome" | image_github }}
* A new version (`v0.3.6`) of `eia` is on CRAN - API wrapper for US Energy Information Administration open data. See the [release notes](https://github.com/ropensci/eia/blob/master/NEWS.md) for changes. Checkout the [docs](https://docs.ropensci.org/eia/) to get started. {{ "eia" | image_cran }} {{ "eia" | image_github }}
* A new version (`v2.3.6`) of `beautier` is on CRAN - create BEAST2 input files programmatically. See the [release notes](https://github.com/ropensci/beautier/releases/tag/v2.3.6) for changes. Checkout the [docs](https://docs.ropensci.org/beautier) to get started. {{ "beautier" | image_cran }} {{ "beautier" | image_github }}
* A new version (`v2.1`) of `tracerer` is on CRAN - parse and analyze BEAST2 input files programmatically. See the [release notes](https://github.com/ropensci/tracerer/releases/tag/v2.1) for changes. Checkout the [docs](https://docs.ropensci.org/tracerer/) to get started. {{ "tracerer" | image_cran }} {{ "tracerer" | image_github }}
* A new version (`v1.3`) of `writexl` is on CRAN - a portable, light-weight data frame to xlsx exporter. See the [release notes](https://github.com/ropensci/writexl/blob/master/NEWS) for changes. Checkout the [README](https://github.com/ropensci/writexl/#writexl) to get started. {{ "writexl" | image_cran }} {{ "writexl" | image_github }}
* A new version (`v0.27.1`) of `git2r` is on CRAN - an interface to the libgit2 library, a pure C implementation of the Git core methods. See the [release notes](https://github.com/ropensci/git2r/releases/tag/v0.27.1) for changes. Checkout the [README](https://docs.ropensci.org/git2r/) to get started. {{ "git2r" | image_cran }} {{ "git2r" | image_github }}
* A new version (`v0.6.0`) of `stplanr` is on CRAN - sustainable transport planning. See the [release notes](https://github.com/ropensci/stplanr/blob/master/NEWS.md) for changes. Checkout the [docs](https://docs.ropensci.org/stplanr/) to get started. {{ "stplanr" | image_cran }} {{ "stplanr" | image_github }}
* A new version (`v1.0.0`) of `bold` is on CRAN - interface to Bold Systems (<http://www.boldsystems.org/>) API. See the [release notes](https://github.com/ropensci/bold/releases/tag/v1.0.0) for changes. Checkout the [docs](https://docs.ropensci.org/bold/) to get started. {{ "bold" | image_cran }} {{ "bold" | image_github }}
* A new version (`v0.3.4`) of `taxa` is on CRAN - taxonomic classes for groupings of taxonomic names without data, and those with data. See the [release notes](https://github.com/ropensci/taxa/releases/tag/v0.3.4) for changes. Checkout the [docs](https://docs.ropensci.org/taxa) to get started. {{ "taxa" | image_cran }} {{ "taxa" | image_github }}
* A new version (`v0.9.95`) of `taxize` is on CRAN - taxonomic toolbelt for R. See the [release notes](https://github.com/ropensci/taxize/releases/tag/v0.9.95) for changes. Checkout the [taxize book](https://taxize.dev/) to get started. {{ "taxize" | image_cran }} {{ "taxize" | image_github }}

<br><br>

## Software Review ‚úî

In the interest of reducing load on reviewers and editors as we manage the COVID-19 crisis, rOpenSci is temporarily pausing new submissions for software peer review. Please check back at our [software review repository](https://github.com/ropensci/software-review) for updates. In this period new submissions will not be handled, nor new reviewers assigned. Reviews and responses to reviews will be handled on a 'best effort' basis, but no follow-up reminders will be sent. Other rOpenSci community activities continue. We express our continued great appreciation for the work of our authors and reviewers.

<br><br>

## On the blog

[Ma√´lle Salmon](https://ropensci.org/author/ma%C3%ABlle-salmon/) has two blog posts in the past two weeks:

- [Highlights of Hugo Code Highlighting](https://ropensci.org/technotes/2020/04/30/code-highlighting/) - related to work on the rOpenSci website
- [A Roundup of R Tools for Handling BibTeX](https://ropensci.org/technotes/2020/05/07/rmd-citations/) - including discussion of rOpenSci packages [RefManageR][], [bib2df][], and [handlr][]


<br><br>

## Use Cases

The following 23 works use/cite rOpenSci software:

* Adil _et al_. used [pdftools][] in their paper [Identifying Trends in Data Science Articles using Text Mining](https://ieeexplore.ieee.org/document/8981702) [^1]
* Finoshin _et al_. used [rotl][] in their paper [Iron metabolic pathways in the processes of sponge plasticity](https://doi.org/10.1371/journal.pone.0228722) [^2]
* Pimiento & Benton used [paleobioDB][] and [rfishbase][] in their paper [The impact of the Pull of the Recent on extant elasmobranchs](https://doi.org/10.1111/pala.12478) [^3]
* Benhadi‚ÄêMar√≠n _et al_. used [rgbif][] in their paper [Distribution of Bactrocera oleae (Rossi, 1790) throughout the Iberian Peninsula based on a maximum entropy modeling approach](https://doi.org/10.1111/aab.12584) [^4]
* Ahrenfeldt _et al_. used [treeio][] in their paper [Metaphylogenetic analysis of global sewage reveals that bacterial strains associated with human disease show less degree of geographic clustering](https://doi.org/10.1038/s41598-020-59292-w) [^5]
* Cinelli used [hunspell][] in their paper [Generalized rich-club ordering in networks](https://doi.org/10.1093/comnet/cnz002) [^6]
* Alfaro-Ram√≠rez _et al_. used [CoordinateCleaner][] in their paper [Potential reduction of Hartweg ÃÅs Pine (Pinus hartwegii Lindl.) geographic distribution](https://doi.org/10.1371/journal.pone.0229178) [^7]
* Martinez-Feria & Basso used [FedData][] in their paper [Unstable crop yields reveal opportunities for site-specific adaptations to climate variability](https://doi.org/10.1038/s41598-020-59494-2) [^8]
* Rego _et al_. used [rnaturalearth][] in their paper [Diversity of Bacterial Biosynthetic Genes in Maritime Antarctica](https://doi.org/10.3390/microorganisms8020279) [^9]
* Cederwall _et al_. used [taxize][] in their paper [Life under an oil slick: response of a freshwater food web to simulated spills of diluted bitumen in field mesocosms](https://doi.org/10.1139/cjfas-2019-0224) [^10]
* Mossion _et al_. used [taxize][] in their paper [A reference transcriptome for the early-branching fern Botrychium lunaria enables fine-grained resolution of population structure](https://doi.org/10.1101/2020.02.17.952283) [^11]
* Funk _et al_. used [hunspell][] in their paper [A Framework for Applying Natural Language Processing in Digital Health Interventions](https://doi.org/10.2196/13855) [^12]
* Correia _et al_. used [skimr][] in their paper [RNA Sequencing (RNA-Seq) Reveals Extremely Low Levels of Reticulocyte-Derived Globin Gene Transcripts in Peripheral Blood From Horses (Equus caballus) and Cattle (Bos taurus](https://doi.org/10.3389/fgene.2018.00278) [^13]
* Verde Arregoitia _et al_. used [taxize][] and [pdftools][] in their paper [Patterns in research and data sharing for the study of form and function in caviomorph rodents](https://doi.org/10.1093/jmammal/gyaa002) [^14]
* Shivambu _et al_. used [rgbif][] in their paper [Impact assessment of seven alien invasive bird species already introduced to South Africa](https://doi.org/10.1007/s10530-020-02221-9) [^15]
* Jhwueng & O‚ÄôMeara used [rotl][] in their paper [On the Matrix Condition of Phylogenetic Tree](https://doi.org/10.1177/1176934320901721) [^16]
* Cooke _et al_. used [rredlist][] in their paper [Ecological distinctiveness of birds and mammals at the global scale](https://doi.org/10.1016/j.gecco.2020.e00970) [^17]
* Palanisamy used [pdftools][] in their paper [An analysis on hidden markov model prediction factors using California wildfire and meteorological records](https://bit.ly/2yCWk2M) [^18]
* Panayiotou used [rdflib][] in their paper [An Ontological Analysis and Natural Language Processing of Figures of Speech](https://doi.org/10.5121/ijaia.2020.11102) [^19]
* Ji _et al_. used [rredlist][] in their paper [Large-scale Quantification of Vertebrate Biodiversity in Ailaoshan Nature Reserve from Leech iDNA](https://doi.org/10.1101/2020.02.10.941336) [^20]
* Li & Guo used [rgbif][] in their paper [Substantially adaptive potential in polyploid cyprinid fishes: evidence from biogeographic, phylogenetic and genomic studies](https://doi.org/10.1098/rspb.2019.3008) [^21]
* Silva _et al_. used [NLMR][] and [landscapetools][] in their paper [Revisiting reptile home ranges: moving beyond traditional estimators with dynamic Brownian Bridge Movement Models](https://doi.org/10.1101/2020.02.10.941278) [^22]
* Schwalb‚ÄêWillmann _et al_. used [magick][] in their paper [moveVis: Animating movement trajectories in synchronicity with static or temporally dynamic environmental data in R](https://doi.org/10.1111/2041-210x.13374) [^23]

<br><br>

## From the Forum

We have a [discussion forum](https://discuss.ropensci.org) (using [Discourse](https://www.discourse.org/)) for the rOpenSci community. It's a really nice way to have conversations on the internet. From time to time we'll highlight recent discussions of interest.

<br>

A few new use cases were shared in the forum recently:

* [Rami Krispin](https://twitter.com/Rami_Krispin) shared: [Visualize covid19 cases using rnaturalearth package](https://discuss.ropensci.org/t/visualize-covid19-cases-using-rnaturalearth-package/2058); it leveraged [rnaturalearth][], an rOpenSci package maintained by Andy South
* [Miles McBain](https://twitter.com/milesmcbain) shared: [{drake} at Queensland Fire and Emergency Services](https://discuss.ropensci.org/t/drake-at-queensland-fire-and-emergency-services/2054); it leveraged [drake][], an rOpenSci package maintained by Will Landau
* Alberto Cairo shared [a use case](https://discuss.ropensci.org/t/open-trade-statistics-https-tradestatistics-io/2053) for the rOpenSci package [tradestatistics][] maintained by [Mauricio Vargas](https://twitter.com/pachamaltese)


<br><br>

## In The News

- `testing`: [Why You Should Document Your Tests](https://hynek.me/articles/document-your-tests/)
    - Should you document your tests? I for one, only sometimes include comments in my tests, but perhaps they should be documented better? Do you comment or document your tests? This blog post is about Python, but the ideas apply to any language.

<br><br>

## Call For Maintainers

Part of the mission of rOpenSci is making sustainable software that users can rely on. Some software maintainers need to give up maintenance due to a variety of circumstances. When that happens we try to find new maintainers. Checkout our [Contributing Guide](https://devguide.ropensci.org/contributingguide.html) for why and how to contribute to rOpenSci.

We've had eight recent examples of maintainer transitions within rOpenSci:

- [RSelenium][]: now maintained by [Ju Kim](https://github.com/juyeongkim)
- [chromer][]: now maintained by [Paula Andrea](https://github.com/orchid00)
- [qualtRics][]: now maintained by [Julia Silge](https://github.com/juliasilge)
- [rsnps][]: now maintained by [Julia Gustavsen](https://github.com/jooolia) and [Sina R√ºeger](https://github.com/sinarueeger)
- [rdpla][]: now maintained by [Alyssa Columbus](https://github.com/acolum)
- [webchem][]: now maintained by [Erik Sapper](https://github.com/eriksapper)
- [mregions][]: in process of moving to maintenance by [VLIZ](http://www.vliz.be)
- [rWBclimate][]: now maintained by [Sergio Ibarra-Espinosa](https://github.com/ibarraespinosa) and [Amanda Rehbein](https://github.com/salvatirehbein)

We've got one package in need of a new maintainer:

- [rinat][]: The current maintainer is looking for a new maintainer. [Email Scott](mailto:myrmecocystus@gmail.com) if you're interested.

<br><br>

## Get involved with rOpenSci

We maintain a [Contributing Guide](https://devguide.ropensci.org/contributingguide.html) that can help direct you to the right place, whether you want to make code contributions, non-code contributions, or other things like sharing use cases.

<br><br>

<hr style="display: block; height: 1px; border: 0; border-top: 3px solid #7CCCC8; margin: 1em 0; padding: 0; ">

<br><br>

### Keep up with rOpenSci

* Mailing list: Sign up with an email address to get this newsletter sent to your inbox -> [ropensci.org/#subscribe](https://ropensci.org/#subscribe)
* Alternatively, you can subscribe to this newsletter via our XML feed at <https://news.ropensci.org/feed.xml> or our JSON feed at <https://news.ropensci.org/feed.json>
* rOpenSci on Twitter: [@ropensci](https://twitter.com/ropensci)
* The rOpenSci blog at [ropensci.org/blog](https://ropensci.org/blog) - you can subscribe in any RSS aggregator, or manually via <https://ropensci.org/feed.xml>. We also announce new blog posts on our Twitter account.

<br>

#### Footnotes

[^1]: Adil, S. H., Ebrahim, M., Ali, S. S. A., & Raza, K. 2019. Identifying Trends in Data Science Articles using Text Mining. In 2019 4th International Conference on Emerging Trends in Engineering, Sciences and Technology (ICEEST) (pp. 1-7). IEEE. <https://ieeexplore.ieee.org/document/8981702>
[^2]: Finoshin, A. D., Adameyko, K. I., Mikhailov, K. V., Kravchuk, O. I., Georgiev, A. A., Gornostaev, N. G., ‚Ä¶ Lyupina, Y. V. (2020). Iron metabolic pathways in the processes of sponge plasticity. PLOS ONE, 15(2), e0228722. <https://doi.org/10.1371/journal.pone.0228722>
[^3]: Pimiento, C., & Benton, M. J. (2020). The impact of the Pull of the Recent on extant elasmobranchs. Palaeontology. <https://doi.org/10.1111/pala.12478>
[^4]: Benhadi‚ÄêMar√≠n, J., Santos, S. A. P., Baptista, P., & Pereira, J. A. (2020). Distribution of Bactrocera oleae (Rossi, 1790) throughout the Iberian Peninsula based on a maximum entropy modeling approach. Annals of Applied Biology. <https://doi.org/10.1111/aab.12584>
[^5]: Ahrenfeldt, J., Waisi, M., Loft, I. C., Clausen, P. T. L. C., Alles√∏e, R., Szarvas, J., ‚Ä¶ Lund, O. (2020). Metaphylogenetic analysis of global sewage reveals that bacterial strains associated with human disease show less degree of geographic clustering. Scientific Reports, 10(1). <https://doi.org/10.1038/s41598-020-59292-w>
[^6]: Cinelli, M. (2019). Generalized rich-club ordering in networks. Journal of Complex Networks, 7(5), 702‚Äì719. <https://doi.org/10.1093/comnet/cnz002>
[^7]: Alfaro-Ram√≠rez, F. U., Ram√≠rez-Albores, J. E., Vargas-Hern√°ndez, J. J., Franco-Maass, S., & P√©rez-Su√°rez, M. (2020). Potential reduction of Hartweg¬¥s Pine (Pinus hartwegii Lindl.) geographic distribution. PLOS ONE, 15(2), e0229178. <https://doi.org/10.1371/journal.pone.0229178>
[^8]: Martinez-Feria, R. A., & Basso, B. (2020). Unstable crop yields reveal opportunities for site-specific adaptations to climate variability. Scientific Reports, 10(1). <https://doi.org/10.1038/s41598-020-59494-2>
[^9]: Rego, A., Sousa, A. G. G., Santos, J. P., Pascoal, F., Can√°rio, J., Le√£o, P. N., & Magalh√£es, C. (2020). Diversity of Bacterial Biosynthetic Genes in Maritime Antarctica. Microorganisms, 8(2), 279. <https://doi.org/10.3390/microorganisms8020279>
[^10]: Cederwall, J., Black, T. A., Blais, J. M., Hanson, M. L., Hollebone, B. P., Palace, V. P., ‚Ä¶ Orihel, D. M. (2020). Life under an oil slick: response of a freshwater food web to simulated spills of diluted bitumen in field mesocosms. Canadian Journal of Fisheries and Aquatic Sciences, 77(5), 779‚Äì788. <https://doi.org/10.1139/cjfas-2019-0224>
[^11]: Mossion, V., Dauphin, B., Grant, J., Zemp, N., & Croll, D. (2020). A reference transcriptome for the early-branching fern Botrychium lunaria enables fine-grained resolution of population structure. <https://doi.org/10.1101/2020.02.17.952283>
[^12]: Funk, B., Sadeh-Sharvit, S., Fitzsimmons-Craft, E. E., Trockel, M. T., Monterubio, G. E., Goel, N. J., ‚Ä¶ Taylor, C. B. (2020). A Framework for Applying Natural Language Processing in Digital Health Interventions. Journal of Medical Internet Research, 22(2), e13855. <https://doi.org/10.2196/13855>
[^13]: Correia, C. N., McLoughlin, K. E., Nalpas, N. C., Magee, D. A., Browne, J. A., Rue-Albrecht, K., ‚Ä¶ MacHugh, D. E. (2018). RNA Sequencing (RNA-Seq) Reveals Extremely Low Levels of Reticulocyte-Derived Globin Gene Transcripts in Peripheral Blood From Horses (Equus caballus) and Cattle (Bos taurus). Frontiers in Genetics, 9. <https://doi.org/10.3389/fgene.2018.00278>
[^14]: Verde Arregoitia, L. D., Teta, P., & D‚ÄôEl√≠a, G. (2020). Patterns in research and data sharing for the study of form and function in caviomorph rodents. Journal of Mammalogy. <https://doi.org/10.1093/jmammal/gyaa002>
[^15]: Shivambu, T. C., Shivambu, N., & Downs, C. T. (2020). Impact assessment of seven alien invasive bird species already introduced to South Africa. Biological Invasions. <https://doi.org/10.1007/s10530-020-02221-9>
[^16]: Jhwueng, D.-C., & O‚ÄôMeara, B. C. (2020). On the Matrix Condition of Phylogenetic Tree. Evolutionary Bioinformatics, 16, 117693432090172. <https://doi.org/10.1177/1176934320901721>
[^17]: Cooke, R. S. C., Eigenbrod, F., & Bates, A. E. (2020). Ecological distinctiveness of birds and mammals at the global scale. Global Ecology and Conservation, 22, e00970. <https://doi.org/10.1016/j.gecco.2020.e00970>
[^18]: Palanisamy, S. (2019). An analysis on hidden markov model prediction factors using California wildfire and meteorological records (Doctoral dissertation, California State University, Sacramento). <https://bit.ly/2yCWk2M>
[^19]: Panayiotou, C. (2020). An Ontological Analysis and Natural Language Processing of Figures of Speech. International Journal of Artificial Intelligence & Applications, 11(1), 17‚Äì30. <https://doi.org/10.5121/ijaia.2020.11102>
[^20]: Ji, Y., Baker, C. C., Li, Y., Popescu, V. D., Wang, Z., Wang, J., ‚Ä¶ Yu, D. W. (2020). Large-scale Quantification of Vertebrate Biodiversity in Ailaoshan Nature Reserve from Leech iDNA. <https://doi.org/10.1101/2020.02.10.941336>
[^21]: Li, X., & Guo, B. (2020). Substantially adaptive potential in polyploid cyprinid fishes: evidence from biogeographic, phylogenetic and genomic studies. Proceedings of the Royal Society B: Biological Sciences, 287(1920), 20193008. <https://doi.org/10.1098/rspb.2019.3008>
[^22]: Silva, I., Crane, M., Marshall, B. M., & Strine, C. T. (2020). Revisiting reptile home ranges: moving beyond traditional estimators with dynamic Brownian Bridge Movement Models. <https://doi.org/10.1101/2020.02.10.941278>
[^23]: Schwalb‚ÄêWillmann, J., Remelgado, R., Safi, K., & Wegmann, M. (2020). moveVis: Animating movement trajectories in synchronicity with static or temporally dynamic environmental data in R. Methods in Ecology and Evolution. <https://doi.org/10.1111/2041-210x.13374>


[RSelenium]: https://github.com/ropensci/RSelenium
[chromer]: https://github.com/ropensci/chromer
[qualtRics]: https://github.com/ropensci/qualtRics
[rsnps]: https://github.com/ropensci/rsnps
[rdpla]: https://github.com/ropensci/rdpla
[webchem]: https://github.com/ropensci/webchem
[stplanr]: https://github.com/ropensci/stplanr
[mregions]: https://github.com/ropensci/mregions
[rWBclimate]: https://github.com/ropensci/rWBclimate
[rinat]: https://github.com/ropensci/rinat

[pdftools]: https://github.com/ropensci/pdftools
[rotl]: https://github.com/ropensci/rotl
[paleobioDB]: https://github.com/ropensci/paleobioDB
[rfishbase]: https://github.com/ropensci/rfishbase
[rgbif]: https://github.com/ropensci/rgbif
[hunspell]: https://github.com/ropensci/hunspell
[CoordinateCleaner]: https://github.com/ropensci/CoordinateCleaner
[FedData]: https://github.com/ropensci/FedData
[rnaturalearth]: https://github.com/ropensci/rnaturalearth
[taxize]: https://github.com/ropensci/taxize
[skimr]: https://github.com/ropensci/skimr
[rredlist]: https://github.com/ropensci/rredlist
[rdflib]: https://github.com/ropensci/rdflib
[NLMR]: https://github.com/ropensci/NLMR
[landscapetools]: https://github.com/ropensci/landscapetools
[magick]: https://github.com/ropensci/magick
[treeio]: https://github.com/YuLab-SMU/treeio
[RefManageR]: https://github.com/ropensci/RefManageR
[bib2df]: https://github.com/ropensci/bib2df
[handlr]: https://github.com/ropensci/handlr
[tradestatistics]: https://github.com/ropensci/tradestatistics
[drake]: https://github.com/ropensci/drake
[rnaturalearth]: https://github.com/ropensci/rnaturalearth
